{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We will continue with the plan delineated in the 12/07 note on going deeper into the simplex CNV analysis. But in this notebook let's focus on the PennCNV analysis, after the joint calling algorithm that we ran in that notebook.\n",
    "\n",
    "So, we start with a traditional PennCNV QC:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "NOTICE: the --qclrrsd argument is set as 0.3 by default\n",
      "NOTICE: the --qcbafdrift argument is set as 0.01 by default\n",
      "NOTICE: the --qcwf argument is set as 0.05 by default\n",
      "WARNING: CNV calls from 97 files are skipped due to lack of QC measure in qclogfile all_simplex.log\n",
      "NOTICE: Writting 0 file names that pass QC to qcpass file all_simplex.qcpass\n",
      "NOTICE: Writting 0 records of QC summary to qcsum file all_simplex.qcsum\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "module load penncnv\n",
    "\n",
    "cd /data/sudregp/cnv/penncnv/results\n",
    "cat CCGO*.log CLIA*.log > all_simplex.log\n",
    "cat CCGO*.jointcnv CLIA*.jointcnv > all_simplex.jointcnv\n",
    "filter_cnv.pl all_simplex.jointcnv -qclogfile all_simplex.log \\\n",
    "    -qcpassout all_simplex.qcpass \\\n",
    "    -qcsumout all_simplex.qcsum -out all_simplex"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "So, it turns out that the default values used in QC are not output to the log file if using the joint call. Does that mean that joint calls are not as affected by those metrics? Let's go on with the analysis without doing QC, and then we can check if there any any outliers. If yes, then we can do the usual rawcnv call just to get those QC metrics, and apply them to the joint calls.\n",
    "\n",
    "So, before we try to import these results into pseq, let's do a quick check of the numbers:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CCGO_800979.jointcnv: 1 denovo, 28 inherited\n",
      "CCGO_800980.jointcnv: 3 denovo, 43 inherited\n",
      "CLIA_400121.jointcnv: 0 denovo, 110 inherited\n",
      "CLIA_400122.jointcnv: 3 denovo, 110 inherited\n",
      "CLIA_400123.jointcnv: 3 denovo, 53 inherited\n",
      "CLIA_400125.jointcnv: 283 denovo, 71 inherited\n",
      "CLIA_400126.jointcnv: 382 denovo, 457 inherited\n",
      "CLIA_400127.jointcnv: 10 denovo, 71 inherited\n",
      "CLIA_400128.jointcnv: 1 denovo, 84 inherited\n",
      "CLIA_400129.jointcnv: 1 denovo, 94 inherited\n",
      "CLIA_400130.jointcnv: 0 denovo, 215 inherited\n",
      "CLIA_400131.jointcnv: 4 denovo, 161 inherited\n",
      "CLIA_400132.jointcnv: 77 denovo, 263 inherited\n",
      "CLIA_400133.jointcnv: 1 denovo, 188 inherited\n",
      "CLIA_400134.jointcnv: 3 denovo, 148 inherited\n",
      "CLIA_400135.jointcnv: 8 denovo, 325 inherited\n",
      "CLIA_400138.jointcnv: 16 denovo, 120 inherited\n",
      "CLIA_400139.jointcnv: 49 denovo, 269 inherited\n",
      "CLIA_400140.jointcnv: 2 denovo, 196 inherited\n",
      "CLIA_400142.jointcnv: 29 denovo, 284 inherited\n",
      "CLIA_400144.jointcnv: 0 denovo, 143 inherited\n",
      "CLIA_400148.jointcnv: 5 denovo, 117 inherited\n",
      "CLIA_400149.jointcnv: 600 denovo, 152 inherited\n",
      "CLIA_400152.jointcnv: 0 denovo, 152 inherited\n",
      "CLIA_400153.jointcnv: 7 denovo, 89 inherited\n",
      "CLIA_400155.jointcnv: 57 denovo, 282 inherited\n",
      "CLIA_400158.jointcnv: 45 denovo, 116 inherited\n",
      "CLIA_400159.jointcnv: 2 denovo, 177 inherited\n",
      "CLIA_400162.jointcnv: 95 denovo, 226 inherited\n",
      "CLIA_400163.jointcnv: 359 denovo, 195 inherited\n",
      "CLIA_400166.jointcnv: 11 denovo, 193 inherited\n",
      "CLIA_400170.jointcnv: 0 denovo, 172 inherited\n",
      "CLIA_400172.jointcnv: 229 denovo, 111 inherited\n",
      "CLIA_400175.jointcnv: 50 denovo, 144 inherited\n",
      "CLIA_400178.jointcnv: 5 denovo, 56 inherited\n",
      "CLIA_400179.jointcnv: 30 denovo, 50 inherited\n",
      "CLIA_400180.jointcnv: 0 denovo, 105 inherited\n",
      "CLIA_400181.jointcnv: 286 denovo, 68 inherited\n",
      "CLIA_400189.jointcnv: 0 denovo, 155 inherited\n",
      "CLIA_400191.jointcnv: 0 denovo, 160 inherited\n",
      "CLIA_400193.jointcnv: 0 denovo, 179 inherited\n",
      "CLIA_400195.jointcnv: 4 denovo, 344 inherited\n",
      "CLIA_400200.jointcnv: 9 denovo, 306 inherited\n",
      "CLIA_400201.jointcnv: 0 denovo, 169 inherited\n",
      "CLIA_400203.jointcnv: 916 denovo, 24 inherited\n",
      "CLIA_400204.jointcnv: 0 denovo, 190 inherited\n",
      "CLIA_400205.jointcnv: 1 denovo, 222 inherited\n",
      "CLIA_400209.jointcnv: 3 denovo, 280 inherited\n",
      "CLIA_400210.jointcnv: 0 denovo, 350 inherited\n",
      "CLIA_400212.jointcnv: 1 denovo, 168 inherited\n",
      "CLIA_400216.jointcnv: 6 denovo, 79 inherited\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd /data/sudregp/cnv/penncnv/results\n",
    "\n",
    "for triocnv in `ls -1 C*.jointcnv`; do\n",
    "    rm denovo.txt inherited.txt 2>/dev/null\n",
    "    grep mother ${triocnv} > mom_snps;\n",
    "    grep father ${triocnv} > dad_snps;\n",
    "    cat mom_snps dad_snps > parent_snps;\n",
    "    for snp in `grep offspring ${triocnv} | cut -d' ' -f 1`; do\n",
    "        if ! grep -q $snp parent_snps; then\n",
    "            echo \"$snp\" >> denovo.txt\n",
    "        else\n",
    "            echo \"$snp\" >> inherited.txt\n",
    "        fi;\n",
    "   done\n",
    "   echo ${triocnv}: `cat denovo.txt 2>/dev/null | wc -l` denovo, `cat inherited.txt 2>/dev/null | wc -l` inherited;\n",
    "   rm *_snps;\n",
    "done"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Well, there are a few trios that would make you wonder about their data quality. Let me create a QC file and see if these are samples that would be excluded (in terminal):"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "module load penncnv\n",
    "\n",
    "cd /data/sudregp/cnv/penncnv\n",
    "\n",
    "pfb_file=InfiniumExome.pfb\n",
    "gc_file=InfiniumExome.hg19.gcmodel\n",
    "detect_cnv.pl -test -hmm ~/autodenovo/penncnv_example.hmm -pfb $pfb_file -log results/all_simplex_but1.log InfiniumExome/* -out results/all_simplex_but1.rawcnv;\n",
    "\n",
    "pfb_file=HumanExome.pfb\n",
    "gc_file=HumanExome.hg19.gcmodel\n",
    "detect_cnv.pl -test -hmm ~/autodenovo/penncnv_example.hmm -pfb $pfb_file -log results/fam_10369.log HumanExome/* -out results/fam_10369.rawcnv;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "NOTICE: the --qclrrsd argument is set as 0.3 by default\n",
      "NOTICE: the --qcbafdrift argument is set as 0.01 by default\n",
      "NOTICE: the --qcwf argument is set as 0.05 by default\n",
      "NOTICE: Writting 122 file names that pass QC to qcpass file all_simplex.qcpass\n",
      "NOTICE: Writting 130 records of QC summary to qcsum file all_simplex.qcsum\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "module load penncnv\n",
    "\n",
    "cd /data/sudregp/cnv/penncnv/results\n",
    "cat all_simplex_but1.log fam_10369.log > all_simplex_rawcnv.log\n",
    "cat all_simplex_but1.rawcnv fam_10369.rawcnv > all_simplex.rawcnv\n",
    "filter_cnv.pl all_simplex.rawcnv -qclogfile all_simplex_rawcnv.log \\\n",
    "    -qcpassout all_simplex.qcpass \\\n",
    "    -qcsumout all_simplex.qcsum -out all_simplex"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "import matplotlib.pyplot as plt\n",
    "import seaborn as sns\n",
    "import pandas as pd\n",
    "\n",
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# figure out who is who in each trio\n",
    "import glob\n",
    "data_dir = '/data/sudregp/cnv/penncnv/'\n",
    "ped_file = '/data/sudregp/cnv/simplex.ped'\n",
    "wes_prefix = ['CLIA', 'CCGO', 'WPS']\n",
    "trios = []\n",
    "affected = []\n",
    "controls = []\n",
    "samples = []\n",
    "fid = open(ped_file, 'r')\n",
    "for line in fid:\n",
    "    famid, sid, fa, mo, sex, aff = line.rstrip().split('\\t')\n",
    "    if fa[:4] in wes_prefix and mo[:4] in wes_prefix and sid[:4] in wes_prefix:\n",
    "        fam = {}\n",
    "        fam['child'] = sid\n",
    "        if aff == '1':\n",
    "            affected.append(sid)\n",
    "        else:\n",
    "            controls.append(sid)\n",
    "        fam['father'] = fa\n",
    "        fam['mother'] = mo\n",
    "        fam['famid'] = famid\n",
    "        trios.append(fam)\n",
    "        samples += [sid, fa, mo]\n",
    "fid.close()\n",
    "samples = set(samples)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<seaborn.axisgrid.JointGrid at 0x2aaaf1a6d5d0>"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAa8AAAGoCAYAAADxbmq5AAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAALEgAACxIB0t1+/AAAIABJREFUeJzt3Xt0VfWd9/HP15BopDpcq5JIxUoBuSQIgi4cFZWb4HjB\nesOpWvu4mKXTsT6Nwmpt69NxFRs747I6w6M+Ym9KqwJFhxqraFtHxxIFBdRYRKoElWsslwhJ+D5/\nnJN4SM4J54Rz++W8X2tlcfbt7O/ZhPNh//Zv/7a5uwAACMlhuS4AAIBUEV4AgOAQXgCA4BBeAIDg\nEF4AgOAQXgCA4BBeAIDgEF4AgOAQXgCA4PTIdQFpxnAhAEJnuS4gBJx5AQCC093OvLrk0Vc/iDv/\nqvEDs1wJACAZnHkBAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkN4AQCCQ3gB\nAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkN4AQCC\nQ3gBAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkN4\nAQCCQ3gBAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkN4AQCC0yPXBeSzR1/9\nIO78q8YPzHIlAIBYnHkBAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkNvwy6I1wuRHogAkD2EVx4hFIEw\n8W83+wivNEl0T1g8qfxSp3KvWb7cl8Y/ZACZZu6e6xrSxsyekdSvC5v2k7Q1zeV0Vb7UQh0d5Ust\n1NFRvtSSjjq2uvvUdBTTnXWr8OoqM6t197G5rkPKn1qoo6N8qYU6OsqXWvKljkJAb0MAQHAILwBA\ncAiviAdyXUCMfKmFOjrKl1qoo6N8qSVf6uj2uOYFAAgOZ14AgOAQXgCA4BBeAIDgEF4AgOB0q/Ca\nOnWqS+KHH374Cfknad3wOy9p3Sq8tm7Nh9FhACA7Cvk7r1uFFwCgMBBeAIDgEF4AgODwPC/kvaam\nJm3cuFGfffZZrksB0uaII45QeXm5iouLc11KkAgv5L2NGzfqqKOO0gknnCAzy3U5wCFzd23btk0b\nN27UoEGDcl1OkGg2RN777LPP1LdvX4IL3YaZqW/fvrQmHALCC0EguNDd8Dt9aAgvAEBwCC8AB3jm\nmWc0ZMgQnXTSSZo3b17cdT799FNdcMEFqqio0PDhw7VgwQJJkSbecePGtc3//ve/n83S21RXV6uy\nslKVlZUaMWKEioqKtH379g7rLV++XKeccopGjBiha665Rs3NzSntp6qqSkOHDtWoUaN08cUXq6Gh\noW3Zm2++qdNPP13Dhw/XyJEjaSJMN3fP6I+kqZLqJK2TNCfO8gslvSlplaRaSWfELNsgaXXrsoPt\na8yYMY7u56233sp1CRnR1NSUkfdtbm4+pG1PPPFEf++993zv3r0+atQoX7t2bYf17rzzTr/11lvd\n3X3z5s3eu3dv37t3r+/fv9937tzp7u779u3zcePG+SuvvNLletJh6dKlPnHixA7zW1pavLy83Ovq\n6tzd/fbbb/eHHnoopfeuqalp+3u89dZb245JU1OTjxw50letWuXu7lu3bo3795Lgdzvp79du+J2X\n9GfP6JmXmRVJul/SNEknS7rSzE5ut9rzkircvVLS1yU91G75RHevdPexmawVSGTDhg0aOnSoZs2a\npWHDhunSSy/Vnj17JEmvvfaazjrrLI0ZM0ZTpkzRRx99JEl68MEHdeqpp6qiokIzZ85sW//aa6/V\n7NmzNX78eN166636wx/+0HaGMHr0aO3cuVPurqqqKo0YMUIjR47Ur3/9a0nSiy++qLPPPluXXnpp\nWz0efZjsCSecoNtuu02nnHKKHn/88S5/1j//+c866aSTdOKJJ6qkpERXXHGFfvvb33ZYz8zaat21\na5f69OmjHj16yMz0hS98QVLkFoempqa2azvz58/X/PnzO7zXI488ogsvvFBnn322Bg8erDvuuKPL\n9cfz2GOP6corr+wwf9u2bSopKdFXvvIVSdKkSZP05JNPSpJ2796tr3/96xo3bpxGjx4d9xhI0uTJ\nk9WjR6TT9mmnnaaNGzdKkp599lmNGjVKFRUVkqS+ffuqqKgorZ+r4KWSdKn+SDpdUk3M9FxJcw+y\n/tsx0xsk9Ut2f93wfyHw3J95vf/++y7JX3rpJXd3v+6667y6utr37dvnp59+um/evNnd3RcuXOjX\nXXedu0f+p93qO9/5jt97773u7n7NNdf49OnT2/4XPmPGjLb33blzpzc1NfkTTzzh5513njc3N/vH\nH3/sxx9/vG/atMlfeOEFP/roo/3DDz/0lpYWP+200/xPf/qTu7t/6Utf8rvuuitu/b/85S+9oqKi\nw8/MmTM7rPv444/79ddf3zb985//3G+88cYO6/3tb3/zs88+24899ljv2bOnP/30023LmpubvaKi\nwnv27Nl2JtKZBQsW+LHHHutbt271PXv2+PDhw33FihUd1rvsssvifo6f/exnCd979+7d3rt3b9+2\nbVuHZfv37/eBAwe27eub3/ymjxgxwt3d586d67/4xS/c3X3Hjh0+ePBg37VrV6efY8aMGW3b/Pu/\n/7tfffXVPnnyZB89enTCvxvOvDpI+rNn+j6vMkkfxkxvlDS+/UpmdrGkH0n6oqTpMYtc0nNm1iLp\n/7r7A3G2vUHSDZI0cODA9FUOxDj++OM1YcIESdLVV1+te++9V1OnTtWaNWs0adIkSVJLS4uOO+44\nSdKaNWv03e9+Vw0NDdq1a5emTJnS9l5f/epX2/4XPmHCBN1yyy2aNWuWLrnkEpWXl+ull17SlVde\nqaKiIh1zzDE666yztGLFCh199NEaN26cysvLJUmVlZXasGGDzjjjDEnS5ZdfHrf2WbNmadasWWk9\nHjU1NaqsrNTy5cv13nvvadKkSfr7v/97HX300SoqKtKqVavU0NCgiy++WGvWrNGIESM6fb9Jkyap\nb9++kqRLLrlEL730ksaOPbCxpfUMNBVPPfWUJkyYoD59+nRYZmZauHChvvWtb2nv3r2aPHly29/L\ns88+q6VLl+ruu++WFLmW98EHH2jYsGFx93PnnXeqR48ebce5ublZL730klasWKEjjzxS5557rsaM\nGaNzzz035c8Qp26+85QnNym7+2JJi83sTEk/lHRedNEZ7l5vZl+U9Hsze8fd/9hu2wckPSBJY8eO\nTWlIfSBZ7bs1m5ncXcOHD9crr7zSYf1rr71WS5YsUUVFhR555BG9+OKLbct69uzZ9nrOnDmaPn26\nli1bpgkTJqimpqbTOg4//PC210VFRQd0MIh931i/+tWvVF1d3WH+SSedpCeeeOKAeWVlZfrww8//\nv7lx40aVlZV12HbBggWaM2eOzEwnnXSSBg0apHfeeUfjxo1rW6dXr16aOHGinnnmmYOGV7zj297l\nl1+uurq6DvNvueUWfe1rX4v7vgsXLozbZNjq9NNP15/+9CdJkcB69913JUVapJ588kkNGTLkgPWv\nu+46rVy5UgMGDNCyZcskRZo9n376aT3//PNtdZeXl+vMM89Uv379JEnnn3++Xn/99bSEF995Uamc\npqX6oxSbDaPrrFecpkJJP5D07c627Yan0PD8aTZ8+eWX3d39+uuv97vvvtv37t3rX/7yl9vm79u3\nz9esWePu7n379vVPPvnE9+3b5+edd55fc8017h5pNnz88cfb3nvdunVtr2fOnOmLFy/2J5980idP\nnuzNzc2+efNmHzhwoH/00Uf+wgsv+PTp09vWv/HGG33BggXuHmk23LJlyyF/1qamJh80aJCvX7++\nrcNG62eKNXv2bP/+97/v7u4ff/yxDxgwwLds2eKbN2/2HTt2uLv7nj17/IwzzvCnnnrK3d1/+tOf\n+k9/+tMO77VgwQI/7rjjfNu2bb5nzx4fOXJk3GbDVDU0NHjv3r07be775JNP3N39s88+83POOcef\nf/55d480G954442+f/9+d3d//fXX427/u9/9zocNG9bWdNxq+/btPnr0aN+9e7c3NTX5ueeee0DT\naiuaDTvIjw4bklZIGmxmg8ysRNIVkpbGrmBmJ1n0vytmdoqkwyVtM7OeZnZUdH5PSZMlrclwvUBc\nQ4YM0f33369hw4Zpx44d+qd/+ieVlJToiSee0G233aaKigpVVlbq5ZdfliT98Ic/1Pjx4zVhwgQN\nHTo04fvec889GjFihEaNGqXi4mJNmzZNF198cdvF/nPOOUc//vGPdeyxx2blc/bo0UP33XefpkyZ\nomHDhumyyy7T8OHDJR3Y4eL222/Xyy+/rJEjR+rcc8/VXXfdpX79+umjjz7SxIkTNWrUKJ166qma\nNGmSZsyYIUl655132poG2xs3bpxmzpypUaNGaebMmR2aDLti8eLFmjx5cocz0vPPP1+bNm2SFOlS\nP2zYMI0aNUoXXHCBzjnnnLbP19TUpFGjRmn48OG6/fbb4+7jpptu0s6dOzVp0iRVVlZq9uzZkqTe\nvXvrlltu0amnnqrKykqdcsopmj59etz3QNeYe2bPOs3sfEn3SCqS9LC732lmsyXJ3eeb2W2Sviap\nSVKjpCp3f8nMTpS0OPo2PSQ96u53dravsWPHem1tbaY+CnLk7bffTnitIRs2bNigGTNmaM0a/u90\nKGbMmKFFixappKTkgPmPPPKIamtrdd999+WostxJ8Lud9NAb3fA7L+nPnvFrXu6+TNKydvPmx7y+\nS9JdcbZbL6ki0/UByI6nn3461yWgG8mLDhtAPjvhhBM468qga6+9Vtdee22uy0BgGB4KQch08zaQ\nbfxOHxrOvJD3jjjiCG3bto3HoiAIO/bs0yeffqZ9LftVUnSYjvm7I9T7yAOv87lHnud1xBFH5KjK\n8BFeyHvl5eXauHGjtmzZkutSgE7t2deshj1N2h9zUlVvUq8ji3VkyYFft61PUkbXEF7Ie8XFxTxt\nFkGYMG+56hsaO8wv61Wq/55zTg4q6r645gUAabIpTnB1Nv9Qbd+9T4+++kFG3jvfEV4AkCYDepWm\nNB9dR3gBQJpUTRmi0uIDH31SWlykqilDEmyBruKaFwCkyUWjI4MYV9fUaVNDowb0KlXVlCFt85E+\nhBcApNFFo8sIqyyg2RAAEBzCCwAQHMILABAcwgsAEBzCCwAQHMILAALVp2eJrho/MNdl5AThBQAI\nDuEFAAgO4QUACA7hBQAIDuEFAAgO4QUACA7hBQAIDuEFAAgO4QUACA7hBQAIDuEFAAgO4QUACA7h\nBQAIDuEFAAgO4QUACA7hBQAIDuEFAAgO4QUACA7hBQAIDuEFAAgO4QUACA7hBQAIDuEFAAgO4QUA\nCA7hBQAIDuEFAAgO4QUACE7Gw8vMpppZnZmtM7M5cZZfaGZvmtkqM6s1szOS3RYAUJgyGl5mViTp\nfknTJJ0s6UozO7ndas9LqnD3Sklfl/RQCtsCAApQps+8xkla5+7r3X2fpIWSLoxdwd13ubtHJ3tK\n8mS3BQAUpkyHV5mkD2OmN0bnHcDMLjazdyT9lyJnX0lvCwAoPHnRYcPdF7v7UEkXSfphKtua2Q3R\na2W1W7ZsyUyBAJAn+M6LyHR41Us6Pma6PDovLnf/o6QTzaxfstu6+wPuPtbdx/bv3z89VQNAnuI7\nLyLT4bVC0mAzG2RmJZKukLQ0dgUzO8nMLPr6FEmHS9qWzLYAgMLUI5Nv7u7NZnaTpBpJRZIedve1\nZjY7uny+pJmSvmZmTZIaJV0e7cARd9tM1gsACIN93tEvfGPHjvXa2tpclwEAh8KSXbEbfucl/dnz\nosMGAACpILwAAMEhvAAAwSG8AADBIbwAAMEhvAAAwSG8AADBIbwAAMEhvAAAwSG8AADBIbwAAMEh\nvAAAwSG8AADBIbwAAMEhvAAAwSG8AADBIbwAIFDbd+/To69+kOsycoLwAgAEp0euCwCA7mTJynpV\n19RpU0OjBvQqVdWUIbpodFmuy+p2CC8ASJMlK+s1d9FqNTa1SJLqGxo1d9FqSSLA0oxmQwBIk+qa\nurbgatXY1KLqmrocVdR9EV4AkCabGhpTmo+uI7wAIE0G9CpNaT66jvACgDSpmjJEpcVFB8wrLS5S\n1ZQhOaqo+6LDBgCkSWunDHobZh7hBQBpdNHoMsIqC2g2BIBA9elZoqvGD8x1GTlBeAEAgkN4AQCC\nQ3gBAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkN4\nAQCCQ3gBAIJDeAEAgkN4AQCCk/HwMrOpZlZnZuvMbE6c5bPM7E0zW21mL5tZRcyyDdH5q8ysNtO1\nAgDC0COTb25mRZLulzRJ0kZJK8xsqbu/FbPa+5LOcvcdZjZN0gOSxscsn+juWzNZJwAgLJk+8xon\naZ27r3f3fZIWSrowdgV3f9ndd0Qn/0dSeYZrAgAELtPhVSbpw5jpjdF5iVwv6Xcx0y7pOTN7zcxu\niLeBmd1gZrVmVrtly5ZDLhgA8lnsd95f6z/Wo69+kOuSciJvOmyY2URFwuu2mNlnuHulpGmSbjSz\nM9tv5+4PuPtYdx/bv3//LFULALkR+513VK8+uS4nZzIdXvWSjo+ZLo/OO4CZjZL0kKQL3X1b63x3\nr4/+uVnSYkWaIQEABS7T4bVC0mAzG2RmJZKukLQ0dgUzGyhpkaR/dPd3Y+b3NLOjWl9LmixpTYbr\nBQAEIKO9Dd292cxuklQjqUjSw+6+1sxmR5fPl/Q9SX0l/YeZSVKzu4+VdIykxdF5PSQ96u7PZLJe\nAEAYMhpekuTuyyQtazdvfszrb0j6Rpzt1kuqaD8fAIC86bABAECyCC8AQHAy3myIzFuysl7VNXXa\n1NCoAb1KVTVliC4a3dntdAAQNsIrcEtW1mvuotVqbGqRJNU3NGruotWSRIAB6LZoNgxcdU1dW3C1\namxqUXVNXY4qAoDMI7wCt6mhMaX5ALqPPj1LdNX4gbkuIycIr8AN6FWa0nwA6A4Ir8BVTRmi0uKi\nA+aVFhepasqQHFUEAJlHh43AtXbKoLchgEJCeHUDF40uI6wAFBSaDQEAwSG8AADBIbwAAMEhvAAA\nwSG8AADBSaq3oZn1lXSVpKHRWW9Leszdt2WqMAAAEjnomZeZDZO0RtIYSe9K+oukUyWtNrOhnW0L\nAEAmJHPm9UNJ/+Luv4mdaWYzJd0paWYmCgMAIJFkrnmNbB9ckuTuT0oakf6SAADoXDLhtbuLywAA\nyIhkmg2/aGa3xJlvkvqnuR4AAA4qmfB6UNJRCZY9lMZaAABIykHDy93vyEYhAAAkK5mu8v/LzAZH\nX5uZPWxmn5rZm2Y2OvMlAgBwoGQ6bPyLpA3R11dKqpB0oqRbJN2bmbIAAEgsmfBqdvem6OsZkn7u\n7tvc/TlJPTNXGgAA8SUTXvvN7DgzO0LSuZKei1lWmpmyAABILJneht+TVCupSNJSd18rSWZ2lqT1\nGawNAIC4kult+LSZfUnSUe6+I2ZRraTLWyfMbJK7/z4DNQIAcICkHoni7s3tgkvuvtvdd8XMuiut\nlQEAkEA6n+dlaXwvAAASSmd4eRrfCwCAhHiSMgAgOOkMrw1pfC8AABLqcniZ2SQza+td6O6XpKck\nAAA6l8zYhueY2btmtsvMfmlmI82sVtI8Sf+Z+RIBADhQMmdeP5F0g6S+kp6Q9IqkR9x9jLsvymRx\nAADEk8wIG+7uL0ZfLzGzene/L4M1AQDQqWTCq5eZxV7P6hE7zdkXACDbkgmvP0i6IGb6jzHTLonw\nAgBkVTJjG16XjUIAAEhWUl3lzazIzPrFTJeY2Q1m9nbmSgMAIL5kuspfIWm7pDfN7A9mNlmRR6FM\nkzQrw/UBANBBMte8vitpjLuvM7NTFOkqf6m7P5XZ0gAAiC+ZZsN97r5Oktz9dUl/SSW4zGyqmdWZ\n2TozmxNn+Swze9PMVpvZy2ZWkey2AIDClMyZ1xfN7JaY6V6x0+7+b4k2NLMiSfdLmiRpo6QVZrbU\n3d+KWe19SWe5+w4zmybpAUnjk9wWAFCAkjnzelDSUTE/sdNfOMi24yStc/f17r5P0kJJF8au4O4v\nxzzo8n8klSe7LQCgMCXTVf6ORMvM7OaDbF4m6cOY6Y2Sxney/vWSfpfKtmZ2gyLDV2ngwIEHKQcA\nwsZ3XsShPhLlloOvkhwzm6hIeN2Wynbu/oC7j3X3sf37909XOQCQl/jOi0jmmldn7CDL6yUdHzNd\nHp134JuYjZL0kKRp7r4tlW0BAIXnUM+8/CDLV0gabGaDzKxE0hWSlsauYGYDFRli6h/d/d1UtgUA\nFKaDnnmZ2U7FDymTVNrZtu7ebGY3SaqRVCTpYXdfa2azo8vnS/qeIo9b+Q8zk6Tm6Clx3G2T/2gA\ngO4qmQ4bRx3KDtx9maRl7ebNj3n9DUnfSHZbAAAOtdkQAICsI7wAAMEhvAAAwTnUrvLIoiUr61Vd\nU6dNDY0a0KtUVVOG6KLRZbkuCwCyjvAKxJKV9Zq7aLUam1okSfUNjZq7aLUkEWBAgdq+e58effWD\nDvOvGt/9R96g2TAQ1TV1bcHVqrGpRdU1dTmqCAByh/AKxKaGxpTmA0B3RngFYkCv+PeDJ5oPAN0Z\n4RWIqilDVFpcdMC80uIiVU0ZkqOKACB36LARiNZOGfQ2BADCK2nZ7qaeaH+EFQAQXknJdjd1usUD\nQOe45pWEbHdT7w7d4pesrNeEecs1aM5/acK85VqykkexAUgfzrySkO1u6qF3i+fMEUCmceaVhGx3\nUw+9W3x3OHMEkN8IryRku5t66N3iQz9zBJD/aDZMQja7qbf2MmxsalGRmVrcVRZYt/gBvUpVHyeo\nQjlzBJD/CK8kZaObevtrRS3ubWdcoQSXFDlzjP0cUlhnjgDyH82GeaS7XCu6aHSZfnTJSJX1KpVJ\nKutVqh9dMjKoAAaQ3zjzSlEmb1buTteKuKEaQCZx5pWC1ma9+oZGuT7vAp6ue5hC72UIANlCeKUg\n0816ofcyBIBsodkwBZlu1mPwXQBIDuGVQLxrW9noAs61IgA4OJoN40h0bWvi0P406wFAHuDMK45E\n17ZeeGeLfnTJSJr1AOS1R1/9oNPlV40fmKVKMofwUscmwnhNg5La5v/3nHPatvnWr1epuqaOEAOA\nLCr48Io3ArpJ8gTrVz3+hmr/ul1PvlbPqOkAkCMFf80rXhNhouCSpKb9rsde/bBbjIQBAKEq+DOv\nrnRzb/H48Vbf0KgJ85ZzPQwAMqzgz7y60s29yCzufJMyNvoGAOBzBR9e8Ua16EzxYaYrxx/fYZt4\n18loSgSAzCj48Io3AnrvI4vjrmuSqr9aoX+9aGSHbRJdJwtxUF0AyHcFf81L6jiqRfseiFLkZuTY\nx3q032bCvOU8gBEAsoTwilqysl53PLVWO/Y0SZJKiw9T7yOL1bCnKanOF509gDGTj1EBgEJU8M2G\nUiS4qp54oy24JKmxab927GmSS/r40890869XacK85Qk7YCR6AKOkjD5GBQAKEWdeitzr1dSS+O6u\n1q7xB7sZOd6guhPmLU94TxhnXwBy4WDDR8XK16GkOPNSap0qUu1B2J2ejgwA+YLwUuqdKlIJHp6O\nDADpR3gp0tmiuCj+jcfxpBI8PB0ZANKv4K95tfYE7OyaV6xUg4enIwNA+hV0eMW7n6szZb1KNXFo\n/7ZHoSQbRDwdGQDSq6DDK96I8vGYpPfnTY/7+BQehQIA2VfQ17yS7XjReo0r0ROW//dv3uC+LQDI\nooyHl5lNNbM6M1tnZnPiLB9qZq+Y2V4z+3a7ZRvMbLWZrTKz2nTX1ivBGIbt7d7brCUr6xOGXYs7\nNx4DQBZlNLzMrEjS/ZKmSTpZ0pVmdnK71bZL+qakuxO8zUR3r3T3semuL8FjuTpoaGzSzb9epQRP\nQpHECPIAkE2ZvuY1TtI6d18vSWa2UNKFkt5qXcHdN0vabGbTM1xLB582Nh18pRj7DxJ23HgMoLtJ\ndjSObI/EkelmwzJJH8ZMb4zOS5ZLes7MXjOzG+KtYGY3mFmtmdVu2bIlpeLSfaMwNx4DyLTY77yd\nDdtzXU7O5HuHjTPcvVKRZscbzezM9iu4+wPuPtbdx/bv3z+lN0/1QZSxuPEYQC7Efucd1atPrsvJ\nmUyHV72k42Omy6PzkuLu9dE/N0tarEgzZNpcNLpMM8ek3sW9dcT49iPI010eALIj09e8VkgabGaD\nFAmtKyRdlcyGZtZT0mHuvjP6erKk/5PuAp9+46OU1m89w+LGYwDInYyGl7s3m9lNkmokFUl62N3X\nmtns6PL5ZnaspFpJR0vab2Y3K9IzsZ+kxRbp4tdD0qPu/ky6a2xIotOGKXLxrYyhnQAgL2R8hA13\nXyZpWbt582Nef6xIc2J7f5NUkdnqktMaXP8955xclwIAUP532Mi43kneqFxPN3gAyBsFHV5LVtZr\nX/P+pNYt6uwOZQBAVhXswLxLVtar6ok3kn4USkuyw3EAADKuYM+8UnmGlxQ58xo05780Yd5yxjAE\ngBwr2DOvVIdyaj3z4jEoANBRssNIxdOVoaUK9szrUIZyamxq0R1PrU1jNQCAVBRseB3qUE479jTR\nfAgAOVKw4ZWOJj8egQIAuVGw4SVFbjw+FDwCBQByo6DD61CbDnkECgDkRkGH16E0HfIIFADInYLt\nKt8VRWba764BDNALADlFeCWptLiIZ3YBQJ4o6GbDVBBcAJA/OPNKQlmvUoILANKkKyNqtFfwZ16H\n9zj4IaBjBgDkl4IPr71JPBKFsy4AyC8FH14Hc2QxhwgA8g3fzAexp2m/vrtkda7LAADEILyS8Mv/\n+YBBeAEgjxBeSWIQXgDIH4RXkhiEFwDyB+GVJAbhBYD8QXglofgw414vAMgjhNdBHFl8mKq/WsG9\nXgCQRwp+eKieJUXava8l7rKrTxuof71oZJYrAoCuScewS6Eo+DOvOy9OHE4vvLMli5UAAJJV8OHV\nWXMgPQxGXWN6AAAJ2UlEQVQBID8VfHhJkVHj46GHIQDkJ8JLkVHjS4uLDphXWlxED0MAyFOEV1Ts\no1F6H1nMwycBII8VfG/DJSvrNXfRajU2fd7j8LOmgz8mBQCQOwV/5lVdU3dAcElSY1MLYxkCQB4r\n+PCqT9CjkJ6GAJC/Cjq8lqyslyVYRk9DAMhfBX3Nq7qmTh5nvkn0NAQQjEIaWaNVQZ95JWoadHV+\n8zIAILcKOrwSNQ0mumkZAJAfCjq8uDkZAMJU0Ne8WpsGq2vqtKmhUQN6lapqyhCaDAEgzxV0eEmR\nACOsACAsBd1sCAAIE+EFAAgO4QUACE7Gw8vMpppZnZmtM7M5cZYPNbNXzGyvmX07lW0BAIUpo+Fl\nZkWS7pc0TdLJkq40s5PbrbZd0jcl3d2FbQEABSjTvQ3HSVrn7uslycwWSrpQ0lutK7j7ZkmbzWx6\nqtsCAKRHX/3ggOlCGC4q082GZZI+jJneGJ2Xtm3N7AYzqzWz2i1btnS5UAAIQex33s6G7bkuJ2eC\n77Dh7g+4+1h3H9u/f/9clwMAGRX7nXdUrz65LidnMh1e9ZKOj5kuj87L9LYAgG4s0+G1QtJgMxtk\nZiWSrpC0NAvbAgC6sYx22HD3ZjO7SVKNpCJJD7v7WjObHV0+38yOlVQr6WhJ+83sZkknu/vf4m2b\n7hqXrKzXHU+t1Y49TZKkXqXF+sE/DGfIKADIYxkf29Ddl0la1m7e/JjXHyvSJJjUtum0ZGW9qp54\nQ00tnz+SsqGxSVWPvyGJZ3oBQL4KvsPGoaiuqTsguFo17Xf979+8oSUrucQGAPmooMMr0ZOUJanF\nXXMXrSbAACAPFXR4JXqScqvGphZV19RlqRoAQLIK+nleVVOGdLjm1V5nZ2cAkEt9epYUxGga8RR0\neLV2yIjtbdjewc7OAADZV9DNhlIkwFZ+b7LuubxSpcVFBywrLS5S1ZQhOaoMAJBIQZ95xWo9C6uu\nqdOmhkYN6FWqqilD6C4PAHmI8Ipx0egywgoAAlDwzYYAgPAQXgCA4BBeAIDgEF4AgOAQXgCA4BBe\nAIDgEF4AgOAQXgCA4BBeAIDgEF4AgOAQXgCA4BBeAIDgEF4AgOAQXgCA4BBeAIDgEF4AgOAQXgCA\n4BBeAIDgEF4AgOAQXgCA4Ji757qGtDGzLZL+2oVN+0namuZyuipfaqGOjvKlFuroKF9qSUcdW919\najIrmtkzya7b3XSr8OoqM6t197G5rkPKn1qoo6N8qYU6OsqXWvKljkJAsyEAIDiEFwAgOIRXxAO5\nLiBGvtRCHR3lSy3U0VG+1JIvdXR7XPMCAASHMy8AQHAILwBAcAo+vMxsqpnVmdk6M5uT5X1vMLPV\nZrbKzGqj8/qY2e/N7C/RP3tnaN8Pm9lmM1sTMy/hvs1sbvQY1ZnZlAzX8QMzq48el1Vmdn4W6jje\nzF4ws7fMbK2Z/Ut0flaPSSd1ZPWYmNkRZvZnM3sjWscd0fm5+B1JVEvWf0+i711kZivN7OnodNaP\nCSS5e8H+SCqS9J6kEyWVSHpD0slZ3P8GSf3azfuxpDnR13Mk3ZWhfZ8p6RRJaw62b0knR4/N4ZIG\nRY9ZUQbr+IGkb8dZN5N1HCfplOjroyS9G91fVo9JJ3Vk9ZhIMklfiL4ulvSqpNNy9DuSqJas/55E\n3/8WSY9KejpX/2748YI/8xonaZ27r3f3fZIWSrowxzVdKOln0dc/k3RRJnbi7n+UtD3JfV8oaaG7\n73X39yWtU+TYZaqORDJZx0fu/nr09U5Jb0sqU5aPSSd1JJKpOtzdd0Uni6M/rtz8jiSqJZGM1WJm\n5ZKmS3qo3f6yekxAs2GZpA9jpjeq8y+KdHNJz5nZa2Z2Q3TeMe7+UfT1x5KOyWI9ifadi+P0z2b2\nZrRZsbUZJit1mNkJkkYr8j/8nB2TdnVIWT4m0eaxVZI2S/q9u+fseCSoRcr+78k9km6VtD9mXj79\nuykYhR5euXaGu1dKmibpRjM7M3ahR9oecnIvQy73Lek/FWnKrZT0kaSfZGvHZvYFSU9Kutnd/xa7\nLJvHJE4dWT8m7t4S/f0slzTOzEa0W56145GglqweEzObIWmzu7/WSZ25/HdTUAo9vOolHR8zXR6d\nlxXuXh/9c7OkxYo0KXxiZsdJUvTPzdmqp5N9Z/U4ufsn0S+r/ZIe1OdNLRmtw8yKFQmMX7n7oujs\nrB+TeHXk6phE990g6QVJU5Xj35HYWnJwTCZI+gcz26DIJYZzzOyXypN/N4Wm0MNrhaTBZjbIzEok\nXSFpaTZ2bGY9zeyo1teSJktaE93/NdHVrpH022zUE5Vo30slXWFmh5vZIEmDJf05U0W0fhFEXazI\nccloHWZmkv6fpLfd/d9iFmX1mCSqI9vHxMz6m1mv6OtSSZMkvaMc/I4kqiXbx8Td57p7ubufoMh3\nxXJ3v1p58u+m4OS6x0iufySdr0iPrvckfSeL+z1RkZ5Ib0ha27pvSX0lPS/pL5Kek9QnQ/t/TJGm\nliZF2uKv72zfkr4TPUZ1kqZluI5fSFot6U1FvgCOy0IdZyjS3POmpFXRn/OzfUw6qSOrx0TSKEkr\no/tbI+l7B/v9zODfTaJasv57EvP+Z+vz3oZZPyb8OMNDAQDCU+jNhgCAABFeAIDgEF4AgOAQXgCA\n4BBeAIDgEF7otszMzewnMdPfNrMfpPH9v2ZmayzyZICVZvbt6PxHoqOdHx6d7he9sVVmtt7MhrR7\nn3vM7LZ01QUUAsIL3dleSZeYWb90v7GZTZN0s6TJ7j5SkVHOP41ZpUXS1+NsulCRG1xb3+cwSZdG\n5wNIEuGF7qxZ0gOSvtV+QfTs6NKY6V3RP882sz+Y2W+jZ0nzzGxW9HlSq83sy9FN5iryOI5NkuSR\nkcMfjNnFPZK+ZWY92u36MUmXx0yfKemv7v7XQ/2wQCEhvNDd3S9plpn9XQrbVEiaLWmYpH+U9BV3\nH6fIYzD+ObrOCEkJB2iV9IGkl6Lbt3H31ZL2m1lFdNYVigQagBQQXujWPDIi+88lfTOFzVZ45Lla\nexUZ2ufZ6PzVkk5I4X1+JKlKHf+dPabImHc9FHn20+MpvCcAEV4oDPcoMmZiz5h5zYr+/kevO5XE\nLNsb83p/zPR+Sa3NgGsljelsp+7+F0XGJrys3aKF0XnnSXrT3T9J9oMAiCC80O25+3ZJv1EkwFpt\n0Ofh8w+KPJ03FT+SVG1mx0qSmZWY2TfirHenpG+3q+c9SVslzRNNhkCXEF4oFD+RFNvr8EFJZ5nZ\nG5JOl7Q7lTdz92WS7lPkSdhrJb0u6eg467Uua+8xSUMlLYqzDMBBMKo8ACA4nHkBAIJDeAEAgkN4\nAQCCQ3gBAIJDeAEAgkN4AQCCQ3gBAILz/wHWB3OXAIVFRAAAAABJRU5ErkJggg==\n",
      "text/plain": [
       "<matplotlib.figure.Figure at 0x2aaaf1a6d610>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "df = pd.read_table('/data/sudregp/cnv/penncnv/results/all_simplex.qcsum')\n",
    "# we ran for all samples, but let's look at only stats for samples in our simplex study\n",
    "keep_me = [i for i in range(df.shape[0]) if df.File[i].split('/')[-1] in samples]\n",
    "df = df.iloc[keep_me]\n",
    "sns.jointplot(x='NumCNV', y='LRR_SD', data=df)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "So, let's look at the cases that seem to be joint calling outliers. Where do they fall in this plot?"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>File</th>\n",
       "      <th>LRR_mean</th>\n",
       "      <th>LRR_median</th>\n",
       "      <th>LRR_SD</th>\n",
       "      <th>BAF_mean</th>\n",
       "      <th>BAF_median</th>\n",
       "      <th>BAF_SD</th>\n",
       "      <th>BAF_drift</th>\n",
       "      <th>WF</th>\n",
       "      <th>NumCNV</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>InfiniumExome/CLIA_400149</td>\n",
       "      <td>-0.0550</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.3398</td>\n",
       "      <td>0.5041</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.0555</td>\n",
       "      <td>0.000391</td>\n",
       "      <td>0.0940</td>\n",
       "      <td>404</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>InfiniumExome/CLIA_400172</td>\n",
       "      <td>0.0251</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.1802</td>\n",
       "      <td>0.4999</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.0361</td>\n",
       "      <td>0.000078</td>\n",
       "      <td>0.0495</td>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>121</th>\n",
       "      <td>InfiniumExome/CLIA_400203</td>\n",
       "      <td>-0.0510</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.2749</td>\n",
       "      <td>0.5001</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.0531</td>\n",
       "      <td>0.000348</td>\n",
       "      <td>-0.0916</td>\n",
       "      <td>405</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                          File  LRR_mean  LRR_median  LRR_SD  BAF_mean  \\\n",
       "1    InfiniumExome/CLIA_400149   -0.0550         0.0  0.3398    0.5041   \n",
       "8    InfiniumExome/CLIA_400172    0.0251         0.0  0.1802    0.4999   \n",
       "121  InfiniumExome/CLIA_400203   -0.0510         0.0  0.2749    0.5001   \n",
       "\n",
       "     BAF_median  BAF_SD  BAF_drift      WF  NumCNV  \n",
       "1           0.5  0.0555   0.000391  0.0940     404  \n",
       "8           0.5  0.0361   0.000078  0.0495     126  \n",
       "121         0.5  0.0531   0.000348 -0.0916     405  "
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df[df.NumCNV > 100]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Yep... 400203 is our biggest offender. The other two are also kids and also quite high... OK, so let's use these raw CNV calls to clean up the joint CNV data as well:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<seaborn.axisgrid.JointGrid at 0x2aaad00ee990>"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAa8AAAGoCAYAAADxbmq5AAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAALEgAACxIB0t1+/AAAIABJREFUeJzt3XuUVOWZ7/HfY9MENCKiKNJAhJEgcpcWNJjoJGHAS4SA\nE7mYiOaMy5lkjPGI4mhiMnFGE8wsx+gKRxNv8YIRGeQ4TIg53mLGIG1AAZWIykA3GEDEoBJomuf8\nUdVtUV3dXV1Vu3a9Vd/PWr2o2peqp6qa+vX77ne/29xdAACE5JC4CwAAoLMILwBAcAgvAEBwCC8A\nQHAILwBAcAgvAEBwCC8AQHAILwBAcAgvAEBwusRdQIExXQiA0FncBYSAlhcAIDjl1vIqSw+t2FTQ\nx5s1fkBBHw8Aio2WFwAgOIQXACA4hBcAIDiEFwAgOIQXACA4hBcAIDiEFwAgOIQXACA4hBcAIDiE\nFwAgOIQXACA4hBcAIDiEFwAgOIQXACA4hBcAIDiEFwAgOIQXACA4hBcAIDiEFwAgOIQXACA4hBcA\nIDiEFwAgOIQXACA4hBcAIDiEFwAgOF3iLgDF99CKTQV9vFnjBxT08QCgI7S8AADBIbwAAMEhvAAA\nwSG8AADBIbwAAMEhvAAAwWGofAQKPRQdAHAwWl4AgOAQXgCA4BBeAIDgEF4AgOAQXgCA4Ji7x11D\nIeX8YhghWDqY6BcVzuIuIAS0vAAAweE8L5QcLtkCoCOEF8peIcOQIARKA92GAIDgEF4AgOAQXgCA\n4BBeAIDgEF4AgOAw2hCIUamfHF/o0ZWlfBpEKdeG1mh5AQCCQ8sLQJtKvWWIykXLCwAQnLKamNfM\nfiXp6Bx3P1rSjgKWk69Sq0cqvZpKrR6p9GoqtXqk0qup1OrZ4e6T4y6i1JVVeOXDzOrcvTbuOpqV\nWj1S6dVUavVIpVdTqdUjlV5NpVYPskO3IQAgOIQXACA4hNfH7oy7gDSlVo9UejWVWj1S6dVUavVI\npVdTqdWDLHDMCwAQHFpeAIDgEF4AgOAQXgCA4BBeAIDglFV4TZ482SXxww8//IT8k7Uy/M7LWlmF\n144dpTTDCwBEq5K/88oqvAAAlYHwAgAEh/ACAASHi1Gi5DU2Nqq+vl5/+ctf4i4FKJhu3bqpX79+\nqq6ujruUIBFeKHn19fU6/PDDdfzxx8vM4i4HyJu7691331V9fb0GDhwYdzlBotsQJe8vf/mLjjrq\nKIILZcPMdNRRR9GbkAfCC0EguFBu+J3OD+EFAAgO4QXgIL/61a80ZMgQnXDCCbr55pszbvPMM8/o\niCOO0OjRozV69Gj98z//c8u6Sy65RMccc4yGDx9erJI7VV8qd9d1112nT3/60xo6dKhuu+22Tj3P\nzp07NXHiRA0ePFgTJ07Ue++9J0l68MEHW5579OjROuSQQ7R69eq8XxdSuHvZ/IwdO9ZRfl599dW4\nS4hEY2NjJI+7f//+vPYdNGiQv/nmm753714fOXKkr1u3rtV2Tz/9tJ9zzjkZH+PZZ5/1l156yYcN\nG5ZzHflqr75Ud999t3/1q1/1pqYmd3f/05/+1KnnmTt3rt90003u7n7TTTf51Vdf3WqbV155xQcN\nGpRx/zZ+tyv5Oy/r107LC+jAxo0bdeKJJ2r27NkaOnSozj//fH300UeSpJdeeklnnHGGxo4dq0mT\nJmnr1q2SpLvuukunnHKKRo0apenTp7dsP2fOHF122WUaP368rr76aj377LMtf52PGTNGu3fvlrtr\n7ty5Gj58uEaMGKFHHnlEUqI1ceaZZ+r8889vqceTF5M9/vjjdc011+jkk0/Wo48+mvNrffHFF3XC\nCSdo0KBB6tq1q2bMmKHHH3+8U4/xuc99Tr169Wq1fMGCBVqwYEGr5ffee6+mTJmiM888U4MHD9b3\nv//9nOvvrJ/+9Kf67ne/q0MOSXwVHnPMMZKkDz/8UJdcconGjRunMWPGtPkePP7447roooskSRdd\ndJGWLFnSapuHH35YM2bMiOgVVLDOJF2p/5ThXyHw+Fteb7/9tkvy559/3t3dL774Yp8/f77v27fP\nTzvtNN+2bZu7uy9cuNAvvvhid3ffsWNHy/7XXXed33bbbe7uftFFF/k555zT0jo699xzWx539+7d\n3tjY6IsWLfIvfvGLvn//fn/nnXe8f//+vmXLFn/66ae9R48evnnzZm9qavJTTz3Vf/vb37q7+6c+\n9Sn/4Q9/mLH+Bx54wEeNGtXqZ/r06a22ffTRR/3rX/96y/3777/fv/GNb7Ta7umnn/YjjzzSR4wY\n4ZMnT/a1a9e2es+ybXndc8893qdPH9+xY4d/9NFHPmzYMF+5cmWr7b7yla9kfB333Xdfp+tr1qtX\nL7/xxht97NixPnnyZP/jH//o7u7XXnut/+IXv3B39/fee88HDx7sH3zwQav9jzjiiJbbBw4cOOh+\ns0GDBvmaNWsyPj8tr1ayfu2c5wVkoX///powYYIk6cILL9Rtt92myZMna+3atZo4caIkqampSccd\nd5wkae3atbr++uu1a9cuffDBB5o0aVLLY/3t3/6tqqqqJEkTJkzQlVdeqdmzZ2vatGnq16+fnn/+\nec2cOVNVVVU69thjdcYZZ2jlypXq0aOHxo0bp379+kmSRo8erY0bN+r000+XJF1wwQUZa589e7Zm\nz55d0Pfj5JNP1qZNm/TJT35Sy5Yt09SpU/XGG2/k/HgTJ07UUUcdJUmaNm2ann/+edXW1h60TXML\ntJD17d27V926dVNdXZ0WL16sSy65RL/97W/161//WkuXLtUtt9wiKXG6xqZNmzR06NA2n9PMWo0g\nXLFihQ499NBYj/+VK8ILyEL6l5KZyd01bNgwvfDCC622nzNnjpYsWaJRo0bp3nvv1TPPPNOy7rDD\nDmu5PW/ePJ1zzjlatmyZJkyYoOXLl7dbxyc+8YmW21VVVdq/f3/Gx0314IMPav78+a2Wn3DCCVq0\naNFBy2pqarR58+aW+/X19aqpqWm1b48ePVpun3322fqHf/gH7dixQ0cffXS79bcl0/ub7oILLtD6\n9etbLb/yyiv1ta99Laf6+vXrp2nTpkmSvvzlL+viiy+WlOiReuyxxzRkyJCDtr/44ou1atUq9e3b\nV8uWLdOxxx6rrVu36rjjjtPWrVtbuh2bLVy4UDNnzuzo5SMHHPMCsrBp06aWkHrooYd0+umna8iQ\nIdq+fXvL8sbGRq1bt06StHv3bh133HFqbGzUgw8+2ObjvvnmmxoxYoSuueYanXLKKXr99df12c9+\nVo888oiampq0fft2Pffccxo3blzOtc+ePVurV69u9ZMeXJJ0yimn6I033tDbb7+tffv2aeHChTrv\nvPNabffOO++0HG978cUXdeDAgZaWU1tuv/123X777RnXPfnkk9q5c6f27NmjJUuWtLRyUz3yyCMZ\nX0d6cHWmvqlTp+rpp5+WJD377LP69Kc/LUmaNGmSfvKTn7Q8xqpVqyRJ99xzj1avXq1ly5ZJks47\n7zzdd999kqT77rtPU6ZMaXnsAwcO6Je//CXHuyJCeAFZGDJkiO644w4NHTpU7733nv7+7/9eXbt2\n1aJFi3TNNddo1KhRGj16tP77v/9bkvSDH/xA48eP14QJE3TiiSe2+bi33nqrhg8frpEjR6q6ulpn\nnXWWvvzlL2vkyJEaNWqUPv/5z+tHP/qR+vTpU5TX2aVLF91+++2aNGmShg4dqq985SsaNmyYpIMH\nXCxatEjDhw/XqFGjdPnll2vhwoUtraWZM2fqtNNO0/r169WvXz/9/Oc/lyS9/vrrbQbcuHHjNH36\ndI0cOVLTp09v1WXYWe3Vd/bZZ2vLli2SEi3fxx57TCNGjNC1116rn/3sZ5Kk73znO2psbNTIkSM1\nbNgwfec738n4PPPmzdOTTz6pwYMH6ze/+Y3mzZvXsu65555T//79NWjQoLxeCzKz5r8sykFtba3X\n1dXFXQYK7LXXXmv3WEPUNm7cqHPPPVdr166NrYZycO6552rx4sXq2rXrQcvvvfde1dXVtdkqK2dt\n/G5nPfVGGX7nZf3aOeYFoCieeOKJuEtAGSG8gA4cf/zxtLoiNGfOHM2ZMyfuMhAYjnkhCOXUvQ1I\n/E7ni/BCyevWrZveffdd/rOjbLgnrufVrVu3uEsJFt2GKHn9+vVTfX29tm/fHncpQME0X0kZuSG8\nUPKqq6u52iyAg9BtCAAIDuEFAAgO4QUACA7hBQAIDuEFAAgO4QUACA7hBQAIDuEFAAgO4QUACA7h\nBQAIDuEFAAgOcxsCKIolqxo0f/l6bdm1R317dtfcSUM0dUxN3GUhUIQXgMgtWdWgaxev0Z7GJklS\nw649unbxGkkiwJATug0BRG7+8vUtwdVsT2OT5i9fH1NFCB0tLwCR27JrT6eWIzs7P9ynh1ZsOmjZ\nrPEDYqqmuGh5AYhc357dO7Uc6Ejk4WVmk81svZltMLN5GdafaGYvmNleM7sqbd23zWydma01s4fN\njGtmAwGaO2mIuldXHbSse3WV5k4aElNFCF2k4WVmVZLukHSWpJMkzTSzk9I22ynpckm3pO1bk1xe\n6+7DJVVJmhFlvQCiMXVMjW6aNkI1PbvLJNX07K6bpo1gsAZyFvUxr3GSNrj7W5JkZgslTZH0avMG\n7r5N0jYzO6eN+rqbWaOkQyVtibheABGZOqaGsELBRN1tWCNpc8r9+uSyDrl7gxKtsU2Stkp6391/\nnb6dmV1qZnVmVrd9+/YClAwApSv1O2/3rp1xlxObkh2wYWZHKtFKGyipr6TDzOzC9O3c/U53r3X3\n2t69exe7TAAoqtTvvMN79oq7nNhEHV4Nkvqn3O+XXJaNL0p62923u3ujpMWSPlPg+gAAAYo6vFZK\nGmxmA82sqxIDLpZmue8mSaea2aFmZpK+IOm1iOoEAAQk0gEb7r7fzL4pabkSowXvdvd1ZnZZcv0C\nM+sjqU5SD0kHzOwKSSe5+wozWyTpD5L2S1ol6c4o6wUAhCHyGTbcfZmkZWnLFqTcfkeJ7sRM+94g\n6YZICwQABKdkB2wAANAWwgsAEBzCCwAQHMILABAcwgsAEBzCCwAQHC5GCQAFtGRVg+YvX68tu/ao\nb8/umjtpCBMSR4DwAoACWbKqQdcuXqM9jU2SpIZde3Tt4jWSRIAVGN2GAFAg85evbwmuZnsamzR/\n+fqYKipfhBcAFMiWXXs6tRy5o9sQAAqkb8/uasgQVH17di9aDQ+t2HTQ/VnjBxTtuYuJlhcAFMjc\nSUPUvbrqoGXdq6s0d9KQmCoqX7S8AKBAmgdlMNoweoQXABTQ1DE1hFUR0G0IAAgO4QUACA7hBQAI\nDse8gCSm9QHCQXgBYlofIDR0GwJiWh8gNIQXIKb1AUJDeAFqe/qeYk7rAyB7hBcgpvUBQsOADUBM\n6wOEhvAKCEO5oxXHtD5xfKb8HqEcEF6BYCh3+YnjM+X3COWCY16BYCh3+YnjM+X3COWC8AoEQ7nL\nTxyfKb9HKBeEVyAYyl1+4vhM+T1CuSC8AsFQ7vITx2fK7xHKBQM2AsFQ7ugVexReHJ8pv0coF+bu\ncddQMLW1tV5XVxd3GQhQ+ig8KdEiuWnaCL7YUWyW7YaDho70G+99ot1tZo0fkHdBRZT1a6flBaj9\nUXiEF0L20IpNLbcDC7J2ccwLEKPwgNAQXoAYhQeEhvACxCg8IDQc8wLEKDwgNIQXkBTHxLwAckO3\nIQAgOIQXACA4hBcAIDiEFwAgOJGHl5lNNrP1ZrbBzOZlWH+imb1gZnvN7Kq0dT3NbJGZvW5mr5nZ\naVHXCwAofZGONjSzKkl3SJooqV7SSjNb6u6vpmy2U9LlkqZmeIh/l/Qrdz/fzLpKOjTKepEZl40H\nUGqiHio/TtIGd39LksxsoaQpklrCy923SdpmZuek7mhmR0j6nKQ5ye32SdoXcb1lK9cA4rLxAEpR\n1N2GNZI2p9yvTy7LxkBJ2yXdY2arzOxnZnZY+kZmdqmZ1ZlZ3fbt2/OvuAw1B1DDrj1yfRxAS1Y1\ndLgvl40HSkvqd97uXTvjLic2pTxgo4ukkyX91N3HSPpQUqtjZu5+p7vXuntt7969i11jEPIJoEqa\nsHbJqgZNuPkpDZz3n5pw81NZhXuIKuV1lqvU77zDe/aKu5zYRB1eDZL6p9zvl1yWjXpJ9e6+Inl/\nkRJhhk7KJ4AqZcLafFqnIamU14nyF3V4rZQ02MwGJgdczJC0NJsd3f0dSZvNrHlm1C8o5VgZspdP\nAFXKhLWV0j1aKa8T5S/S8HL3/ZK+KWm5pNck/dLd15nZZWZ2mSSZWR8zq5d0paTrzazezHokH+If\nJT1oZq9IGi3pX6Ost1zlE0BTx9TopmkjVNOzu0xSTc/uZXl14UrpHq2U14nyF/nEvO6+TNKytGUL\nUm6/o0R3YqZ9V0uqjbTACpDvjOmVMGFt357d1ZDhC7zcukcr5XWi/DGrfIWohADKx9xJQw46JUAq\nz+7RSnmdKH+EF6DKuZ5XpbxOlD9z97hrKJja2lqvq6uLuwwAyIdlu+GgoSP9xnufKOiTzxo/oKCP\n10lZv/ZSPs8LAICMCC8AQHA45oWSxGTAANpDeKHkMBkwgI4QXuhQsVtB7c0CQXgBkAivnFVKt1Yc\nrSBmgQDQEQZs5KCSJjeNYy68SpkMGEDuCK8cVNLkpnG0giplMmCUJy45Uxx0G+agkrq14pgLj1kg\nECoGGxUP4ZWDSprcNK658JiLESFisFHx0G2Yg0rq1po6pkbTx9aoyhKztlSZafpYggXIpJJ6ZeJG\neOWgUq5xJSW6QR57qUFNyTkwm9z12EsN9OMDGTDYqHgIL7SrkganAPmqpF6ZuHHMKweVdFA2n26Q\n0M6FC61elB4GGxUP4ZWDSjoom+vglNACPrR6UboYbFQcdBvmoJIOyubaDRJad2No9QKVjvDKQSUd\nlM11cEpoAR9avUClo9swB3Gd+xSXXLpBQjsXLrR6JY7RobLR8spBJQ2Vz1Voo65Cq7eS5tcEMqHl\nlSMOyrYvtFFXodVbSYOGgEwIrwoRRxdTaAEfUr0co0NUHlqxKed9Z40fUMBK2kd4VYB8h4Ffv2SN\nHl6xWU3uqjLTzPH9dePUEZHWjPaFeIwOKCSOeVWAfIaBX79kjR74/aaDpod64PebdP2SNZHUiuyE\ndowOKDTCqwLk08X08IrNnVqO4mDQECod3YYByfW4VT5dTM0trmyXo3hCOkYHFBotr0DkMzR67qQh\nqq6yg5ZVV1lWXUzNl0LJdjkAFAPhFYNcLhOe9/RF6Q2lLBtOM8f379RyACgGwqvIcm1B5XPcav7y\n9Wo8cHBaNR7wrILvxqkjdOGpAw66GOWFpw5gtCGAWHHMq8hyPbk0n+NW+Z4TdOPUEYQVgJJCy6vI\ncg2SfIZGV9JEwrl0yQIID+FVZLkGST5Do+dOGqLqQ9IGbByS3YCNkDDfH1A56DYssnxmpM9raHT6\n4MAyHCzIfH9A5SC8iiyOCWDnL1+vxqa0ARtNHvmXerHnU2S+P6ByEF4xKPbJpXF8qec7n2Iujuhe\nrV17GjMuB1BeOOZVAeIYsJH3eWk5aOu8ac6nBsoP4VUB4pjENY7W3q6PWre62lsOIFxZdRua2VGS\nZkk6MbnoNUkPu/u7URWGwonjOFvPQ6v1XobQ6HlodF14XCYEqBwdhpeZDZX0lKTlklYpMU7tFEn/\nZGafd/fXoy0RhVDs42xtzdsb5Xy++YzkBBCWbFpeP5D0LXf/ZepCM5su6V8kTY+isGLJdURcPiPp\n4njOYns/w8CJ9pYXQhwtTADxyCa8Rrj7+ekL3f0xM/vXCGoqmlxHxOUzki6O54xDXF14XCYEqAzZ\nDNj4MMd1kiQzm2xm681sg5nNy7D+RDN7wcz2mtlVGdZXmdkqM3sii1o7JdcRcfmMpIvjOePAlX4B\nRCmbltcxZnZlhuUmqXd7O5pZlaQ7JE2UVC9ppZktdfdXUzbbKelySVPbeJhvKTFApEcWtXZKriPi\n8hlJF8dzxoEuPKDyPLRiU077zRo/oNP7ZNPyukvS4Rl+PinpZx3sO07SBnd/y933SVooaUrqBu6+\nzd1XSmp1MMTM+kk6J4vnyUmu5z/lc95UHM8JAOWmw5aXu38/j8evkbQ55X69pPGd2P9WSVcrEZYZ\nmdmlki6VpAEDOpfeuY5Oy2dUWxzPGYfQjtEhOyENGipXqd95R/ep3Pe+w5aXmf2dmQ1O3jYzu9vM\n3jezV8xsTFSFmdm5kra5+0vtbefud7p7rbvX9u7dbi9mK7nO1J7PDO9xPGccQjtGh44xa39pSP3O\nO7xnr7jLiU02x7y+Jene5O2ZkkZJGiRpjKTbJH22nX0bJKVeL75fclk2Jkg6z8zOltRNUg8ze8Dd\nL8xy/6zEMTqtEkbEhXaMDh1j1n6UkmyOee139+bjUedKut/d33X330g6rIN9V0oabGYDzayrpBmS\nlmZTmLtf6+793P345H5PFTq4chXHX6Ch/dXLMbrywx8kKCXZhNcBMzvOzLpJ+oKk36Ssa/ebyN33\nS/qmErNzvCbpl+6+zswuM7PLJMnM+phZvaQrJV1vZvVmVvCRhYUUR5dYaN1wDJUvP/xBglKSTbfh\ndyXVSaqStNTd10mSmZ0h6a2Odnb3ZZKWpS1bkHL7HSW6E9t7jGckPZNFrUWR71+guRz0jvOv3lzq\nZah8+cl30BCDPVBI2Yw2fMLMPiXpcHd/L2VVnaQLmu+Y2UR3fzKCGktOPrNH5DoKL9+JbvOZkirX\nUYOVcGyvkuTzBwmjT1FoWV0Sxd33pwWX3P1Dd/8gZdEPC1pZCcunSyzX7r98JrrN53hZaN2ViNbU\nMTX63bzP6+2bz9Hv5n0+6+Dh9wiFVsgrKVfMJf/y+Qs01+6/TFcIbm95qnxGieXTXUk3EZox2AOF\nVsjwivBiF6Un1y6xXLscq8zUlKGZVZXFZYLz+eLItd58u4niCD7CNjpcaw2FxpWUiyzXLsdMwdXe\n8lT5jBLLtd58uok4FaH8MPoUhVbI8NpYwMcqW7nOlFHTRtC0tTxVPl8cU8fUaPrYmpYWXpWZpo/t\nuNWZT2uPUxHKT2gzxKD05dxtaGYTJV3t7hMlyd2nFayqMpdLl2M+w5SnjqlR3f/s1MMrNqvJPesA\nkhItkkde3NzSwmty1yMvblbtp3q1u38+3URxHB/hmEz0GH2KQspmbsPPm9kfzewDM3vAzEaYWZ2k\nmyX9NPoSy8+SVQ2acPNTGjjvPzXh5qey6prK5y/XJasa9NhLDQcF0GMvNWT1vN9buk6NBw7ummw8\n4Pre0nXt7pdPay+Ok2E5ARcISzYtrx8rMYPxC5LOSv47z91vj7KwcrVkVYPmPvpySyA07NqjuY++\nLCm686byGW2Y6yjHfEZkxjGDfmiz9gOVLpvw8uQMF5K0xMwaCK7ctdeS6eiLPdfRcHF1ieUatnHM\nzsGMIEBYsgmvnmaWejyrS+p9d19c+LLKV64tmXyGnudz/OnINmb2ODLLmT1yxWz/ANqTzWjDZyV9\nKeXnuZTb50ZXGlLlMxoun+NPN3xpmKqrDj6XrLrKdMOXhmVRde5yOS4IoHJkM7fhxcUoBO3L1HJq\nb3mqfLrE4uhOYx48AB3Jaqi8mVVJOtLddyTvd5U0R9K33X1odOWVnxC74YrdncZFDwF0pMPwMrMZ\nkv6PpA/N7A1J/yLpbiUuNDk72vLKzw1fGqa5i15WY9PHgzaK0Q0XEs65AsrPrPEDCvp42bS8rpc0\n1t03mNnJSgyVP9/d/29BK6kQuXbD5TO3Yb6KPecf8+AB6Eg24bXP3TdIkrv/wczeILjyk0s33Mzx\n/fXA7zdlXB6lOI4/cc4VgI5kE17HmNmVKfd7pt53938rfFlId+PUEZJ00BRPM8f3b1nekdl3vaDf\nvbmz5f6Ev+qlB//utA73i+P4E+dcAehINuF1l6TD27hfUZdBiduNU0dkHVap0oNLkn735k7NvuuF\nDgMstBOcAVSGbIbKf7+tdWZ2RWHLQXtyPfaUHlwdLU/F8ScApSjfS6Jc2fEmKIS4rjc1d9IQVR+S\ndpLyIcbxJwCxyje8oh/qBkkxX28q/VPmUwcQs3zDi2NeRZLPsadPdMn8Mbe1PNX85esPOidNkhqb\nnIs0AohVNtfz2m1mf87ws1tS3yLUCEmHdq3q1PJUP5w+Umk9fzrEEss7wgnDAEpRNgM2Du9oG0Tv\no31NnVqeKp+h5wzYAFCKsprbEPFrq382237bXIeec8IwgFJEeAXCTMowO5Sinh2KE4YBlCLCKxDd\nuxyijxoPZFweNU4YBlBqCK9A7MkQXO0tL6RcT44u9oS+ACoH4RWII7pXa9ee1tcBO6J7tNcBy3Vi\nXi4oCSBK0fc5oSDaOrYV9TGvXE+OjvWkagBlj/AKxK4MV19ub3mh5HqeV6bh9e0tB4DOoNswEHGd\nb9Xz0Gq9lyEgex7afndlXBfP5DgbUBloeQXir0/s3anlhbK3MfNJ0G0tb5YpuNpbXghxTV4MoPgI\nrxgsWdWgCTc/pYHz/lMTbn4qqy/Xp1/f3qnlhZJpeH57y5vVtNEibGt5IXCcDagcdBsWWa6j8EKb\nYzCOmTlCe4+AuM0aPyDuEnJGy6vIcm0dtHWMqaNjT/nq2cZQ/LaWN5s6pkY3TRuhmp7dZUq0uG6a\nNiLS409tHf9jHkag/NDyKrJcR+G1dagowkNIkqTvnTdMcx99WY0HPn6i6kNM3ztvWIf7FntmDuZh\nBCoH4VVkpsyT6XY0Bu/9DCcot7e8UEKa2zCkWgHkh/Aqslxnh4/z0iQhzW0YUq0Acscxr0DMnTRE\n3asPvvAkXWIAKhUtryI7so2Tfo/sYOAFXWIA8LHIw8vMJkv6d0lVkn7m7jenrT9R0j2STpZ0nbvf\nklzeX9L9ko5VolftTnf/96jrjdoNXxqmuYteVmNTygCIKtMNXyq9ARAAUKoiDS8zq5J0h6SJkuol\nrTSzpe7+aspmOyVdLmlq2u77Jf1vd/+DmR0u6SUzezJt3+DQggKA/EXd8honaYO7vyVJZrZQ0hRJ\nLQHk7tuyFEUbAAANmklEQVQkbTOzc1J3dPetkrYmb+82s9ck1aTuGypaUACQn6jDq0bS5pT79ZLG\nd/ZBzOx4SWMkrciw7lJJl0rSgAHhni0eNSasBcpD6nfe0X0q9/9wyY82NLNPSnpM0hXu/uf09e5+\np7vXuntt797RTlIbKiasBcpH6nfe4T17xV1ObKIOrwZJ/VPu90suy4qZVSsRXA+6++IC11YxmLAW\nQLmJOrxWShpsZgPNrKukGZKWZrOjmZmkn0t6zd3/LcIayx4T1gIoN5Ee83L3/Wb2TUnLlRgqf7e7\nrzOzy5LrF5hZH0l1knpIOmBmV0g6SdJISV+VtMbMVicf8p/cfVmUNZejOGfnAIAoRH6eVzJslqUt\nW5By+x0luhPTPa+Op/xDFpiwFkC5YYaNCsC5ZQDKDeFVITi3DEA5Kfmh8gAApCO8AADBIbwAAMHh\nmBcAlJFZ4ytjmjxaXgCA4BBeAIDgEF4AgOBwzCsgXNYEABIIr0A0X9akeYqn5suaSCLAAFQcug0D\nwWVNAOBjhFcguKwJAHyM8ApEW5cv4bImACoR4RWIuZOGqHt11UHLuKwJgErFgI1AcFkTAPgY4ZWj\nOIatc1kTAEggvHLAsHUAiBfHvHLAsHUAiBfhlQOGrQNAvAivHDBsHQDiRXjlgGHrABAvBmzkgGHr\nABAvwitHDFsHgPjQbQgACA7hBQAIDt2GABCoXod11azxA+IuIxa0vAAAwSG8AADBIbwAAMEhvAAA\nwSG8AADBIbwAAMEhvAAAwSG8AADBIbwAAMEhvAAAwSG8AADBIbwAAMEhvAAAwSG8AADBIbwAAMGJ\nPLzMbLKZrTezDWY2L8P6E83sBTPba2ZXdWZfAEBlijS8zKxK0h2SzpJ0kqSZZnZS2mY7JV0u6ZYc\n9gUAVKCoW17jJG1w97fcfZ+khZKmpG7g7tvcfaWkxs7uCwCoTFGHV42kzSn365PLot4XAFDGgh+w\nYWaXmlmdmdVt37497nIAIFJ85yVEHV4Nkvqn3O+XXFawfd39Tnevdffa3r1751woAISA77yELhE/\n/kpJg81soBLBM0PSrCLsCwBlb+eH+/TQik0Z180aP6DI1RRXpOHl7vvN7JuSlkuqknS3u68zs8uS\n6xeYWR9JdZJ6SDpgZldIOsnd/5xp3yjrBQCEIeqWl9x9maRlacsWpNx+R4kuwaz2BQAg+AEbAIDK\nQ3gBAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkN4AQCCQ3gBAIIT+UnK5WrJqgbNX75eW3btUd+e3TV3\n0hBNHZPdpPf57AsAILxysmRVg65dvEZ7GpskSQ279ujaxWskqcMQymdfAEAC3YY5mL98fUv4NNvT\n2KT5y9dHui8AIIHwysGWXXs6tbxQ+wIAEgivHPTt2b1Tywu1LwAggfDKwdxJQ9S9uuqgZd2rqzR3\n0pBI9wUAJDBgIwfNAytyGTGYz74AgATCK0dTx9TkHDj57BsHhvYDKDWEF9rF0H4ApYhjXmgXQ/sB\nlCJaXmgXQ/uB0tXrsK6aNX5A3GXEgpYX2sXQfgCliPBCuxjaD6AU0W2IdjG0H0ApIrzQodCG9gMo\nf3QbAgCCQ3gBAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkN4AQCCQ3gBAIJD\neAEAgkN4AQCCQ3gBAIJDeAEAgkN4AQCCQ3gBAIJDeAEAgkN4AQCCQ3gBAIITeXiZ2WQzW29mG8xs\nXob1Zma3Jde/YmYnp6z7tpmtM7O1ZvawmXWLul4AQOmLNLzMrErSHZLOknSSpJlmdlLaZmdJGpz8\nuVTST5P71ki6XFKtuw+XVCVpRpT1AgDCEHXLa5ykDe7+lrvvk7RQ0pS0baZIut8Tfi+pp5kdl1zX\nRVJ3M+si6VBJWyKuFwAQgKjDq0bS5pT79cllHW7j7g2SbpG0SdJWSe+7+6/Tn8DMLjWzOjOr2759\ne0GLB4BSw3deQskO2DCzI5VolQ2U1FfSYWZ2Yfp27n6nu9e6e23v3r2LXSYAFBXfeQlRh1eDpP4p\n9/sll2WzzRclve3u2929UdJiSZ+JsFYAQCCiDq+Vkgab2UAz66rEgIuladsslfS15KjDU5XoHtyq\nRHfhqWZ2qJmZpC9Iei3iegEAAegS5YO7+34z+6ak5UqMFrzb3deZ2WXJ9QskLZN0tqQNkj6SdHFy\n3QozWyTpD5L2S1ol6c4o6wUAhCHS8JIkd1+mREClLluQctslfaONfW+QdEOkBQIAglOyAzYAAGgL\n4QUACA7hBQAIDuEFAAgO4QUACI4lBvuVBzPbLul/ctz9aEk7ClhOvkqtHqn0aiq1eqTSq6nU6pFK\nr6ZSq2eHu0/OZkMz+1W225absgqvfJhZnbvXxl1Hs1KrRyq9mkqtHqn0aiq1eqTSq6nU6kF26DYE\nAASH8AIABIfw+lipTT1VavVIpVdTqdUjlV5NpVaPVHo1lVo9yALHvAAAwaHlBQAIDuEFAAhORYWX\nmU02s/VmtsHM5mVYb2Z2W3L9K2Z2csT19Dezp83sVTNbZ2bfyrDNmWb2vpmtTv58N+KaNprZmuRz\n1WVYX+z3aEjKa19tZn82syvSton8PTKzu81sm5mtTVnWy8yeNLM3kv8e2ca+7f7eFbCe+Wb2evJz\n+Q8z69nGvu1+xgWu6Xtm1pDy2Zzdxr7Feo8eSallo5mtbmPfSN4jFJC7V8SPEtcTe1PSIEldJb0s\n6aS0bc6W9F+STNKpklZEXNNxkk5O3j5c0h8z1HSmpCeK+D5tlHR0O+uL+h5l+AzfkfSpYr9Hkj4n\n6WRJa1OW/UjSvOTteZJ+mMvvXQHr+RtJXZK3f5ipnmw+4wLX9D1JV2XxuRblPUpb/2NJ3y3me8RP\n4X4qqeU1TtIGd3/L3fdJWihpSto2UyTd7wm/l9TTzI6LqiB33+ruf0je3q3ElaJronq+Ainqe5Tm\nC5LedPdcZ1HJmbs/J2ln2uIpku5L3r5P0tQMu2bze1eQetz91+6+P3n395L65fs8+daUpaK9R82S\nV2f/iqSH830exKOSwqtG0uaU+/VqHRTZbBMJMzte0hhJKzKs/kyyK+i/zGxYxKW4pN+Y2UtmdmmG\n9bG9R5JmqO0vm2K+R82OdfetydvvSDo2wzZxvV+XKNFCzqSjz7jQ/jH52dzdRtdqHO/RZyX9yd3f\naGN9sd8jdFIlhVfJMrNPSnpM0hXu/ue01X+QNMDdR0r6iaQlEZdzuruPlnSWpG+Y2ecifr6smFlX\nSedJejTD6mK/R624uyvxhRc7M7tO0n5JD7axSTE/458q0R04WtJWJbrqSsFMtd/qKsn/B/hYJYVX\ng6T+Kff7JZd1dpuCMrNqJYLrQXdfnL7e3f/s7h8kby+TVG1mR0dVj7s3JP/dJuk/lOjSSVX09yjp\nLEl/cPc/pa8o9nuU4k/NXabJf7dl2Kao75eZzZF0rqTZyUBtJYvPuGDc/U/u3uTuByTd1cZzFfs9\n6iJpmqRH2tqmmO8RclNJ4bVS0mAzG5j8K36GpKVp2yyV9LXkiLpTJb2f0i1UcMl+959Les3d/62N\nbfokt5OZjVPiM3s3onoOM7PDm28rMQBgbdpmRX2PUrT5l3Ix36M0SyVdlLx9kaTHM2yTze9dQZjZ\nZElXSzrP3T9qY5tsPuNC1pR6PPTLbTxX0d6jpC9Ket3d6zOtLPZ7hBzFPWKkmD9KjJT7oxIjm65L\nLrtM0mXJ2ybpjuT6NZJqI67ndCW6ml6RtDr5c3ZaTd+UtE6JEVi/l/SZCOsZlHyel5PPGft7lHzO\nw5QIoyNSlhX1PVIiOLdKalTimMzXJR0l6f9JekPSbyT1Sm7bV9Ky9n7vIqpngxLHjpp/lxak19PW\nZxxhTb9I/p68okQgHRfne5Rcfm/z707KtkV5j/gp3A/TQwEAglNJ3YYAgDJBeAEAgkN4AQCCQ3gB\nAIJDeAEAgkN4oWyZmZvZj1PuX2Vm3yvg43/NzNYmZx9fZWZXJZffm5xJ/RPJ+0eb2cbk7bfMbEja\n49xqZtcUqi6gEhBeKGd7JU2LYrYNMztL0hWS/sbdRygxw/77KZs0KTG/YLqFSpyE2/w4h0g6P7kc\nQJYIL5Sz/ZLulPTt9BXJ1tH5Kfc/SP57ppk9a2aPJ1tJN5vZbDN7MdnC+qvkLtcqcamPLZLk7nvd\n/a6Up7hV0reTUxGleljSBSn3PyfpfzyGmfKBkBFeKHd3SJptZkd0Yp9RSszgMVTSVyV92t3HSfqZ\npH9MbjNc0kvtPMYmSc8n92/h7mskHTCzUclF7c2UD6ANhBfKmidm6b9f0uWd2G2lJ661tleJ6Yp+\nnVy+RtLxnXicmyTNVev/Zw9LmpFslU1V5pnyAbSD8EIluFWJefYOS1m2X8nf/+Rxp64p6/am3D6Q\ncv+ApOZuwHWSxrb3pJ64VtRqJS56mGphctkXJb3iGWbKB9A+wgtlz913SvqlEgHWbKM+Dp/zJFV3\n8mFvkjTfzPpIieuNmdn/yrDdv0i6Kq2eNyXtkHSz6DIEckJ4oVL8WFLqqMO7JJ1hZi9LOk3Sh515\nME9cN+x2Ja62u06JC2L2yLBd87p0D0s6UVKra7gB6BizygMAgkPLCwAQHMILABAcwgsAEBzCCwAQ\nHMILABAcwgsAEBzCCwAQnP8Pofk2AkQZM88AAAAASUVORK5CYII=\n",
      "text/plain": [
       "<matplotlib.figure.Figure at 0x2aaaf4f68590>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sns.jointplot(x='NumCNV', y='LRR_SD', data=df[df.NumCNV <= 20])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This seems like a potential cutoff. But we can also play with 10. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "df20 = df[df.NumCNV <= 20]\n",
    "df10 = df[df.NumCNV <= 10]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now, let's check that the remaining subjects have decent numbers:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "fid = open('/data/sudregp/cnv/penncnv/good_kids_numCNVse20.txt', 'w')\n",
    "for f in df20.File:\n",
    "    sample = f.split('/')[-1]\n",
    "    # control and affected only include kids!\n",
    "    if sample in controls + affected:\n",
    "        fid.write(sample + '\\n')\n",
    "fid.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CLIA_400140.jointcnv: 2 denovo, 196 inherited\n",
      "CLIA_400138.jointcnv: 16 denovo, 120 inherited\n",
      "CLIA_400121.jointcnv: 0 denovo, 110 inherited\n",
      "CLIA_400191.jointcnv: 0 denovo, 160 inherited\n",
      "CLIA_400148.jointcnv: 5 denovo, 117 inherited\n",
      "CLIA_400153.jointcnv: 7 denovo, 89 inherited\n",
      "CLIA_400216.jointcnv: 6 denovo, 79 inherited\n",
      "CLIA_400189.jointcnv: 0 denovo, 155 inherited\n",
      "CLIA_400180.jointcnv: 0 denovo, 105 inherited\n",
      "CLIA_400166.jointcnv: 11 denovo, 193 inherited\n",
      "CLIA_400122.jointcnv: 3 denovo, 110 inherited\n",
      "CLIA_400204.jointcnv: 0 denovo, 190 inherited\n",
      "CCGO_800979.jointcnv: 1 denovo, 28 inherited\n",
      "CLIA_400162.jointcnv: 95 denovo, 226 inherited\n",
      "CLIA_400158.jointcnv: 45 denovo, 116 inherited\n",
      "CLIA_400170.jointcnv: 0 denovo, 172 inherited\n",
      "CLIA_400123.jointcnv: 3 denovo, 53 inherited\n",
      "CLIA_400209.jointcnv: 3 denovo, 280 inherited\n",
      "CLIA_400134.jointcnv: 3 denovo, 148 inherited\n",
      "CLIA_400131.jointcnv: 4 denovo, 161 inherited\n",
      "CLIA_400195.jointcnv: 4 denovo, 344 inherited\n",
      "CLIA_400135.jointcnv: 8 denovo, 325 inherited\n",
      "CLIA_400142.jointcnv: 29 denovo, 284 inherited\n",
      "CLIA_400155.jointcnv: 57 denovo, 282 inherited\n",
      "CLIA_400152.jointcnv: 0 denovo, 152 inherited\n",
      "CLIA_400178.jointcnv: 5 denovo, 56 inherited\n",
      "CLIA_400139.jointcnv: 49 denovo, 269 inherited\n",
      "CLIA_400128.jointcnv: 1 denovo, 84 inherited\n",
      "CCGO_800980.jointcnv: 3 denovo, 43 inherited\n",
      "CLIA_400159.jointcnv: 2 denovo, 177 inherited\n",
      "CLIA_400133.jointcnv: 1 denovo, 188 inherited\n",
      "CLIA_400132.jointcnv: 77 denovo, 263 inherited\n",
      "CLIA_400175.jointcnv: 50 denovo, 144 inherited\n",
      "CLIA_400129.jointcnv: 1 denovo, 94 inherited\n",
      "CLIA_400144.jointcnv: 0 denovo, 143 inherited\n",
      "CLIA_400201.jointcnv: 0 denovo, 169 inherited\n",
      "CLIA_400200.jointcnv: 9 denovo, 306 inherited\n",
      "CLIA_400179.jointcnv: 30 denovo, 50 inherited\n",
      "CLIA_400193.jointcnv: 0 denovo, 179 inherited\n",
      "CLIA_400212.jointcnv: 1 denovo, 168 inherited\n",
      "CLIA_400210.jointcnv: 0 denovo, 350 inherited\n",
      "CLIA_400205.jointcnv: 1 denovo, 222 inherited\n",
      "CLIA_400130.jointcnv: 0 denovo, 215 inherited\n",
      "CLIA_400127.jointcnv: 10 denovo, 71 inherited\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd /data/sudregp/cnv/penncnv/results\n",
    "\n",
    "while read sample; do\n",
    "    triocnv=${sample}'.jointcnv'\n",
    "    rm denovo.txt inherited.txt 2>/dev/null\n",
    "    grep mother ${triocnv} > mom_snps;\n",
    "    grep father ${triocnv} > dad_snps;\n",
    "    cat mom_snps dad_snps > parent_snps;\n",
    "    for snp in `grep offspring ${triocnv} | cut -d' ' -f 1`; do\n",
    "        if ! grep -q $snp parent_snps; then\n",
    "            echo \"$snp\" >> denovo.txt\n",
    "        else\n",
    "            echo \"$snp\" >> inherited.txt\n",
    "        fi;\n",
    "   done\n",
    "   echo ${triocnv}: `cat denovo.txt 2>/dev/null | wc -l` denovo, `cat inherited.txt 2>/dev/null | wc -l` inherited;\n",
    "   rm *_snps;\n",
    "done < ../good_kids_numCNVse20.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This is a bit better, and it might go down even further if we restrict the CNVs by size, or play a bit with the tool parameters. It might also change if we restrict it to genes only, or brain only regions. So, let's check how to import these results into pseq."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import re\n",
    "\n",
    "\n",
    "regex = re.compile('^chr(\\d+):(\\d+)-(\\d+)\\s+numsnp=(\\d+)\\s+\\S+\\s+\\S+cn=(\\d+)\\s+\\S+/(\\S+)')\n",
    "fout = open('/data/sudregp/cnv/penncnv/joint_call_numCNVse20.cnv', 'w')\n",
    "\n",
    "for f in df20.File:\n",
    "    good_sample = f.split('/')[-1]\n",
    "    if good_sample in controls + affected:\n",
    "        famid = [t['famid'] for t in trios if t['child'] == good_sample][0]\n",
    "        fname = '/data/sudregp/cnv/penncnv/results/%s.jointcnv' % good_sample\n",
    "        fid = open(fname, 'r')\n",
    "        for line in fid:\n",
    "            match = regex.search(line)\n",
    "            chr, start, end, numsnp, cn, sample = match.groups()\n",
    "            fout.write('\\t'.join([famid, sample, chr, start, end, cn, numsnp,\n",
    "                                  '0']) + '\\n')\n",
    "fout.close()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now we import it into PLINK:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ joint_call_numCNVse20.log ]\n",
      "Analysis started: Fri Dec  8 14:17:04 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--cfile joint_call_numCNVse20\n",
      "\t--cnv-make-map\n",
      "\t--out joint_call_numCNVse20\n",
      "\t--noweb\n",
      "\t--1\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ joint_call_numCNVse20.cnv ]\n",
      "Writing new MAP file to [ joint_call_numCNVse20.cnv.map ]\n",
      "Wrote 9189 unique positions to file\n",
      "\n",
      "Analysis finished: Fri Dec  8 14:17:04 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ plink.log ]\n",
      "Analysis started: Fri Dec  8 14:17:04 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--cfile joint_call_numCNVse20\n",
      "\t--noweb\n",
      "\t--cnv-check-no-overlap\n",
      "\t--1\n",
      "\n",
      "Reading marker information from [ joint_call_numCNVse20.cnv.map ]\n",
      "9189 (of 9189) markers to be included from [ joint_call_numCNVse20.cnv.map ]\n",
      "Reading individual information from [ joint_call_numCNVse20.fam ]\n",
      "Reading pedigree information from [ joint_call_numCNVse20.fam ] \n",
      "99 individuals read from [ joint_call_numCNVse20.fam ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ joint_call_numCNVse20.cnv ]\n",
      "Within-individual CNV overlap detected, involving 1275 CNVs\n",
      "Writing list to [ plink.cnv.overlap ]\n",
      "17850 mapped to a person, of which 17850 passed filters\n",
      "17850 of 17850 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     0       3 / 22\n",
      "     1  1236 / 6106\n",
      "     3  1069 / 9392\n",
      "     4       2 / 20\n",
      "\n",
      "Writing per-individual summary to [ plink.cnv.indiv ]\n",
      "Writing positional summary to [ plink.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Dec  8 14:17:04 2017\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "module load plink/1.07\n",
    "\n",
    "cd /data/sudregp/cnv/penncnv/\n",
    "cp /data/NCR_SBRB/simplex/simplex.ped joint_call_numCNVse20.fam\n",
    "plink --cfile joint_call_numCNVse20 --cnv-make-map --out joint_call_numCNVse20 --noweb --1\n",
    "plink --cfile joint_call_numCNVse20 --noweb --cnv-check-no-overlap  --1"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "And we intersect with the regions that code for genes. Because of the arrays we're using, these should be many. But in any case..."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 96,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ joint_call_numCNVse20_gene_disrupt.log ]\n",
      "Analysis started: Fri Dec  8 14:21:32 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--cfile joint_call_numCNVse20\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-disrupt\n",
      "\t--cnv-intersect gene_locations.txt\n",
      "\t--cnv-write\n",
      "\t--out joint_call_numCNVse20_gene_disrupt\n",
      "\n",
      "Reading marker information from [ joint_call_numCNVse20.cnv.map ]\n",
      "9189 (of 9189) markers to be included from [ joint_call_numCNVse20.cnv.map ]\n",
      "Reading individual information from [ joint_call_numCNVse20.fam ]\n",
      "Reading pedigree information from [ joint_call_numCNVse20.fam ] \n",
      "99 individuals read from [ joint_call_numCNVse20.fam ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading CNV intersection list from [ gene_locations.txt ]\n",
      "Read 304125 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ joint_call_numCNVse20.cnv ]\n",
      "17850 mapped to a person, of which 17850 passed filters\n",
      "17269 intersected with one or more specified region\n",
      "17269 of 17850 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     0       2 / 18\n",
      "     1  1186 / 5881\n",
      "     3  1028 / 9141\n",
      "     4       0 / 13\n",
      "\n",
      "Writing new CNV list to [ joint_call_numCNVse20_gene_disrupt.cnv ]\n",
      "Writing new FAM file to [ joint_call_numCNVse20_gene_disrupt.fam ]\n",
      "\n",
      "Analysis finished: Fri Dec  8 14:24:41 2017\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "module load plink/1.07\n",
    "\n",
    "cd /data/sudregp/cnv/penncnv/\n",
    "cp /data/NCR_SBRB/simplex/xhmm/gene_locations.txt .\n",
    "\n",
    "plink --cfile joint_call_numCNVse20 --noweb --1 --cnv-disrupt \\\n",
    "    --cnv-intersect gene_locations.txt --cnv-write \\\n",
    "    --out joint_call_numCNVse20_gene_disrupt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "OK, the percentage of removed regions makes sense. Now, let's run pseq and make a few plots."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 105,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ joint_call_numCNVse20_gene_disrupt.log ]\n",
      "Analysis started: Fri Dec  8 15:28:10 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--cfile joint_call_numCNVse20_gene_disrupt\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--out joint_call_numCNVse20_gene_disrupt\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ joint_call_numCNVse20_gene_disrupt.cnv ]\n",
      "Writing new MAP file to [ joint_call_numCNVse20_gene_disrupt.cnv.map ]\n",
      "Wrote 8931 unique positions to file\n",
      "\n",
      "Analysis finished: Fri Dec  8 15:28:10 2017\n",
      "\n",
      "PLINK v1.90p 64-bit (4 Dec 2017)               www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2017 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to joint_call_numCNVse20_gene_disrupt.vcf.log.\n",
      "Options in effect:\n",
      "  --1\n",
      "  --cfile joint_call_numCNVse20_gene_disrupt\n",
      "  --out joint_call_numCNVse20_gene_disrupt.vcf\n",
      "  --recode vcf\n",
      "\n",
      "258199 MB RAM detected; reserving 129099 MB for main workspace.\n",
      ".cnv.map file successfully loaded.  (Subsequent functions have not been\n",
      "implemented yet.)\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "module load plink/1.07\n",
    "\n",
    "cd /data/sudregp/cnv/penncnv/\n",
    "plink --cfile joint_call_numCNVse20_gene_disrupt --cnv-make-map --noweb --1 \\\n",
    "    --out joint_call_numCNVse20_gene_disrupt\n",
    "    \n",
    "/data/NCR_SBRB/software/plink_1.9_dev_12082017/plink --cfile joint_call_numCNVse20_gene_disrupt --1 --recode vcf \\\n",
    "    --out joint_call_numCNVse20_gene_disrupt.vcf"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "So, unfortunately here I reached a dead-end, as the old PLINK doesn't export VCFs, and the new one doesn't have exporting CNVs implemented yet. So, I cannot export my calls from it. I also could not find a way to convert the calls directly from PennCNV to VCF, which would also work. That's the route I took with XHMM. So, we'll need to use the PennCNV to do the CNV calling, and then import them separately into PLINK to assess different markers (e.g. gene disruption, brain expression, denovo, etc). \n",
    "\n",
    "The file I created to import into PLINK should have all the joint call variants for all samples: joint_call_numCNVse20.cnv. So, I could just use that if all I'm looking for is total CNV burden. If I'm looking for denovo variants, I could do it manually (as before), or use PennCNV's tool. The nice thing about PennCNV's tool is that it assigns p-values to all de-novo CNVs. For example:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CLIA_400127.jointcnv: 10 denovo, 71 inherited\n",
      "chr6:170102887-170102970 numsnp=3 length=84 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm596102 endsnp=exm596109 offspring statepath=332\n",
      "chr8:144990587-144992782 numsnp=33 length=2,196 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm728992 endsnp=exm729175 offspring statepath=532\n",
      "chr9:140167307-140196334 numsnp=6 length=29,028 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm803033 endsnp=exm803109 offspring statepath=332-532\n",
      "chr13:53420234-53422337 numsnp=7 length=2,104 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm1070833 endsnp=exm1070902 offspring statepath=332\n",
      "chr16:68056661-68057101 numsnp=9 length=441 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm1865122 endsnp=exm1251113 offspring statepath=532\n",
      "chr19:5455990-5456142 numsnp=6 length=153 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm1411591 endsnp=exm1411604 offspring statepath=332\n",
      "chr19:44223113-44223833 numsnp=9 length=721 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm1476396 endsnp=exm1476441 offspring statepath=332\n",
      "chr19:45579592-45656147 numsnp=9 length=76,556 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm1479878 endsnp=exm1479929 offspring statepath=332\n",
      "chr9:140245516-140245893 numsnp=4 length=378 state5,cn=3 InfiniumExome/CLIA_400127 startsnp=exm803195 endsnp=exm803208 offspring statepath=535\n",
      "chr15:42363395-42363488 numsnp=3 length=94 state5,cn=3 InfiniumExome/CLIA_400127 startsnp=exm1153179 endsnp=exm1153188 offspring statepath=335\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "cd /data/sudregp/cnv/penncnv/results\n",
    "\n",
    "triocnv='CLIA_400127.jointcnv'\n",
    "rm denovo.txt inherited.txt 2>/dev/null\n",
    "grep mother ${triocnv} > mom_snps;\n",
    "grep father ${triocnv} > dad_snps;\n",
    "cat mom_snps dad_snps > parent_snps;\n",
    "grep offspring ${triocnv} > kid_snps;\n",
    "while read line; do\n",
    "    IFS=' ' read -ra arr <<< \"$line\"\n",
    "    snp=${arr[0]}\n",
    "    if ! grep -q $snp parent_snps; then\n",
    "        echo $line >> denovo.txt\n",
    "    else\n",
    "        echo $line >> inherited.txt\n",
    "    fi;\n",
    "done < kid_snps;\n",
    "echo ${triocnv}: `cat denovo.txt 2>/dev/null | wc -l` denovo, `cat inherited.txt 2>/dev/null | wc -l` inherited;\n",
    "rm *_snps;\n",
    "head denovo.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'child': 'CLIA_400127',\n",
       "  'famid': '10033',\n",
       "  'father': 'CLIA_400197',\n",
       "  'mother': 'CLIA_400167'}]"
      ]
     },
     "execution_count": 127,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "[t for t in trios if t['child'] == 'CLIA_400127']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 131,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,MT,0 were discarded)\n",
      "NOTICE: For the region chr6:170102887-170102970, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr6:170102887-170102970, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr6:170102887-170102970, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "module load penncnv\n",
    "\n",
    "cd /data/sudregp/cnv/penncnv\n",
    "box='InfiniumExome'\n",
    "infer_snp_allele.pl -pfb ${box}.pfb -hmm penncnv_example.hmm -denovocn 1 \\\n",
    "    ${box}/CLIA_400197 ${box}/CLIA_400167 ${box}/CLIA_400127 -out tempfile \\\n",
    "    --startsnp exm596102 --endsnp exm596109"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Or let's look at a bigger region:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 132,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,0,MT were discarded)\n",
      "NOTICE: For the region chr8:144990587-144992782, 33 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr8:144990587-144992782, 33 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr8:144990587-144992782, 33 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "module load penncnv\n",
    "\n",
    "cd /data/sudregp/cnv/penncnv\n",
    "box='InfiniumExome'\n",
    "infer_snp_allele.pl -pfb ${box}.pfb -hmm penncnv_example.hmm -denovocn 1 \\\n",
    "    ${box}/CLIA_400197 ${box}/CLIA_400167 ${box}/CLIA_400127 -out tempfile \\\n",
    "    --startsnp exm728992 --endsnp exm729175"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Still, very low p-value... good I guess. Let's try the other command, for inherited CNVs:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 134,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "chr1:877556-878744 numsnp=5 length=1,189 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm116 endsnp=exm158 offspring statepath=322\n",
      "chr1:1431109-1431163 numsnp=3 length=55 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm3724 endsnp=exm3727 offspring statepath=322\n",
      "chr1:11008295-11008532 numsnp=5 length=238 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm13773 endsnp=exm13789 offspring statepath=232\n",
      "chr1:108185282-108298086 numsnp=7 length=112,805 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm79173 endsnp=exm79215 offspring statepath=232\n",
      "chr1:114682286-114682445 numsnp=5 length=160 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm85950 endsnp=exm85959 offspring statepath=232\n",
      "chr1:205492387-205492679 numsnp=5 length=293 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm142359 endsnp=exm142369 offspring statepath=322\n",
      "chr1:236347143-236347189 numsnp=3 length=47 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm161587 endsnp=exm161591 offspring statepath=232\n",
      "chr2:74438406-74450058 numsnp=3 length=11,653 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm203762 endsnp=exm203779 offspring statepath=232\n",
      "chr2:242755734-242755880 numsnp=4 length=147 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm285162 endsnp=exm285174 offspring statepath=322\n",
      "chr3:100567662-100568897 numsnp=4 length=1,236 state2,cn=1 InfiniumExome/CLIA_400127 startsnp=exm334153 endsnp=exm334160 offspring statepath=232\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "cd /data/sudregp/cnv/penncnv/results\n",
    "head inherited.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 137,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,XY,MT were discarded)\n",
      "NOTICE: For the region chr11:64600723-64603925, 14 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr11:64600723-64603925, 14 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr11:64600723-64603925, 14 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "module load penncnv\n",
    "\n",
    "cd /data/sudregp/cnv/penncnv\n",
    "box='InfiniumExome'\n",
    "infer_snp_allele.pl -pfb ${box}.pfb -hmm penncnv_example.hmm -denovocn 3 \\\n",
    "    ${box}/CLIA_400197 ${box}/CLIA_400167 ${box}/CLIA_400127 -out tempfile \\\n",
    "    --startsnp exm924016 --endsnp exm924087"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Not sure what's going on here... these should be inherited, so why is there no evidence of parental origin for this CNV? That's actually true  when I look at tempfile... is the joint calling screwing something up? Let's see if we can find evidence for any of the 81 CNVs found in the offspring:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 140,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,0,XY were discarded)\n",
      "NOTICE: For the region chr1:877556-878744, 5 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr1:877556-878744, 5 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr1:877556-878744, 5 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,MT,0 were discarded)\n",
      "NOTICE: For the region chr1:1431109-1431163, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr1:1431109-1431163, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr1:1431109-1431163, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,0,MT were discarded)\n",
      "NOTICE: For the region chr1:11008295-11008532, 5 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr1:11008295-11008532, 5 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr1:11008295-11008532, 5 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,0,XY were discarded)\n",
      "NOTICE: For the region chr1:108185282-108298086, 7 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr1:108185282-108298086, 7 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr1:108185282-108298086, 7 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,0,XY were discarded)\n",
      "NOTICE: For the region chr1:114682286-114682445, 5 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr1:114682286-114682445, 5 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr1:114682286-114682445, 5 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,XY,0 were discarded)\n",
      "NOTICE: For the region chr1:205492387-205492679, 5 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr1:205492387-205492679, 5 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr1:205492387-205492679, 5 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,0,MT were discarded)\n",
      "NOTICE: For the region chr1:236347143-236347189, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr1:236347143-236347189, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr1:236347143-236347189, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,MT,0 were discarded)\n",
      "NOTICE: For the region chr2:74438406-74450058, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr2:74438406-74450058, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr2:74438406-74450058, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,0,XY were discarded)\n",
      "NOTICE: For the region chr2:242755734-242755880, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr2:242755734-242755880, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr2:242755734-242755880, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,0,MT were discarded)\n",
      "NOTICE: For the region chr3:100567662-100568897, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr3:100567662-100568897, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr3:100567662-100568897, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,MT,XY were discarded)\n",
      "NOTICE: For the region chr4:123747942-123747947, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr4:123747942-123747947, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr4:123747942-123747947, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,MT,0 were discarded)\n",
      "NOTICE: For the region chr5:1038004-1038109, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr5:1038004-1038109, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr5:1038004-1038109, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,XY,0 were discarded)\n",
      "NOTICE: For the region chr5:33989503-33989565, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr5:33989503-33989565, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr5:33989503-33989565, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,0,XY were discarded)\n",
      "NOTICE: For the region chr5:128364064-128365384, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr5:128364064-128365384, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr5:128364064-128365384, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,XY,0 were discarded)\n",
      "NOTICE: For the region chr6:75822972-75823017, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr6:75822972-75823017, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr6:75822972-75823017, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,MT,XY were discarded)\n",
      "NOTICE: For the region chr6:101079008-101079090, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr6:101079008-101079090, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr6:101079008-101079090, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,XY,MT were discarded)\n",
      "NOTICE: For the region chr6:107955663-107956308, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr6:107955663-107956308, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr6:107955663-107956308, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,0,MT were discarded)\n",
      "NOTICE: For the region chr6:146755132-146755553, 5 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr6:146755132-146755553, 5 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr6:146755132-146755553, 5 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,MT,XY were discarded)\n",
      "NOTICE: For the region chr6:152129399-152129456, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr6:152129399-152129456, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr6:152129399-152129456, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,MT,XY were discarded)\n",
      "NOTICE: For the region chr6:170102887-170102970, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr6:170102887-170102970, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr6:170102887-170102970, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,XY,0 were discarded)\n",
      "NOTICE: For the region chr7:6193628-6193722, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr7:6193628-6193722, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr7:6193628-6193722, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,0,XY were discarded)\n",
      "NOTICE: For the region chr8:39550138-39550180, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr8:39550138-39550180, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr8:39550138-39550180, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,XY,MT were discarded)\n",
      "NOTICE: For the region chr8:98828352-98831434, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr8:98828352-98831434, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr8:98828352-98831434, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,XY,0 were discarded)\n",
      "NOTICE: For the region chr8:144990587-144992782, 33 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr8:144990587-144992782, 33 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr8:144990587-144992782, 33 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,0,MT were discarded)\n",
      "NOTICE: For the region chr9:126144043-126145981, 8 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr9:126144043-126145981, 8 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr9:126144043-126145981, 8 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,MT,XY were discarded)\n",
      "NOTICE: For the region chr9:140167307-140196334, 6 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr9:140167307-140196334, 6 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr9:140167307-140196334, 6 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,XY,MT were discarded)\n",
      "NOTICE: For the region chr9:140328047-140328756, 5 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr9:140328047-140328756, 5 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr9:140328047-140328756, 5 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,XY,0 were discarded)\n",
      "NOTICE: For the region chr11:47609867-47611940, 5 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr11:47609867-47611940, 5 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr11:47609867-47611940, 5 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,0,XY were discarded)\n",
      "NOTICE: For the region chr11:70200492-70200500, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr11:70200492-70200500, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr11:70200492-70200500, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,MT,0 were discarded)\n",
      "NOTICE: For the region chr12:33031883-33031889, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr12:33031883-33031889, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr12:33031883-33031889, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,0,MT were discarded)\n",
      "NOTICE: For the region chr12:53096894-53097191, 5 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr12:53096894-53097191, 5 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr12:53096894-53097191, 5 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,0,MT were discarded)\n",
      "NOTICE: For the region chr13:53420234-53422337, 7 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr13:53420234-53422337, 7 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr13:53420234-53422337, 7 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,XY,MT were discarded)\n",
      "NOTICE: For the region chr14:105349452-105349508, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr14:105349452-105349508, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr14:105349452-105349508, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,MT,0 were discarded)\n",
      "NOTICE: For the region chr15:34014942-34015060, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr15:34014942-34015060, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr15:34014942-34015060, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,XY,0 were discarded)\n",
      "NOTICE: For the region chr16:825451-825663, 6 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr16:825451-825663, 6 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr16:825451-825663, 6 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,MT,0 were discarded)\n",
      "NOTICE: For the region chr16:68056661-68057101, 9 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr16:68056661-68057101, 9 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr16:68056661-68057101, 9 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,XY,MT were discarded)\n",
      "NOTICE: For the region chr16:89167117-89167447, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr16:89167117-89167447, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr16:89167117-89167447, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,MT,XY were discarded)\n",
      "NOTICE: For the region chr17:38552663-38552674, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr17:38552663-38552674, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr17:38552663-38552674, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,0,MT were discarded)\n",
      "NOTICE: For the region chr17:43922918-43922927, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr17:43922918-43922927, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr17:43922918-43922927, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,XY,MT were discarded)\n",
      "NOTICE: For the region chr17:46847395-46847447, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr17:46847395-46847447, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr17:46847395-46847447, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,MT,XY were discarded)\n",
      "NOTICE: For the region chr18:20581551-20606167, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr18:20581551-20606167, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr18:20581551-20606167, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,XY,MT were discarded)\n",
      "NOTICE: For the region chr18:34846544-34852992, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr18:34846544-34852992, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr18:34846544-34852992, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,XY,0 were discarded)\n",
      "NOTICE: For the region chr19:407900-408325, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr19:407900-408325, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr19:407900-408325, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,XY,0 were discarded)\n",
      "NOTICE: For the region chr19:5455990-5456142, 6 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr19:5455990-5456142, 6 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr19:5455990-5456142, 6 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,0,XY were discarded)\n",
      "NOTICE: For the region chr19:20044186-20045097, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr19:20044186-20045097, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr19:20044186-20045097, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,XY,0 were discarded)\n",
      "NOTICE: For the region chr19:38964277-38964306, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr19:38964277-38964306, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr19:38964277-38964306, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,0,MT were discarded)\n",
      "NOTICE: For the region chr19:44223113-44223833, 9 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr19:44223113-44223833, 9 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr19:44223113-44223833, 9 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,XY,0 were discarded)\n",
      "NOTICE: For the region chr19:45579592-45656147, 9 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr19:45579592-45656147, 9 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr19:45579592-45656147, 9 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,XY,MT were discarded)\n",
      "NOTICE: For the region chr20:478390-656205, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr20:478390-656205, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr20:478390-656205, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,XY,0 were discarded)\n",
      "NOTICE: For the region chr22:46751377-46759981, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr22:46751377-46759981, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr22:46751377-46759981, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=1 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,XY,0 were discarded)\n",
      "NOTICE: For the region chr1:1663944-1686024, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr1:1663944-1686024, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr1:1663944-1686024, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,MT,0 were discarded)\n",
      "NOTICE: For the region chr1:6614303-6615469, 6 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr1:6614303-6615469, 6 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr1:6614303-6615469, 6 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,MT,XY were discarded)\n",
      "NOTICE: For the region chr1:17281235-17281359, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr1:17281235-17281359, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr1:17281235-17281359, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,MT,XY were discarded)\n",
      "NOTICE: For the region chr1:43204251-43205696, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr1:43204251-43205696, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr1:43204251-43205696, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,XY,MT were discarded)\n",
      "NOTICE: For the region chr1:203144500-203144874, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr1:203144500-203144874, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr1:203144500-203144874, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,MT,XY were discarded)\n",
      "NOTICE: For the region chr1:204971736-205027737, 9 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr1:204971736-205027737, 9 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr1:204971736-205027737, 9 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,XY,MT were discarded)\n",
      "NOTICE: For the region chr2:232458725-232458818, 5 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr2:232458725-232458818, 5 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr2:232458725-232458818, 5 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,XY,0 were discarded)\n",
      "NOTICE: For the region chr2:239147490-239147730, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr2:239147490-239147730, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr2:239147490-239147730, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,MT,XY were discarded)\n",
      "NOTICE: For the region chr2:241569440-241569454, 5 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr2:241569440-241569454, 5 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr2:241569440-241569454, 5 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,XY,MT were discarded)\n",
      "NOTICE: For the region chr3:128711946-128755953, 9 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr3:128711946-128755953, 9 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr3:128711946-128755953, 9 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,XY,MT were discarded)\n",
      "NOTICE: For the region chr3:165491264-165491288, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr3:165491264-165491288, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr3:165491264-165491288, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,0,MT were discarded)\n",
      "NOTICE: For the region chr5:5462181-5462494, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr5:5462181-5462494, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr5:5462181-5462494, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,0,MT were discarded)\n",
      "NOTICE: For the region chr6:33306794-33312308, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr6:33306794-33312308, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr6:33306794-33312308, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,0,MT were discarded)\n",
      "NOTICE: For the region chr7:48312999-48313003, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr7:48312999-48313003, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr7:48312999-48313003, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,0,MT were discarded)\n",
      "NOTICE: For the region chr9:132084144-132084375, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr9:132084144-132084375, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr9:132084144-132084375, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,MT,0 were discarded)\n",
      "NOTICE: For the region chr9:139277730-139277886, 5 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr9:139277730-139277886, 5 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr9:139277730-139277886, 5 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,0,XY were discarded)\n",
      "NOTICE: For the region chr9:140245516-140245893, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr9:140245516-140245893, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr9:140245516-140245893, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,MT,0 were discarded)\n",
      "NOTICE: For the region chr11:3111815-3114820, 10 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr11:3111815-3114820, 10 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr11:3111815-3114820, 10 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,MT,0 were discarded)\n",
      "NOTICE: For the region chr11:64600723-64603925, 14 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr11:64600723-64603925, 14 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr11:64600723-64603925, 14 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,0,MT were discarded)\n",
      "NOTICE: For the region chr12:55368254-55368339, 5 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr12:55368254-55368339, 5 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr12:55368254-55368339, 5 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,XY,0 were discarded)\n",
      "NOTICE: For the region chr12:72175835-72181905, 6 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr12:72175835-72181905, 6 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr12:72175835-72181905, 6 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,MT,0 were discarded)\n",
      "NOTICE: For the region chr13:108922368-108922556, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr13:108922368-108922556, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr13:108922368-108922556, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,XY,MT were discarded)\n",
      "NOTICE: For the region chr14:93022148-93022287, 5 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr14:93022148-93022287, 5 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr14:93022148-93022287, 5 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,0,MT were discarded)\n",
      "NOTICE: For the region chr15:42363395-42363488, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr15:42363395-42363488, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr15:42363395-42363488, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: Evidence for parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 ): Marker= 3 Paternal_origin(F)= 0 Maternal_origin(M)= 2 P-value= 0.5\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,0,XY were discarded)\n",
      "NOTICE: For the region chr15:62226423-62228915, 3 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr15:62226423-62228915, 3 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr15:62226423-62228915, 3 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,0,XY were discarded)\n",
      "NOTICE: For the region chr16:31150508-31152920, 5 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr16:31150508-31152920, 5 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr16:31150508-31152920, 5 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,MT,XY were discarded)\n",
      "NOTICE: For the region chr16:57095409-57101373, 7 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr16:57095409-57101373, 7 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr16:57095409-57101373, 7 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr 0,MT,XY were discarded)\n",
      "NOTICE: For the region chr17:56393814-56396637, 6 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr17:56393814-56396637, 6 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr17:56393814-56396637, 6 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,0,XY were discarded)\n",
      "NOTICE: For the region chr17:80395065-80397586, 5 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr17:80395065-80397586, 5 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr17:80395065-80397586, 5 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,0,XY were discarded)\n",
      "NOTICE: For the region chr19:57909827-57910163, 4 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr19:57909827-57910163, 4 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr19:57909827-57910163, 4 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,MT,0 were discarded)\n",
      "NOTICE: For the region chr22:50942805-50943232, 7 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr22:50942805-50943232, 7 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr22:50942805-50943232, 7 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "module load penncnv\n",
    "\n",
    "cd /data/sudregp/cnv/penncnv\n",
    "box='InfiniumExome'\n",
    "triocnv='results/CLIA_400127.jointcnv'\n",
    "grep offspring ${triocnv} > kid_snps;\n",
    "while read line; do\n",
    "    IFS=' ' read -ra cnv <<< \"$line\"\n",
    "    IFS='=' read -ra arr <<< \"${cnv[3]}\"\n",
    "    ncnv=${arr[1]}\n",
    "    IFS='=' read -ra arr <<< \"${cnv[5]}\"\n",
    "    pstart=${arr[1]}\n",
    "    IFS='=' read -ra arr <<< \"${cnv[6]}\"\n",
    "    pend=${arr[1]}\n",
    "    \n",
    "    infer_snp_allele.pl -pfb ${box}.pfb -hmm penncnv_example.hmm -denovocn $ncnv \\\n",
    "    ${box}/CLIA_400197 ${box}/CLIA_400167 ${box}/CLIA_400127 -out tempfile \\\n",
    "    --startsnp $pstart --endsnp $pend\n",
    "done < kid_snps;\n",
    "rm *_snps;"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "But I think that's because parents are assumed to be CN=2 when using denovo calls. Let's check what happens when we do that, and then don't:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 143,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "chr11:64600723-64603925       numsnp=14     length=3,203       state5,cn=3 InfiniumExome/CLIA_400167 startsnp=exm924016 endsnp=exm924087 mother statepath=555\n",
      "chr11:64600723-64603925       numsnp=14     length=3,203       state5,cn=3 InfiniumExome/CLIA_400127 startsnp=exm924016 endsnp=exm924087 offspring statepath=555\n",
      "Name\tLRR_F\tLRR_M\tLRR_O\tBAF_F\tBAF_M\tBAF_O\tGENO_F\tGENO_M\tGENO_O\tOrigin\n",
      "exm924016\t0.1814\t0.2236\t0.1997\t1.0000\t1.0000\t0.9557\tBB\tBB\tBBB\t?\n",
      "exm1728116\t0.4458\t0.3950\t0.3197\t1.0000\t1.0000\t0.9945\tBB\tBB\tBBB\t?\n",
      "exm924027\t0.4728\t0.4719\t0.1905\t1.0000\t1.0000\t0.9887\tBB\tBB\tBBB\t?\n",
      "exm924031\t0.1405\t0.1147\t-0.0097\t1.0000\t1.0000\t1.0000\tBB\tBB\tBBB\t?\n",
      "exm924033\t0.3733\t0.0748\t0.1137\t1.0000\t0.9973\t1.0000\tBB\tBB\tBBB\t?\n",
      "exm924036\t0.4765\t0.0757\t0.2052\t1.0000\t1.0000\t1.0000\tBB\tBB\tBBB\t?\n",
      "exm924047\t0.5972\t0.4006\t0.3598\t1.0000\t1.0000\t1.0000\tBB\tBB\tBBB\t?\n",
      "exm924054\t0.3457\t0.1414\t0.4115\t1.0000\t0.9966\t0.9984\tBB\tBB\tBBB\t?\n",
      "exm924061\t0.2227\t0.1777\t0.1742\t1.0000\t1.0000\t0.9989\tBB\tBB\tBBB\t?\n",
      "Name\tLRR(1)\tLRR(2)\tLRR(3)\tBAF(1)\tBAF(2)\tBAF(3)\tGENO(1)\tGENO(2)\tGENO(3)\n",
      "exm924016\t0.1814\t0.2236\t0.1997\t1.0000\t1.0000\t0.9557\tBB\tBBB\tBBB\n",
      "exm1728116\t0.4458\t0.3950\t0.3197\t1.0000\t1.0000\t0.9945\tBB\tBBB\tBBB\n",
      "exm924027\t0.4728\t0.4719\t0.1905\t1.0000\t1.0000\t0.9887\tBB\tBBB\tBBB\n",
      "exm924031\t0.1405\t0.1147\t-0.0097\t1.0000\t1.0000\t1.0000\tBB\tBBB\tBBB\n",
      "exm924033\t0.3733\t0.0748\t0.1137\t1.0000\t0.9973\t1.0000\tBB\tBBB\tBBB\n",
      "exm924036\t0.4765\t0.0757\t0.2052\t1.0000\t1.0000\t1.0000\tBB\tBBB\tBBB\n",
      "exm924047\t0.5972\t0.4006\t0.3598\t1.0000\t1.0000\t1.0000\tBB\tBBB\tBBB\n",
      "exm924054\t0.3457\t0.1414\t0.4115\t1.0000\t0.9966\t0.9984\tBB\tBBB\tBBB\n",
      "exm924061\t0.2227\t0.1777\t0.1742\t1.0000\t1.0000\t0.9989\tBB\tBBB\tBBB\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr MT,XY,0 were discarded)\n",
      "NOTICE: For the region chr11:64600723-64603925, 14 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr11:64600723-64603925, 14 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr11:64600723-64603925, 14 markers were identified from InfiniumExome/CLIA_400127\n",
      "NOTICE: Analyzing trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127\n",
      "NOTICE: No evidence of parental origin for the putative de novo CNVs (de novo CN=3 in trio InfiniumExome/CLIA_400197 InfiniumExome/CLIA_400167 InfiniumExome/CLIA_400127 )\n",
      "NOTICE: Reading marker coordinates and population frequency of B allele (PFB) from InfiniumExome.pfb ... Done with 242917 records (428 records in chr XY,0,MT were discarded)\n",
      "NOTICE: For the region chr11:64600723-64603925, 14 markers were identified from InfiniumExome/CLIA_400197\n",
      "NOTICE: For the region chr11:64600723-64603925, 14 markers were identified from InfiniumExome/CLIA_400167\n",
      "NOTICE: For the region chr11:64600723-64603925, 14 markers were identified from InfiniumExome/CLIA_400127\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "module load penncnv\n",
    "\n",
    "cd /data/sudregp/cnv/penncnv\n",
    "\n",
    "grep chr11:64600723-64603925 results/CLIA_400127.jointcnv\n",
    "\n",
    "box='InfiniumExome'\n",
    "\n",
    "infer_snp_allele.pl -pfb ${box}.pfb -hmm penncnv_example.hmm -denovocn 3 \\\n",
    "    ${box}/CLIA_400197 ${box}/CLIA_400167 ${box}/CLIA_400127 -out tempfile \\\n",
    "    --startsnp exm924016 --endsnp exm924087\n",
    "    \n",
    "head tempfile\n",
    "\n",
    "infer_snp_allele.pl -pfb ${box}.pfb -hmm penncnv_example.hmm -allcn 233 \\\n",
    "    ${box}/CLIA_400197 ${box}/CLIA_400167 ${box}/CLIA_400127 -out tempfile \\\n",
    "    --startsnp exm924016 --endsnp exm924087\n",
    "    \n",
    "head tempfile"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "OK, so this is a bit better. Basically, it only makes sense to calculate p-values for denovo calls, because of the assumption that both parents are cn=2. And the genotyping changes based on that cn. Now, the question is whether we need to calculate these p-values or not. Let's not do it for now, and just generate a nice set of variants for plink, splitting it into inherited and denovo variants."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 162,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "cd /data/sudregp/cnv/penncnv/results\n",
    "\n",
    "rm denovo_numCNVse20.txt inherited_numCNVse20.txt 2>/dev/null\n",
    "\n",
    "while read sample; do\n",
    "    triocnv=${sample}'.jointcnv'\n",
    "    grep mother ${triocnv} > mom_snps;\n",
    "    grep father ${triocnv} > dad_snps;\n",
    "    cat mom_snps dad_snps > parent_snps;\n",
    "    grep offspring ${triocnv} > kid_snps;\n",
    "    while read line; do\n",
    "        IFS=' ' read -ra arr <<< \"$line\"\n",
    "        snp=${arr[0]}\n",
    "        if ! grep -q $snp parent_snps; then\n",
    "            echo $line >> denovo_numCNVse20.txt\n",
    "        else\n",
    "            echo $line >> inherited_numCNVse20.txt\n",
    "        fi;\n",
    "    done < kid_snps;\n",
    "    rm *_snps;\n",
    "done < ../good_kids_numCNVse20.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 163,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import re\n",
    "\n",
    "\n",
    "regex = re.compile('^chr(\\d+):(\\d+)-(\\d+)\\s+numsnp=(\\d+)\\s+\\S+\\s+\\S+cn=(\\d+)\\s+\\S+/(\\S+)')\n",
    "fout = open('/data/sudregp/cnv/penncnv/denovo_numCNVse20.cnv', 'w')\n",
    "fid = open('/data/sudregp/cnv/penncnv/results/denovo_numCNVse20.txt', 'r')\n",
    "for line in fid:\n",
    "    match = regex.search(line)\n",
    "    chr, start, end, numsnp, cn, sample = match.groups()\n",
    "    fout.write('\\t'.join([famid, sample, chr, start, end, cn, numsnp,\n",
    "                          '0']) + '\\n')\n",
    "fout.close()\n",
    "\n",
    "fout = open('/data/sudregp/cnv/penncnv/inherited_numCNVse20.cnv', 'w')\n",
    "fid = open('/data/sudregp/cnv/penncnv/results/inherited_numCNVse20.txt', 'r')\n",
    "for line in fid:\n",
    "    match = regex.search(line)\n",
    "    chr, start, end, numsnp, cn, sample = match.groups()\n",
    "    fout.write('\\t'.join([famid, sample, chr, start, end, cn, numsnp,\n",
    "                          '0']) + '\\n')\n",
    "fout.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 168,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_numCNVse20.log ]\n",
      "Analysis started: Fri Dec  8 19:01:28 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list denovo_numCNVse20.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--out denovo_numCNVse20\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_numCNVse20.cnv ]\n",
      "Writing new MAP file to [ denovo_numCNVse20.cnv.map ]\n",
      "Wrote 1370 unique positions to file\n",
      "\n",
      "Analysis finished: Fri Dec  8 19:01:28 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_numCNVse20.log ]\n",
      "Analysis started: Fri Dec  8 19:01:28 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--cfile denovo_numCNVse20\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--out denovo_numCNVse20\n",
      "\t--cnv-kb 0\n",
      "\n",
      "Reading marker information from [ denovo_numCNVse20.cnv.map ]\n",
      "1370 (of 1370) markers to be included from [ denovo_numCNVse20.cnv.map ]\n",
      "Reading individual information from [ denovo_numCNVse20.fam ]\n",
      "Reading pedigree information from [ denovo_numCNVse20.fam ] \n",
      "99 individuals read from [ denovo_numCNVse20.fam ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_numCNVse20.cnv ]\n",
      "21 mapped to a person, of which 21 passed filters\n",
      "21 of 542 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        8 / 3\n",
      "     3        2 / 8\n",
      "\n",
      "Writing per-individual summary to [ denovo_numCNVse20.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_numCNVse20.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Dec  8 19:01:28 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_numCNVse20.log ]\n",
      "Analysis started: Fri Dec  8 19:01:28 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list inherited_numCNVse20.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--out inherited_numCNVse20\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_numCNVse20.cnv ]\n",
      "Writing new MAP file to [ inherited_numCNVse20.cnv.map ]\n",
      "Wrote 6577 unique positions to file\n",
      "\n",
      "Analysis finished: Fri Dec  8 19:01:29 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_numCNVse20.log ]\n",
      "Analysis started: Fri Dec  8 19:01:29 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--cfile inherited_numCNVse20\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--out inherited_numCNVse20\n",
      "\t--cnv-kb 0\n",
      "\n",
      "Reading marker information from [ inherited_numCNVse20.cnv.map ]\n",
      "6577 (of 6577) markers to be included from [ inherited_numCNVse20.cnv.map ]\n",
      "Reading individual information from [ inherited_numCNVse20.fam ]\n",
      "Reading pedigree information from [ inherited_numCNVse20.fam ] \n",
      "99 individuals read from [ inherited_numCNVse20.fam ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_numCNVse20.cnv ]\n",
      "264 mapped to a person, of which 264 passed filters\n",
      "264 of 7386 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      42 / 55\n",
      "     3     29 / 138\n",
      "\n",
      "Writing per-individual summary to [ inherited_numCNVse20.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_numCNVse20.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Dec  8 19:01:29 2017\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "module load plink/1.07\n",
    "\n",
    "cd /data/sudregp/cnv/penncnv/\n",
    "\n",
    "for cnv in denovo inherited; do\n",
    "    rm ${cnv}*fam ${cnv}*log ${cnv}*summary ${cnv}*indiv ${cnv}*map\n",
    "    cp /data/NCR_SBRB/simplex/simplex.ped ${cnv}_numCNVse20.fam\n",
    "    plink --cnv-list ${cnv}_numCNVse20.cnv --cnv-make-map --noweb --out ${cnv}_numCNVse20\n",
    "    plink --cfile ${cnv}_numCNVse20 --noweb --1 --out ${cnv}_numCNVse20 --cnv-kb 0\n",
    "#     plink --cfile ${cnv}_numCNVse20 --noweb --1 --cnv-disrupt \\\n",
    "#         --cnv-intersect gene_locations.txt --cnv-write \\\n",
    "#         --out ${cnv}_numCNVse20_gene_disrupt\n",
    "done"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "These numbers imported by PLINK don't look correct. Need to figure out what's going on. Of course, the other option would be to not use PLINK, but then I'll have to figure out how to trim the list of genes and positions manually. The other otpion is trying parseCNV, which looks a bit buggy, but at least it has all the scripts we'd need: QC, denovo calling, region intersection. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# TODO\n",
    "\n",
    "* Give ParseCNV a try (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3597648/pdf/gks1346.pdf)\n",
    "* Understand what we're QCing on!\n",
    "* Worth calculating p-values? For that one trio it was always 0!\n",
    "* Try PennCNV steps with adjusted pipeline to see if we get anything different\n",
    "* Look into sex chromosomes? Something to the idea that adhd is more present in boys...\n",
    "* Play with the minimum size of the CNV\n",
    "* Play with the HMM parameters\n",
    "* Check (and plot) within family differences; the ones with big differences are interesting.\n",
    "* Include parent burden in the analysis\n",
    "* Filter based on neural vs nonneural CNVs (i.e. expressed in the brain\n",
    "* Look at literature-only CNVs (J Chia and A Thapaer for CNVs in ADHD\n",
    "* Match with file of ranked simplex by Wendy (maybe blindly)?\n",
    "* Do all of the above with XHMM and array data. Maybe start with array because it'll be simpler?"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Useful links\n",
    "\n",
    "* http://research.lunenfeld.ca/gwa_studies/files/file/GWAS/Hadley_CNV/Hadley_CNV_practical.pdf\n",
    "* http://penncnvexomeseq.sourceforge.net: interesting idea of how to combine PennCNV and XHMM, but seems very beta...\n",
    "* http://parsecnv.sourceforge.net: seems to come from the same group as above... a bit beta as well\n",
    "* http://penncnv.openbioinformatics.org/en/latest/user-guide/denovo/"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python [default]",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.14"
  },
  "nav_menu": {},
  "toc": {
   "navigate_menu": true,
   "number_sections": true,
   "sideBar": true,
   "threshold": 6,
   "toc_cell": false,
   "toc_section_display": "block",
   "toc_window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
