{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "While I was playing with other tools I noticed that CCGO_800976 was failing in most of them. So, I decided to re-process that sample in GATK and now it works fine. Probably something went off when batch processing all samples. But this means we need to re-run XHMM, as it takes in all samples.\n",
    "\n",
    "Just for clarification, we'll need to trim a couple samples as well. There are a total of 99 samples in the .ped, but CLIA_400169 and CLIA_400168 are part of a family without a dad sequenced, so let's remove those two so that we stick to estimating everything on the trio samples only."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true,
    "scrolled": false
   },
   "outputs": [],
   "source": [
    "# copied all DOC* from /data/NCR_SBRB/simplex/xhmm to ~/data/cnv/xhmm\n",
    "# just rerunning that sample\n",
    "# running on terminal because output clogs up the notebook\n",
    "bash ~/autodenovo/xhmm_get_DOC.sh CCGO_800976"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "exome_targets='/data/NCR_SBRB/simplex/SeqCapEZ_Exome_v3.0_Design_Annotation_files/SeqCap_EZ_Exome_v3_hg19_capture_targets.bed'\n",
    "gatk_memory=\"50g\"\n",
    "ref_fa='/fdb/GATK_resource_bundle/hg19-2.8/ucsc.hg19.fasta'\n",
    "out_dir='/data/sudregp/cnv/xhmm'\n",
    "\n",
    "cd $out_dir\n",
    "module load GATK\n",
    "module load XHMM\n",
    "\n",
    "GATK -m ${gatk_memory} GCContentByInterval -L ${exome_targets} -R ${ref_fa} -o ./DATA.locus_GC.txt\n",
    "cat ./DATA.locus_GC.txt | awk '{if ($2 < 0.1 || $2 > 0.9) print $1}' > ./extreme_gc_targets.txt\n",
    "\n",
    "# merging all subjects in the directory\n",
    "ls -1 *.sample_interval_summary > depth_list.txt;\n",
    "grep -v -f ../exclude.txt depth_list.txt > depth_list2.txt\n",
    "\n",
    "xhmm --mergeGATKdepths --GATKdepthsList=depth_list2.txt -o ./DATA.RD.txt;\n",
    "\n",
    "/usr/local/apps/XHMM/2016-01-04/sources/scripts/interval_list_to_pseq_reg ${exome_targets} > ./EXOME.targets.reg\n",
    "\n",
    "xhmm --matrix -r ./DATA.RD.txt --centerData --centerType target \\\n",
    "-o ./DATA.filtered_centered.RD.txt \\\n",
    "--outputExcludedTargets ./DATA.filtered_centered.RD.txt.filtered_targets.txt \\\n",
    "--outputExcludedSamples ./DATA.filtered_centered.RD.txt.filtered_samples.txt \\\n",
    "--minTargetSize 10 --maxTargetSize 10000 \\\n",
    "--minMeanTargetRD 10 --maxMeanTargetRD 500 \\\n",
    "--minMeanSampleRD 25 --maxMeanSampleRD 200 \\\n",
    "--maxSdSampleRD 150\n",
    "\n",
    "xhmm --PCA -r ./DATA.filtered_centered.RD.txt --PCAfiles ./DATA.RD_PCA\n",
    "\n",
    "xhmm --normalize -r ./DATA.filtered_centered.RD.txt --PCAfiles ./DATA.RD_PCA \\\n",
    "--normalizeOutput ./DATA.PCA_normalized.txt \\\n",
    "--PCnormalizeMethod PVE_mean --PVE_mean_factor 0.7\n",
    "\n",
    "xhmm --matrix -r ./DATA.PCA_normalized.txt --centerData --centerType sample --zScoreData \\\n",
    "-o ./DATA.PCA_normalized.filtered.sample_zscores.RD.txt \\\n",
    "--outputExcludedTargets ./DATA.PCA_normalized.filtered.sample_zscores.RD.txt.filtered_targets.txt \\\n",
    "--outputExcludedSamples ./DATA.PCA_normalized.filtered.sample_zscores.RD.txt.filtered_samples.txt \\\n",
    "--maxSdTargetRD 30\n",
    "\n",
    "xhmm --matrix -r ./DATA.RD.txt \\\n",
    "--excludeTargets ./DATA.filtered_centered.RD.txt.filtered_targets.txt \\\n",
    "--excludeTargets ./DATA.PCA_normalized.filtered.sample_zscores.RD.txt.filtered_targets.txt \\\n",
    "--excludeSamples ./DATA.filtered_centered.RD.txt.filtered_samples.txt \\\n",
    "--excludeSamples ./DATA.PCA_normalized.filtered.sample_zscores.RD.txt.filtered_samples.txt \\\n",
    "-o ./DATA.same_filtered.RD.txt\n",
    "\n",
    "xhmm --discover -p params.txt \\\n",
    "    -r ./DATA.PCA_normalized.filtered.sample_zscores.RD.txt \\\n",
    "    -R ./DATA.same_filtered.RD.txt -c ./DATA.xcnv \\\n",
    "    -a ./DATA.aux_xcnv -s ./DATA\n",
    "\n",
    "xhmm --genotype -p params.txt \\\n",
    "    -r ./DATA.PCA_normalized.filtered.sample_zscores.RD.txt \\\n",
    "    -R ./DATA.same_filtered.RD.txt -g ./DATA.xcnv \\\n",
    "    -F $ref_fa -v ./DATA.vcf"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " failed connectionCreating new project specification file [ DATA.pseq ]\n",
      "Inserted 99 new individuals, updated 0 existing individuals\n",
      "loading : /spin1/users/sudregp/cnv/xhmm/DATA.vcf ( 98 individuals )\n",
      "parsed 8000 rows        \n",
      "/spin1/users/sudregp/cnv/xhmm/DATA.vcf : inserted 8324 variants\n",
      "-------------------------------------------------------------------------------\n",
      "||||||||||||||||||||||||||| PSEQ (v0.10; 14-Jul-14) |||||||||||||||||||||||||||\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Copying this log to file [ DATA.log ]\n",
      "Analysis started Thu Dec 21 13:04:32 2017\n",
      "\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Project : DATA\n",
      "Command : cnv-denovo\n",
      "Options : --noweb\n",
      "          --minSQ 60\n",
      "          --minNQ 60\n",
      "          --out DATA\n",
      "\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Writing to file [ DATA.denovo.cnv ]  : per-site output from cnv-denovo\n",
      "Writing to file [ DATA.denovo.cnv.indiv ]  : per-trio output from cnv-denovo\n",
      "Starting CNV de novo scan...\n",
      "Included 8324 of 8324 variants considered\n",
      "\n",
      "Analysis finished Thu Dec 21 13:04:44 2017\n",
      ":::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::\n"
     ]
    }
   ],
   "source": [
    "module load plinkseq\n",
    "module load plink/1.07\n",
    "\n",
    "cd ~/data/cnv/xhmm\n",
    "\n",
    "rm -rf DATA.pseq DATA_out DATA_res\n",
    "pseq DATA new-project\n",
    "# adding a first column with subject ID for PSEQ\n",
    "cut -f 2 ../simplex.ped > junk.txt\n",
    "paste junk.txt ../simplex.ped > simplex.ped.info\n",
    "pseq DATA load-pedigree --file simplex.ped.info\n",
    "pseq DATA load-vcf --vcf DATA.vcf\n",
    "pseq DATA cnv-denovo --noweb --minSQ 60 --minNQ 60 --out DATA"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now, just like what we did for PennCNV, let's create PLINK files for denovo, inherited, and everything. We do have this knob of quality to play with beforehand, so let's keep that in mind.\n",
    "\n",
    "If I'm just interested in numbers, I could just grab the information from the DATA.denovo.cnv.indiv file, which looks like this:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "#SUMMARY\tCHILD\tCHILD_CNV\tIN_PATERNAL\tIN_MATERNAL\tIN_PATERNAL_AND_MATERNAL\tMISSING_PATERNAL\tMISSING_MATERNAL\tDE_NOVO\tPATERNAL_TRANSMITTED\tPATERNAL_NON_TRANSMITTED\tPATERNAL_UNKNOWN\tMATERNAL_TRANSMITTED\tMATERNAL_NON_TRANSMITTED\tMATERNAL_UNKNOWN\n",
      "SUMMARY\tCLIA_400121\t111\t34\t25\t5\t44\t44\t10\t18\t24\t18\t9\t24\t16\n",
      "SUMMARY\tCLIA_400122\t137\t48\t65\t25\t45\t40\t13\t11\t11\t19\t22\t27\t25\n",
      "SUMMARY\tCLIA_400123\t103\t10\t20\t1\t36\t39\t26\t2\t9\t7\t14\t42\t24\n",
      "SUMMARY\tCLIA_400125\t97\t30\t37\t8\t31\t27\t7\t13\t29\t23\t19\t18\t15\n",
      "SUMMARY\tCLIA_400126\t116\t30\t33\t6\t32\t42\t19\t12\t40\t24\t19\t23\t21\n",
      "SUMMARY\tCLIA_400127\t109\t27\t28\t5\t37\t43\t13\t15\t31\t22\t11\t21\t16\n",
      "SUMMARY\tCLIA_400128\t124\t44\t27\t6\t45\t42\t13\t22\t23\t19\t13\t13\t16\n",
      "SUMMARY\tCLIA_400129\t115\t36\t45\t9\t41\t36\t8\t16\t19\t21\t21\t22\t23\n",
      "SUMMARY\tCLIA_400130\t109\t27\t28\t4\t52\t42\t9\t13\t13\t13\t9\t19\t16\n"
     ]
    }
   ],
   "source": [
    "head DATA.denovo.cnv.indiv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "But I'll likely want to do some further filtering in PLINK, so I'l need the non .indiv file."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "#CLASS\n",
      "DENOVO\n",
      "MATERNAL_NON_TRANSMITTED\n",
      "MATERNAL_TRANSMITTED\n",
      "MATERNAL_UNKNOWN\n",
      "PATERNAL_NON_TRANSMITTED\n",
      "PATERNAL_TRANSMITTED\n",
      "PATERNAL_UNKNOWN\n"
     ]
    }
   ],
   "source": [
    "head DATA.denovo.cnv | cut -f 1 | sort | uniq"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Let's format the file into something PLINK can understand. XHMM has a script to convert from xcnv to cnv, so we just need to filter all DENOVO calls from PSEQ in the main xcnv file:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "if [ ! -e ../kid_samples.txt ]; then\n",
    "    # create a list of kids first\n",
    "    while read line; do\n",
    "        read -r -a array <<< \"$line\";\n",
    "        id=${array[1]}\n",
    "        fa=${array[2]}\n",
    "        mo=${array[3]}\n",
    "\n",
    "        # figure out relationship\n",
    "        if [ $fa != 0 ] && [ $mo != 0 ]; then\n",
    "            child=${id};\n",
    "            echo $child >> ../kid_samples.txt;\n",
    "        fi;\n",
    "    done < ../simplex.ped;\n",
    "fi"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "grep DENOVO DATA.denovo.cnv > pseq_DENOVO.txt\n",
    "# borrow the header row\n",
    "head -1 DATA.xcnv > denovo.xcnv;\n",
    "\n",
    "while read sample; do\n",
    "    grep $sample DATA.xcnv > sample.xcnv;\n",
    "    for cnv in `grep $sample pseq_DENOVO.txt | cut -f 3 -`; do\n",
    "        # replacing .. by -\n",
    "        cnv=`echo $cnv | sed -e 's/\\.\\./\\-/'`;\n",
    "        grep $cnv sample.xcnv >> denovo.xcnv; \n",
    "    done;\n",
    "done < ../kid_samples.txt;\n",
    "/usr/local/apps/XHMM/2016-01-04/sources/scripts/xcnv_to_cnv denovo.xcnv > tmp.cnv\n",
    "# switch around FAMID and IID columns, and remove header\n",
    "awk '{OFS=\"\\t\"; if ( $3 != \"CHR\" ) {print $2, $1, $3, $4, $5, $6, $7, $8 }}' tmp.cnv > denovo.cnv\n",
    "rm sample.xcnv pseq_DENOVO.txt tmp.cnv denovo.xcnv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "grep MATERNAL_TRANSMITTED DATA.denovo.cnv > pseq_TRANSMITTED.txt\n",
    "grep PATERNAL_TRANSMITTED DATA.denovo.cnv >> pseq_TRANSMITTED.txt\n",
    "# borrow the header row\n",
    "head -1 DATA.xcnv > inherited.xcnv;\n",
    "\n",
    "while read sample; do\n",
    "    grep $sample DATA.xcnv > sample.xcnv;\n",
    "    for cnv in `grep $sample pseq_TRANSMITTED.txt | cut -f 3 -`; do\n",
    "        # replacing .. by -\n",
    "        cnv=`echo $cnv | sed -e 's/\\.\\./\\-/'`;\n",
    "        grep $cnv sample.xcnv >> inherited.xcnv; \n",
    "    done;\n",
    "done < ../kid_samples.txt;\n",
    "/usr/local/apps/XHMM/2016-01-04/sources/scripts/xcnv_to_cnv inherited.xcnv > tmp.cnv\n",
    "# switch around FAMID and IID columns, and remove header\n",
    "awk '{OFS=\"\\t\"; if ( $3 != \"CHR\" ) {print $2, $1, $3, $4, $5, $6, $7, $8 }}' tmp.cnv > inherited.cnv\n",
    "rm sample.xcnv pseq_TRANSMITTED.txt tmp.cnv inherited.xcnv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# borrow the header row\n",
    "head -1 DATA.xcnv > all.xcnv;\n",
    "\n",
    "# effectively just filtering DATA.xcnv to keep only kids\n",
    "while read sample; do\n",
    "    grep $sample DATA.xcnv >> all.xcnv;\n",
    "done < ../kid_samples.txt;\n",
    "/usr/local/apps/XHMM/2016-01-04/sources/scripts/xcnv_to_cnv all.xcnv > tmp.cnv\n",
    "# switch around FAMID and IID columns, and remove header\n",
    "awk '{OFS=\"\\t\"; if ( $3 != \"CHR\" ) {print $2, $1, $3, $4, $5, $6, $7, $8 }}' tmp.cnv > all.cnv\n",
    "rm tmp.cnv all.xcnv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The issue here is that PSEQ doesn't carry forward the FAMID, so PLINK doesn't know how to add the information back to the sample IDs, which all have FAMIDs in the .ped. So, we need to create a .ped without FAMIDs, whic is fine since at this point all the family information has already been used, and PLINK doesn't need it anymore."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "rm ones.txt;\n",
    "nsamples=`cat ../simplex.ped | wc -l`\n",
    "for i in `seq 1 $nsamples`; do echo 1 >> ones.txt; done;\n",
    "cut -f 2,3,4,5,6 ../simplex.ped > junk;\n",
    "paste ones.txt junk > ../simplex_nofamid.ped;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all.log ]\n",
      "Analysis started: Thu Dec 21 15:23:26 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list all.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--out all\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing new MAP file to [ all.cnv.map ]\n",
      "Wrote 10382 unique positions to file\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:27 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_burden.log ]\n",
      "Analysis started: Thu Dec 21 15:23:27 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--out all_burden\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "No overlapping samples found\n",
      "7223 mapped to a person, of which 7223 passed filters\n",
      "7223 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1  1761 / 1811\n",
      "     3  1585 / 2066\n",
      "\n",
      "Writing per-individual summary to [ all_burden.cnv.indiv ]\n",
      "Writing positional summary to [ all_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:27 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_genes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:27 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--out all_genes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_genes.reg ]\n",
      "7223 mapped to a person, of which 7223 passed filters\n",
      "6192 intersected with one or more specified region\n",
      "6192 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1  1527 / 1535\n",
      "     3  1342 / 1788\n",
      "\n",
      "Writing per-individual summary to [ all_genes.cnv.indiv ]\n",
      "Writing positional summary to [ all_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:35 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_brainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:35 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--out all_brainGenes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_brainGenes.reg ]\n",
      "7223 mapped to a person, of which 7223 passed filters\n",
      "57 intersected with one or more specified region\n",
      "57 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      14 / 10\n",
      "     3      14 / 19\n",
      "\n",
      "Writing per-individual summary to [ all_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:35 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_delBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:23:35 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--out all_delBurden\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "7223 mapped to a person, of which 3572 passed filters\n",
      "3572 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1  1761 / 1811\n",
      "\n",
      "Writing per-individual summary to [ all_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ all_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:35 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_delGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:35 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--out all_delGenes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_delGenes.reg ]\n",
      "7223 mapped to a person, of which 3572 passed filters\n",
      "3062 intersected with one or more specified region\n",
      "3062 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1  1527 / 1535\n",
      "\n",
      "Writing per-individual summary to [ all_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:39 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_delBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:39 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--out all_delBrainGenes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_delBrainGenes.reg ]\n",
      "7223 mapped to a person, of which 3572 passed filters\n",
      "24 intersected with one or more specified region\n",
      "24 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      14 / 10\n",
      "\n",
      "Writing per-individual summary to [ all_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:40 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_dupBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:23:40 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--out all_dupBurden\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "7223 mapped to a person, of which 3651 passed filters\n",
      "3651 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3  1585 / 2066\n",
      "\n",
      "Writing per-individual summary to [ all_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ all_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:40 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_dupGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:40 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--out all_dupGenes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_dupGenes.reg ]\n",
      "7223 mapped to a person, of which 3651 passed filters\n",
      "3130 intersected with one or more specified region\n",
      "3130 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3  1342 / 1788\n",
      "\n",
      "Writing per-individual summary to [ all_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:44 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_dupBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:44 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--out all_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_dupBrainGenes.reg ]\n",
      "7223 mapped to a person, of which 3651 passed filters\n",
      "33 intersected with one or more specified region\n",
      "33 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      14 / 19\n",
      "\n",
      "Writing per-individual summary to [ all_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:44 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo.log ]\n",
      "Analysis started: Thu Dec 21 15:23:44 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--out denovo\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing new MAP file to [ denovo.cnv.map ]\n",
      "Wrote 1757 unique positions to file\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:44 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_burden.log ]\n",
      "Analysis started: Thu Dec 21 15:23:44 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--out denovo_burden\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "No overlapping samples found\n",
      "665 mapped to a person, of which 665 passed filters\n",
      "665 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    260 / 147\n",
      "     3    118 / 140\n",
      "\n",
      "Writing per-individual summary to [ denovo_burden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:44 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_genes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:44 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--out denovo_genes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_genes.reg ]\n",
      "665 mapped to a person, of which 665 passed filters\n",
      "593 intersected with one or more specified region\n",
      "593 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    241 / 125\n",
      "     3    104 / 123\n",
      "\n",
      "Writing per-individual summary to [ denovo_genes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:45 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_brainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:45 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--out denovo_brainGenes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_brainGenes.reg ]\n",
      "665 mapped to a person, of which 665 passed filters\n",
      "8 intersected with one or more specified region\n",
      "8 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        3 / 3\n",
      "     3        1 / 1\n",
      "\n",
      "Writing per-individual summary to [ denovo_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:45 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_delBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:23:45 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--out denovo_delBurden\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "665 mapped to a person, of which 407 passed filters\n",
      "407 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    260 / 147\n",
      "\n",
      "Writing per-individual summary to [ denovo_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:45 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_delGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:45 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--out denovo_delGenes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_delGenes.reg ]\n",
      "665 mapped to a person, of which 407 passed filters\n",
      "366 intersected with one or more specified region\n",
      "366 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    241 / 125\n",
      "\n",
      "Writing per-individual summary to [ denovo_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:46 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_delBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:46 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--out denovo_delBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_delBrainGenes.reg ]\n",
      "665 mapped to a person, of which 407 passed filters\n",
      "6 intersected with one or more specified region\n",
      "6 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        3 / 3\n",
      "\n",
      "Writing per-individual summary to [ denovo_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:46 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_dupBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:23:46 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--out denovo_dupBurden\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "665 mapped to a person, of which 258 passed filters\n",
      "258 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3    118 / 140\n",
      "\n",
      "Writing per-individual summary to [ denovo_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:46 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_dupGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:46 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--out denovo_dupGenes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_dupGenes.reg ]\n",
      "665 mapped to a person, of which 258 passed filters\n",
      "227 intersected with one or more specified region\n",
      "227 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3    104 / 123\n",
      "\n",
      "Writing per-individual summary to [ denovo_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:47 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_dupBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:47 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--out denovo_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_dupBrainGenes.reg ]\n",
      "665 mapped to a person, of which 258 passed filters\n",
      "2 intersected with one or more specified region\n",
      "2 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 1\n",
      "\n",
      "Writing per-individual summary to [ denovo_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:47 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited.log ]\n",
      "Analysis started: Thu Dec 21 15:23:47 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--out inherited\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing new MAP file to [ inherited.cnv.map ]\n",
      "Wrote 807 unique positions to file\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:47 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_burden.log ]\n",
      "Analysis started: Thu Dec 21 15:23:47 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--out inherited_burden\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "No overlapping samples found\n",
      "401 mapped to a person, of which 401 passed filters\n",
      "401 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1     95 / 119\n",
      "     3      88 / 99\n",
      "\n",
      "Writing per-individual summary to [ inherited_burden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:47 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_genes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:47 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--out inherited_genes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_genes.reg ]\n",
      "401 mapped to a person, of which 401 passed filters\n",
      "322 intersected with one or more specified region\n",
      "322 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      79 / 97\n",
      "     3      69 / 77\n",
      "\n",
      "Writing per-individual summary to [ inherited_genes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:48 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_brainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:48 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--out inherited_brainGenes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_brainGenes.reg ]\n",
      "401 mapped to a person, of which 401 passed filters\n",
      "2 intersected with one or more specified region\n",
      "2 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 1\n",
      "\n",
      "Writing per-individual summary to [ inherited_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:48 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_delBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:23:48 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--out inherited_delBurden\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "401 mapped to a person, of which 214 passed filters\n",
      "214 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1     95 / 119\n",
      "\n",
      "Writing per-individual summary to [ inherited_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:48 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_delGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:48 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--out inherited_delGenes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_delGenes.reg ]\n",
      "401 mapped to a person, of which 214 passed filters\n",
      "176 intersected with one or more specified region\n",
      "176 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      79 / 97\n",
      "\n",
      "Writing per-individual summary to [ inherited_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:49 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_delBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:49 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--out inherited_delBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_delBrainGenes.reg ]\n",
      "401 mapped to a person, of which 214 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:49 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_dupBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:23:49 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--out inherited_dupBurden\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "401 mapped to a person, of which 187 passed filters\n",
      "187 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      88 / 99\n",
      "\n",
      "Writing per-individual summary to [ inherited_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:49 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_dupGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:49 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--out inherited_dupGenes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_dupGenes.reg ]\n",
      "401 mapped to a person, of which 187 passed filters\n",
      "146 intersected with one or more specified region\n",
      "146 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      69 / 77\n",
      "\n",
      "Writing per-individual summary to [ inherited_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:49 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_dupBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:23:49 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--out inherited_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_dupBrainGenes.reg ]\n",
      "401 mapped to a person, of which 187 passed filters\n",
      "2 intersected with one or more specified region\n",
      "2 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 1\n",
      "\n",
      "Writing per-individual summary to [ inherited_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:23:49 2017\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for cnvtype in all denovo inherited; do\n",
    "    cnvname=${cnvtype}.cnv\n",
    "    plink --cnv-list $cnvname --cnv-make-map --noweb --out ${cnvtype};\n",
    "    # whole burden\n",
    "    plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "        --noweb --1 --cnv-check-no-overlap --out ${cnvtype}_burden;\n",
    "    # gene sets\n",
    "    plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "        --noweb --1 --cnv-intersect glist-hg19 --cnv-verbose-report-regions \\\n",
    "        --cnv-subset genes.txt --out ${cnvtype}_genes;\n",
    "    plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "        --noweb --1 --cnv-intersect glist-hg19 --cnv-verbose-report-regions \\\n",
    "        --cnv-subset hg19_allenBrainGene_trimmed.txt \\\n",
    "        --out ${cnvtype}_brainGenes;\n",
    "    # subtypes only\n",
    "    for sub in del dup; do\n",
    "        plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "        --noweb --1 --cnv-${sub} --out ${cnvtype}_${sub}Burden;\n",
    "        # gene sets\n",
    "        plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "            --noweb --1 --cnv-intersect glist-hg19 --cnv-verbose-report-regions \\\n",
    "            --cnv-subset genes.txt --cnv-${sub} \\\n",
    "            --out ${cnvtype}_${sub}Genes;\n",
    "        plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "            --noweb --1 --cnv-intersect glist-hg19 --cnv-verbose-report-regions \\\n",
    "            --cnv-subset hg19_allenBrainGene_trimmed.txt --cnv-${sub} \\\n",
    "            --out ${cnvtype}_${sub}BrainGenes;\n",
    "    done;\n",
    "done"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now let's do some plotting in Python."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<script>requirejs.config({paths: { 'plotly': ['https://cdn.plot.ly/plotly-latest.min']},});if(!window.Plotly) {{require(['plotly'],function(plotly) {window.Plotly=plotly;});}}</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<script>requirejs.config({paths: { 'plotly': ['https://cdn.plot.ly/plotly-latest.min']},});if(!window.Plotly) {{require(['plotly'],function(plotly) {window.Plotly=plotly;});}}</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import numpy as np\n",
    "import matplotlib.pyplot as plt\n",
    "import seaborn as sns\n",
    "import pandas as pd\n",
    "from plotly.offline import download_plotlyjs, init_notebook_mode, plot, iplot\n",
    "\n",
    "init_notebook_mode(connected=True)\n",
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# figure out who is who in each trio\n",
    "import glob\n",
    "ped_file = '/data/sudregp/cnv/simplex.ped'\n",
    "wes_prefix = ['CLIA', 'CCGO', 'WPS']\n",
    "trios = []\n",
    "affected = []\n",
    "controls = []\n",
    "samples = []\n",
    "famids = []\n",
    "fid = open(ped_file, 'r')\n",
    "for line in fid:\n",
    "    famid, sid, fa, mo, sex, aff = line.rstrip().split('\\t')\n",
    "    if fa[:4] in wes_prefix and mo[:4] in wes_prefix and sid[:4] in wes_prefix:\n",
    "        fam = {}\n",
    "        fam['child'] = sid\n",
    "        if aff == '1':\n",
    "            affected.append(sid)\n",
    "        else:\n",
    "            controls.append(sid)\n",
    "        fam['father'] = fa\n",
    "        fam['mother'] = mo\n",
    "        fam['famid'] = famid\n",
    "        trios.append(fam)\n",
    "        samples += [sid, fa, mo]\n",
    "        famids.append(famid)\n",
    "fid.close()\n",
    "samples = set(samples)\n",
    "famids = set(famids)\n",
    "\n",
    "fid = open('/data/sudregp/cnv/kid_samples.txt', 'r')\n",
    "good_kids = [line.rstrip() for line in fid]\n",
    "fid.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def plot_plink_cnvs(fname, t_str, verbose=False):\n",
    "    \n",
    "    import plotly.graph_objs as go\n",
    "    from plotly import tools\n",
    "\n",
    "    df = pd.read_table(fname, delimiter='\\s+', index_col=1)\n",
    "    df.head()\n",
    "\n",
    "    x_red, x_green, red, green, xticks = [], [], [], [], []\n",
    "    red_text, green_text = [], []\n",
    "\n",
    "    # loop through families\n",
    "    f = 0\n",
    "    for fam in famids:\n",
    "        fam_kids = [t['child'] for t in trios if t['famid'] == fam]\n",
    "        found = False\n",
    "        for kid in fam_kids:\n",
    "            if kid in good_kids:\n",
    "                found = True\n",
    "                idx = [i for i in range(df.shape[0]) if df.index[i].find(kid) >= 0][0]\n",
    "                if kid in affected:\n",
    "                    red.append(int(df.iloc[idx]['NSEG']))\n",
    "                    x_red.append(f)\n",
    "                    red_text.append(kid)\n",
    "                else:\n",
    "                    green.append(int(df.iloc[idx]['NSEG']))\n",
    "                    x_green.append(f)\n",
    "                    green_text.append(kid)\n",
    "        # only increase counter if we added a kid\n",
    "        if found:\n",
    "            xticks.append(fam)\n",
    "            f += 1\n",
    "\n",
    "    fig = go.Figure()\n",
    "                \n",
    "    trace0 = go.Scatter(\n",
    "        x = x_red,\n",
    "        y = red,\n",
    "        mode = 'markers',\n",
    "        name = 'affected',\n",
    "        marker = dict(size = 10, color = 'red'),\n",
    "        text = red_text,\n",
    "        hoverinfo='text+y',\n",
    "        showlegend = True\n",
    "    )\n",
    "    trace1 = go.Scatter(\n",
    "        x = x_green,\n",
    "        y = green,\n",
    "        mode = 'markers',\n",
    "        name = 'unaffected',\n",
    "        marker = dict(size = 10, color = 'green'),\n",
    "        hovertext = green_text,\n",
    "        hoverinfo='text+y',\n",
    "        showlegend = True\n",
    "    )\n",
    "    fig['data'] = [trace0, trace1]\n",
    "    fig['layout'].update(height=400, width=800, title=t_str,\n",
    "                             xaxis1=dict(tickvals=range(len(xticks)),\n",
    "                                        ticktext=xticks,\n",
    "                                        zeroline = False),\n",
    "                             hovermode='closest')\n",
    "    iplot(fig)\n",
    "    \n",
    "    # print family order\n",
    "    if verbose:\n",
    "        score = []\n",
    "        for x in range(max(x_green + x_red)):\n",
    "            idx = [i for i, val in enumerate(x_red) if val == x]\n",
    "            if len(idx) == 0:\n",
    "                aff = 0\n",
    "            else:\n",
    "                aff = red[idx[0]]\n",
    "            idx = [i for i, val in enumerate(x_green) if val == x]\n",
    "            if len(idx) == 0:\n",
    "                unaff = 0\n",
    "            else:\n",
    "                unaff = max([green[i] for i in idx])\n",
    "            score.append(aff - unaff)\n",
    "        order = np.argsort(score)[::-1]\n",
    "        fam_names = fig.layout.xaxis1['ticktext']\n",
    "        disrupted = [fam_names[v] for v in order if score[v] > 0]\n",
    "        print 'Best disrupted:', ', '.join(disrupted)\n",
    "        protected = [fam_names[v] for v in order[::-1] if score[v] < 0]\n",
    "        print 'Best protected:', ', '.join(protected)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          19,
          13,
          7,
          16,
          0,
          16,
          7,
          9,
          7,
          10,
          8,
          13,
          7,
          166,
          13,
          7,
          13,
          8,
          26,
          9,
          4
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          7,
          8,
          7,
          3,
          8,
          13,
          0,
          12,
          0,
          9,
          5,
          9,
          12,
          5,
          3,
          13,
          10,
          8,
          17,
          11,
          10,
          9,
          3,
          14,
          6,
          5,
          16,
          21,
          38,
          5
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_burden",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"c2d1a105-a3e4-410a-a17a-4de8458692ff\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"c2d1a105-a3e4-410a-a17a-4de8458692ff\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [19, 13, 7, 16, 0, 16, 7, 9, 7, 10, 8, 13, 7, 166, 13, 7, 13, 8, 26, 9, 4], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [7, 8, 7, 3, 8, 13, 0, 12, 0, 9, 5, 9, 12, 5, 3, 13, 10, 8, 17, 11, 10, 9, 3, 14, 6, 5, 16, 21, 38, 5], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_burden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"c2d1a105-a3e4-410a-a17a-4de8458692ff\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"c2d1a105-a3e4-410a-a17a-4de8458692ff\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [19, 13, 7, 16, 0, 16, 7, 9, 7, 10, 8, 13, 7, 166, 13, 7, 13, 8, 26, 9, 4], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [7, 8, 7, 3, 8, 13, 0, 12, 0, 9, 5, 9, 12, 5, 3, 13, 10, 8, 17, 11, 10, 9, 3, 14, 6, 5, 16, 21, 38, 5], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_burden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 10042, 10090, 10173, 10033, 10178, 10164, 1895, 1893, 10128, 10197\n",
      "Best protected: 10094, 10131, 10448, 10041, 855, 1892, 10182, 10215\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          12,
          4,
          4,
          4,
          0,
          3,
          3,
          5,
          0,
          6,
          3,
          10,
          5,
          165,
          6,
          4,
          6,
          6,
          7,
          3,
          4
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          3,
          3,
          2,
          2,
          6,
          7,
          0,
          7,
          0,
          3,
          3,
          5,
          6,
          3,
          2,
          5,
          2,
          5,
          5,
          6,
          4,
          7,
          2,
          3,
          3,
          4,
          10,
          14,
          24,
          1
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_delBurden",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"5e0ca924-2f1a-4300-a3f0-40d9e6e80795\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"5e0ca924-2f1a-4300-a3f0-40d9e6e80795\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [12, 4, 4, 4, 0, 3, 3, 5, 0, 6, 3, 10, 5, 165, 6, 4, 6, 6, 7, 3, 4], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [3, 3, 2, 2, 6, 7, 0, 7, 0, 3, 3, 5, 6, 3, 2, 5, 2, 5, 5, 6, 4, 7, 2, 3, 3, 4, 10, 14, 24, 1], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_delBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"5e0ca924-2f1a-4300-a3f0-40d9e6e80795\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"5e0ca924-2f1a-4300-a3f0-40d9e6e80795\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [12, 4, 4, 4, 0, 3, 3, 5, 0, 6, 3, 10, 5, 165, 6, 4, 6, 6, 7, 3, 4], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [3, 3, 2, 2, 6, 7, 0, 7, 0, 3, 3, 5, 6, 3, 2, 5, 2, 5, 5, 6, 4, 7, 2, 3, 3, 4, 10, 14, 24, 1], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_delBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 10042, 10153, 10173, 10033, 10197, 1893, 10164, 10448, 10178\n",
      "Best protected: 10094, 855, 1895, 10041, 10128, 10090, 10215, 10131\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          7,
          9,
          3,
          12,
          0,
          13,
          4,
          4,
          7,
          4,
          5,
          3,
          2,
          1,
          7,
          3,
          7,
          2,
          19,
          6,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          4,
          5,
          5,
          1,
          2,
          6,
          0,
          5,
          0,
          6,
          2,
          4,
          6,
          2,
          1,
          8,
          8,
          3,
          12,
          5,
          6,
          2,
          1,
          11,
          3,
          1,
          6,
          7,
          14,
          4
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_dupBurden",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"f46a361e-be48-4b8e-8d24-88cd52873e33\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"f46a361e-be48-4b8e-8d24-88cd52873e33\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [7, 9, 3, 12, 0, 13, 4, 4, 7, 4, 5, 3, 2, 1, 7, 3, 7, 2, 19, 6, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [4, 5, 5, 1, 2, 6, 0, 5, 0, 6, 2, 4, 6, 2, 1, 8, 8, 3, 12, 5, 6, 2, 1, 11, 3, 1, 6, 7, 14, 4], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_dupBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"f46a361e-be48-4b8e-8d24-88cd52873e33\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"f46a361e-be48-4b8e-8d24-88cd52873e33\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [7, 9, 3, 12, 0, 13, 4, 4, 7, 4, 5, 3, 2, 1, 7, 3, 7, 2, 19, 6, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [4, 5, 5, 1, 2, 6, 0, 5, 0, 6, 2, 4, 6, 2, 1, 8, 8, 3, 12, 5, 6, 2, 1, 11, 3, 1, 6, 7, 14, 4], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_dupBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10090, 1895, 10042, 10173, 10128, 10033, 10164, 10178, 855, 1893, 10197\n",
      "Best protected: 10131, 10448, 10406, 10153, 10041, 10215, 10182, 1892, 10094\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          18,
          12,
          6,
          11,
          0,
          13,
          5,
          6,
          7,
          9,
          7,
          11,
          6,
          160,
          12,
          6,
          13,
          8,
          24,
          9,
          2
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          6,
          8,
          6,
          3,
          7,
          8,
          0,
          12,
          0,
          8,
          5,
          7,
          10,
          3,
          3,
          11,
          10,
          7,
          16,
          9,
          7,
          4,
          2,
          14,
          5,
          4,
          15,
          17,
          37,
          4
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_genes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"cb4aed60-8304-47dc-be7c-26ae9ecda439\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"cb4aed60-8304-47dc-be7c-26ae9ecda439\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [18, 12, 6, 11, 0, 13, 5, 6, 7, 9, 7, 11, 6, 160, 12, 6, 13, 8, 24, 9, 2], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [6, 8, 6, 3, 7, 8, 0, 12, 0, 8, 5, 7, 10, 3, 3, 11, 10, 7, 16, 9, 7, 4, 2, 14, 5, 4, 15, 17, 37, 4], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_genes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"cb4aed60-8304-47dc-be7c-26ae9ecda439\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"cb4aed60-8304-47dc-be7c-26ae9ecda439\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [18, 12, 6, 11, 0, 13, 5, 6, 7, 9, 7, 11, 6, 160, 12, 6, 13, 8, 24, 9, 2], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [6, 8, 6, 3, 7, 8, 0, 12, 0, 8, 5, 7, 10, 3, 3, 11, 10, 7, 16, 9, 7, 4, 2, 14, 5, 4, 15, 17, 37, 4], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_genes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 10042, 10173, 10090, 10033, 10178, 10197, 10164, 10128, 1893, 1895\n",
      "Best protected: 10131, 10094, 10448, 10041, 1892, 10182, 10215\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          3,
          0,
          0,
          0,
          1,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          1,
          0,
          0,
          0,
          1,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_brainGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"0403f59d-b202-4fc0-af41-f290197fd889\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"0403f59d-b202-4fc0-af41-f290197fd889\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3, 0, 0, 0, 1, 0, 0, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 0, 0, 0, 1, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_brainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"0403f59d-b202-4fc0-af41-f290197fd889\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"0403f59d-b202-4fc0-af41-f290197fd889\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3, 0, 0, 0, 1, 0, 0, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 0, 0, 0, 1, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_brainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 10197\n",
      "Best protected: 10041, 10215, 1895\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          12,
          4,
          3,
          2,
          0,
          3,
          2,
          3,
          0,
          5,
          3,
          9,
          5,
          159,
          5,
          3,
          6,
          6,
          6,
          3,
          2
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          3,
          3,
          2,
          2,
          5,
          4,
          0,
          7,
          0,
          3,
          3,
          5,
          6,
          2,
          2,
          4,
          2,
          4,
          4,
          4,
          3,
          4,
          1,
          3,
          3,
          3,
          9,
          11,
          23,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_delGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"a297beb7-6d1f-4670-ace3-da9faeec9747\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"a297beb7-6d1f-4670-ace3-da9faeec9747\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [12, 4, 3, 2, 0, 3, 2, 3, 0, 5, 3, 9, 5, 159, 5, 3, 6, 6, 6, 3, 2], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [3, 3, 2, 2, 5, 4, 0, 7, 0, 3, 3, 5, 6, 2, 2, 4, 2, 4, 4, 4, 3, 4, 1, 3, 3, 3, 9, 11, 23, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_delGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"a297beb7-6d1f-4670-ace3-da9faeec9747\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"a297beb7-6d1f-4670-ace3-da9faeec9747\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [12, 4, 3, 2, 0, 3, 2, 3, 0, 5, 3, 9, 5, 159, 5, 3, 6, 6, 6, 3, 2], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [3, 3, 2, 2, 5, 4, 0, 7, 0, 3, 3, 5, 6, 2, 2, 4, 2, 4, 4, 4, 3, 4, 1, 3, 3, 3, 9, 11, 23, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_delGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 10042, 10153, 10173, 10197, 10033, 10164, 10131, 10178\n",
      "Best protected: 10094, 855, 1895, 10041, 10090, 10215, 10128, 1892, 10182\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          3,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_delBrainGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"c6600326-06c8-4bcd-a1dd-07e734c14b9e\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"c6600326-06c8-4bcd-a1dd-07e734c14b9e\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3, 0, 0, 0, 0, 0, 0, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 1, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_delBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"c6600326-06c8-4bcd-a1dd-07e734c14b9e\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"c6600326-06c8-4bcd-a1dd-07e734c14b9e\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3, 0, 0, 0, 0, 0, 0, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 1, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_delBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406\n",
      "Best protected: 10215, 1895\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          6,
          8,
          3,
          9,
          0,
          10,
          3,
          3,
          7,
          4,
          4,
          2,
          1,
          1,
          7,
          3,
          7,
          2,
          18,
          6,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          3,
          5,
          4,
          1,
          2,
          4,
          0,
          5,
          0,
          5,
          2,
          2,
          4,
          1,
          1,
          7,
          8,
          3,
          12,
          5,
          4,
          0,
          1,
          11,
          2,
          1,
          6,
          6,
          14,
          4
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_dupGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"53770d82-a09f-403b-b8ca-a2bb318e0eca\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"53770d82-a09f-403b-b8ca-a2bb318e0eca\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [6, 8, 3, 9, 0, 10, 3, 3, 7, 4, 4, 2, 1, 1, 7, 3, 7, 2, 18, 6, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [3, 5, 4, 1, 2, 4, 0, 5, 0, 5, 2, 2, 4, 1, 1, 7, 8, 3, 12, 5, 4, 0, 1, 11, 2, 1, 6, 6, 14, 4], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_dupGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"53770d82-a09f-403b-b8ca-a2bb318e0eca\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"53770d82-a09f-403b-b8ca-a2bb318e0eca\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [6, 8, 3, 9, 0, 10, 3, 3, 7, 4, 4, 2, 1, 1, 7, 3, 7, 2, 18, 6, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [3, 5, 4, 1, 2, 4, 0, 5, 0, 5, 2, 2, 4, 1, 1, 7, 8, 3, 12, 5, 4, 0, 1, 11, 2, 1, 6, 6, 14, 4], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_dupGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10090, 10042, 10173, 855, 10033, 10128, 1895, 10164, 10178, 10197, 1893\n",
      "Best protected: 10131, 10448, 10153, 10041, 10406, 1892, 10215\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_dupBrainGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"80a6536d-1126-47b9-a510-b576b5d8b5e8\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"80a6536d-1126-47b9-a510-b576b5d8b5e8\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_dupBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"80a6536d-1126-47b9-a510-b576b5d8b5e8\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"80a6536d-1126-47b9-a510-b576b5d8b5e8\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_dupBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10197\n",
      "Best protected: 10041\n"
     ]
    }
   ],
   "source": [
    "import glob\n",
    "\n",
    "data_dir = '/home/sudregp/data/cnv/xhmm/'\n",
    "files = glob.glob(data_dir + '/denovo*cnv.indiv')\n",
    "for f in files:\n",
    "    t_str = '.'.join(f.split('/')[-1].split('.')[:-2])\n",
    "    plot_plink_cnvs(f, t_str, verbose=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "It's interesting that the overall results here differ from what we get from PennCNV, in terms of the most affected families. Also, the results where we don't clean up based on gene sets (and even those, to a certain extent), could certainly use some cleaning. Let's start with just the most common CNVs, for now."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean.log ]\n",
      "Analysis started: Thu Dec 21 15:31:31 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-exclude bad_regions.list\n",
      "\t--cnv-overlap .5\n",
      "\t--cnv-write\n",
      "\t--out all_clean\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading CNV intersection list from [ bad_regions.list ]\n",
      "Read 380994 ranges to exclude from CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "7223 mapped to a person, of which 7223 passed filters\n",
      "191 kept after excluding specific regions\n",
      "191 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      87 / 22\n",
      "     3      29 / 53\n",
      "\n",
      "Writing new CNV list to [ all_clean.cnv ]\n",
      "Writing new FAM file to [ all_clean.fam ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:32:49 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean.log ]\n",
      "Analysis started: Thu Dec 21 15:32:49 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--out all_clean\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "Writing new MAP file to [ all_clean.cnv.map ]\n",
      "Wrote 463 unique positions to file\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:32:49 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean.log ]\n",
      "Analysis started: Thu Dec 21 15:32:49 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-exclude bad_regions.list\n",
      "\t--cnv-overlap .5\n",
      "\t--cnv-write\n",
      "\t--out denovo_clean\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading CNV intersection list from [ bad_regions.list ]\n",
      "Read 380994 ranges to exclude from CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "665 mapped to a person, of which 665 passed filters\n",
      "51 kept after excluding specific regions\n",
      "51 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1       39 / 2\n",
      "     3        8 / 2\n",
      "\n",
      "Writing new CNV list to [ denovo_clean.cnv ]\n",
      "Writing new FAM file to [ denovo_clean.fam ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:32:58 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean.log ]\n",
      "Analysis started: Thu Dec 21 15:32:58 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--out denovo_clean\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "Writing new MAP file to [ denovo_clean.cnv.map ]\n",
      "Wrote 147 unique positions to file\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:32:58 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean.log ]\n",
      "Analysis started: Thu Dec 21 15:32:58 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-exclude bad_regions.list\n",
      "\t--cnv-overlap .5\n",
      "\t--cnv-write\n",
      "\t--out inherited_clean\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading CNV intersection list from [ bad_regions.list ]\n",
      "Read 380994 ranges to exclude from CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "401 mapped to a person, of which 401 passed filters\n",
      "4 kept after excluding specific regions\n",
      "4 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        2 / 1\n",
      "     3        1 / 0\n",
      "\n",
      "Writing new CNV list to [ inherited_clean.cnv ]\n",
      "Writing new FAM file to [ inherited_clean.fam ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:33:05 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean.log ]\n",
      "Analysis started: Thu Dec 21 15:33:05 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--out inherited_clean\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "Writing new MAP file to [ inherited_clean.cnv.map ]\n",
      "Wrote 12 unique positions to file\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:33:05 2017\n",
      "\n"
     ]
    }
   ],
   "source": [
    "cd ~/data/cnv/xhmm\n",
    "module load plink/1.07\n",
    "\n",
    "cat ../penncnv/wellknown_region_hg19 > bad_regions.list\n",
    "\n",
    "for cnvtype in all denovo inherited; do\n",
    "    cnvname=${cnvtype}.cnv\n",
    "\n",
    "    # remove bad regions\n",
    "    plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "        --noweb --1 --cnv-exclude bad_regions.list --cnv-overlap .5 \\\n",
    "        --cnv-write --out ${cnvtype}_clean\n",
    "    plink --cnv-list ${cnvtype}_clean.cnv --cnv-make-map --noweb --1 \\\n",
    "        --out ${cnvtype}_clean\n",
    "done"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "I'm going to crop based on CNV length just at 500 for now, but I still need to histogram them to see what're the best values."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT0_burden.log ]\n",
      "Analysis started: Thu Dec 21 15:39:41 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--cnv-kb 0\n",
      "\t--out all_lenBT0_burden\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "No overlapping samples found\n",
      "7223 mapped to a person, of which 7223 passed filters\n",
      "7223 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1  1761 / 1811\n",
      "     3  1585 / 2066\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT0_burden.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT0_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:39:41 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT0_genes.log ]\n",
      "Analysis started: Thu Dec 21 15:39:41 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-kb 0\n",
      "\t--out all_lenBT0_genes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_lenBT0_genes.reg ]\n",
      "7223 mapped to a person, of which 7223 passed filters\n",
      "6192 intersected with one or more specified region\n",
      "6192 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1  1527 / 1535\n",
      "     3  1342 / 1788\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT0_genes.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT0_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:39:48 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT0_brainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:39:48 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-kb 0\n",
      "\t--out all_lenBT0_brainGenes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_lenBT0_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_lenBT0_brainGenes.reg ]\n",
      "7223 mapped to a person, of which 7223 passed filters\n",
      "57 intersected with one or more specified region\n",
      "57 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      14 / 10\n",
      "     3      14 / 19\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT0_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT0_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:39:49 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT0_delBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:39:49 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out all_lenBT0_delBurden\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "7223 mapped to a person, of which 3572 passed filters\n",
      "3572 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1  1761 / 1811\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT0_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT0_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:39:49 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT0_delGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:39:49 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out all_lenBT0_delGenes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_lenBT0_delGenes.reg ]\n",
      "7223 mapped to a person, of which 3572 passed filters\n",
      "3062 intersected with one or more specified region\n",
      "3062 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1  1527 / 1535\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT0_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT0_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:39:53 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT0_delBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:39:53 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out all_lenBT0_delBrainGenes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_lenBT0_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_lenBT0_delBrainGenes.reg ]\n",
      "7223 mapped to a person, of which 3572 passed filters\n",
      "24 intersected with one or more specified region\n",
      "24 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      14 / 10\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT0_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT0_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:39:53 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT0_dupBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:39:53 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 0\n",
      "\t--out all_lenBT0_dupBurden\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "7223 mapped to a person, of which 3651 passed filters\n",
      "3651 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3  1585 / 2066\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT0_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT0_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:39:53 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT0_dupGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:39:53 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 0\n",
      "\t--out all_lenBT0_dupGenes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_lenBT0_dupGenes.reg ]\n",
      "7223 mapped to a person, of which 3651 passed filters\n",
      "3130 intersected with one or more specified region\n",
      "3130 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3  1342 / 1788\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT0_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT0_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:39:57 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT0_dupBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:39:57 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 0\n",
      "\t--out all_lenBT0_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_lenBT0_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_lenBT0_dupBrainGenes.reg ]\n",
      "7223 mapped to a person, of which 3651 passed filters\n",
      "33 intersected with one or more specified region\n",
      "33 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      14 / 19\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT0_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT0_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:39:58 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT500_burden.log ]\n",
      "Analysis started: Thu Dec 21 15:39:58 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--cnv-kb 500\n",
      "\t--out all_lenBT500_burden\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "No overlapping samples found\n",
      "7223 mapped to a person, of which 69 passed filters\n",
      "69 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      15 / 22\n",
      "     3      13 / 19\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT500_burden.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT500_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:39:58 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT500_genes.log ]\n",
      "Analysis started: Thu Dec 21 15:39:58 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-kb 500\n",
      "\t--out all_lenBT500_genes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_lenBT500_genes.reg ]\n",
      "7223 mapped to a person, of which 69 passed filters\n",
      "68 intersected with one or more specified region\n",
      "68 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      14 / 22\n",
      "     3      13 / 19\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT500_genes.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT500_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:39:59 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT500_brainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:39:59 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-kb 500\n",
      "\t--out all_lenBT500_brainGenes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_lenBT500_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_lenBT500_brainGenes.reg ]\n",
      "7223 mapped to a person, of which 69 passed filters\n",
      "2 intersected with one or more specified region\n",
      "2 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        2 / 0\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT500_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT500_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:39:59 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT500_delBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:39:59 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out all_lenBT500_delBurden\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "7223 mapped to a person, of which 37 passed filters\n",
      "37 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      15 / 22\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT500_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT500_delBurden.cnv.summary ]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Analysis finished: Thu Dec 21 15:39:59 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT500_delGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:39:59 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out all_lenBT500_delGenes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_lenBT500_delGenes.reg ]\n",
      "7223 mapped to a person, of which 37 passed filters\n",
      "36 intersected with one or more specified region\n",
      "36 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      14 / 22\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT500_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT500_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:00 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT500_delBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:00 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out all_lenBT500_delBrainGenes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_lenBT500_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_lenBT500_delBrainGenes.reg ]\n",
      "7223 mapped to a person, of which 37 passed filters\n",
      "2 intersected with one or more specified region\n",
      "2 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        2 / 0\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT500_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT500_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:00 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT500_dupBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:00 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out all_lenBT500_dupBurden\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "7223 mapped to a person, of which 32 passed filters\n",
      "32 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      13 / 19\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT500_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT500_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:00 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT500_dupGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:00 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out all_lenBT500_dupGenes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_lenBT500_dupGenes.reg ]\n",
      "7223 mapped to a person, of which 32 passed filters\n",
      "32 intersected with one or more specified region\n",
      "32 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      13 / 19\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT500_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT500_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:01 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_lenBT500_dupBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:01 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out all_lenBT500_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ all.cnv.map ]\n",
      "10382 (of 10382) markers to be included from [ all.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_lenBT500_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all.cnv ]\n",
      "Writing back list to intersected regions to [ all_lenBT500_dupBrainGenes.reg ]\n",
      "7223 mapped to a person, of which 32 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 7223 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ all_lenBT500_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_lenBT500_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:01 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT0_burden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:01 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--cnv-kb 0\n",
      "\t--out all_clean_lenBT0_burden\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "No overlapping samples found\n",
      "191 mapped to a person, of which 191 passed filters\n",
      "191 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      87 / 22\n",
      "     3      29 / 53\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT0_burden.cnv.indiv ]\n",
      "Writing positional summary to [ all_clean_lenBT0_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:01 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT0_genes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:01 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-kb 0\n",
      "\t--out all_clean_lenBT0_genes\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_clean_lenBT0_genes.reg ]\n",
      "191 mapped to a person, of which 191 passed filters\n",
      "183 intersected with one or more specified region\n",
      "183 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      83 / 22\n",
      "     3      27 / 51\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT0_genes.cnv.indiv ]\n",
      "Writing positional summary to [ all_clean_lenBT0_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:02 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT0_brainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:02 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-kb 0\n",
      "\t--out all_clean_lenBT0_brainGenes\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_clean_lenBT0_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_clean_lenBT0_brainGenes.reg ]\n",
      "191 mapped to a person, of which 191 passed filters\n",
      "5 intersected with one or more specified region\n",
      "5 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        3 / 1\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT0_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_clean_lenBT0_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:02 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT0_delBurden.log ]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Analysis started: Thu Dec 21 15:40:02 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out all_clean_lenBT0_delBurden\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "191 mapped to a person, of which 109 passed filters\n",
      "109 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      87 / 22\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT0_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ all_clean_lenBT0_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:02 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT0_delGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:02 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out all_clean_lenBT0_delGenes\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_clean_lenBT0_delGenes.reg ]\n",
      "191 mapped to a person, of which 109 passed filters\n",
      "105 intersected with one or more specified region\n",
      "105 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      83 / 22\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT0_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_clean_lenBT0_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:02 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT0_delBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:02 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out all_clean_lenBT0_delBrainGenes\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_clean_lenBT0_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_clean_lenBT0_delBrainGenes.reg ]\n",
      "191 mapped to a person, of which 109 passed filters\n",
      "4 intersected with one or more specified region\n",
      "4 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        3 / 1\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT0_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_clean_lenBT0_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:02 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT0_dupBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:02 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 0\n",
      "\t--out all_clean_lenBT0_dupBurden\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "191 mapped to a person, of which 82 passed filters\n",
      "82 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      29 / 53\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT0_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ all_clean_lenBT0_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:02 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT0_dupGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:02 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 0\n",
      "\t--out all_clean_lenBT0_dupGenes\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_clean_lenBT0_dupGenes.reg ]\n",
      "191 mapped to a person, of which 82 passed filters\n",
      "78 intersected with one or more specified region\n",
      "78 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      27 / 51\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT0_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_clean_lenBT0_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:03 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT0_dupBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:03 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 0\n",
      "\t--out all_clean_lenBT0_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_clean_lenBT0_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_clean_lenBT0_dupBrainGenes.reg ]\n",
      "191 mapped to a person, of which 82 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT0_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_clean_lenBT0_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:03 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT500_burden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:03 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--cnv-kb 500\n",
      "\t--out all_clean_lenBT500_burden\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "No overlapping samples found\n",
      "191 mapped to a person, of which 11 passed filters\n",
      "11 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        6 / 4\n",
      "     3        0 / 1\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT500_burden.cnv.indiv ]\n",
      "Writing positional summary to [ all_clean_lenBT500_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:03 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT500_genes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:03 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-kb 500\n",
      "\t--out all_clean_lenBT500_genes\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_clean_lenBT500_genes.reg ]\n",
      "191 mapped to a person, of which 11 passed filters\n",
      "11 intersected with one or more specified region\n",
      "11 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        6 / 4\n",
      "     3        0 / 1\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT500_genes.cnv.indiv ]\n",
      "Writing positional summary to [ all_clean_lenBT500_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:03 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT500_brainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:03 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-kb 500\n",
      "\t--out all_clean_lenBT500_brainGenes\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_clean_lenBT500_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_clean_lenBT500_brainGenes.reg ]\n",
      "191 mapped to a person, of which 11 passed filters\n",
      "2 intersected with one or more specified region\n",
      "2 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        2 / 0\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT500_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_clean_lenBT500_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:03 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT500_delBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:03 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out all_clean_lenBT500_delBurden\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "191 mapped to a person, of which 10 passed filters\n",
      "10 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        6 / 4\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT500_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ all_clean_lenBT500_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:03 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT500_delGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:03 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out all_clean_lenBT500_delGenes\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_clean_lenBT500_delGenes.reg ]\n",
      "191 mapped to a person, of which 10 passed filters\n",
      "10 intersected with one or more specified region\n",
      "10 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        6 / 4\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT500_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_clean_lenBT500_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:04 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT500_delBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:04 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out all_clean_lenBT500_delBrainGenes\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_clean_lenBT500_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_clean_lenBT500_delBrainGenes.reg ]\n",
      "191 mapped to a person, of which 10 passed filters\n",
      "2 intersected with one or more specified region\n",
      "2 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        2 / 0\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT500_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_clean_lenBT500_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:04 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT500_dupBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:04 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out all_clean_lenBT500_dupBurden\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "191 mapped to a person, of which 1 passed filters\n",
      "1 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        0 / 1\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT500_dupBurden.cnv.indiv ]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Writing positional summary to [ all_clean_lenBT500_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:04 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT500_dupGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:04 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out all_clean_lenBT500_dupGenes\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_clean_lenBT500_dupGenes.reg ]\n",
      "191 mapped to a person, of which 1 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        0 / 1\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT500_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_clean_lenBT500_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:04 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_clean_lenBT500_dupBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:04 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out all_clean_lenBT500_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ all_clean.cnv.map ]\n",
      "463 (of 463) markers to be included from [ all_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_clean_lenBT500_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_clean_lenBT500_dupBrainGenes.reg ]\n",
      "191 mapped to a person, of which 1 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 191 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ all_clean_lenBT500_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_clean_lenBT500_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:04 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT0_burden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:04 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_lenBT0_burden\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "No overlapping samples found\n",
      "665 mapped to a person, of which 665 passed filters\n",
      "665 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    260 / 147\n",
      "     3    118 / 140\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT0_burden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT0_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:04 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT0_genes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:04 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_lenBT0_genes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_lenBT0_genes.reg ]\n",
      "665 mapped to a person, of which 665 passed filters\n",
      "593 intersected with one or more specified region\n",
      "593 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    241 / 125\n",
      "     3    104 / 123\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT0_genes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT0_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:05 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT0_brainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:05 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_lenBT0_brainGenes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_lenBT0_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_lenBT0_brainGenes.reg ]\n",
      "665 mapped to a person, of which 665 passed filters\n",
      "8 intersected with one or more specified region\n",
      "8 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        3 / 3\n",
      "     3        1 / 1\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT0_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT0_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:05 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT0_delBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:05 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_lenBT0_delBurden\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "665 mapped to a person, of which 407 passed filters\n",
      "407 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    260 / 147\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT0_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT0_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:05 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT0_delGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:05 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_lenBT0_delGenes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_lenBT0_delGenes.reg ]\n",
      "665 mapped to a person, of which 407 passed filters\n",
      "366 intersected with one or more specified region\n",
      "366 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    241 / 125\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT0_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT0_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:06 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT0_delBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:06 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_lenBT0_delBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_lenBT0_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_lenBT0_delBrainGenes.reg ]\n",
      "665 mapped to a person, of which 407 passed filters\n",
      "6 intersected with one or more specified region\n",
      "6 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        3 / 3\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT0_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT0_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:06 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT0_dupBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:06 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_lenBT0_dupBurden\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "665 mapped to a person, of which 258 passed filters\n",
      "258 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3    118 / 140\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT0_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT0_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:06 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT0_dupGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:06 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_lenBT0_dupGenes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_lenBT0_dupGenes.reg ]\n",
      "665 mapped to a person, of which 258 passed filters\n",
      "227 intersected with one or more specified region\n",
      "227 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3    104 / 123\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT0_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT0_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:07 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT0_dupBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:07 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_lenBT0_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_lenBT0_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_lenBT0_dupBrainGenes.reg ]\n",
      "665 mapped to a person, of which 258 passed filters\n",
      "2 intersected with one or more specified region\n",
      "2 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 1\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT0_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT0_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:07 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT500_burden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:07 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_lenBT500_burden\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "No overlapping samples found\n",
      "665 mapped to a person, of which 1 passed filters\n",
      "1 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT500_burden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT500_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:07 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT500_genes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:07 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_lenBT500_genes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_lenBT500_genes.reg ]\n",
      "665 mapped to a person, of which 1 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT500_genes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT500_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:07 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT500_brainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:07 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_lenBT500_brainGenes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_lenBT500_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_lenBT500_brainGenes.reg ]\n",
      "665 mapped to a person, of which 1 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT500_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT500_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:07 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT500_delBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:07 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_lenBT500_delBurden\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "665 mapped to a person, of which 1 passed filters\n",
      "1 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT500_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT500_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:07 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT500_delGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:07 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_lenBT500_delGenes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_lenBT500_delGenes.reg ]\n",
      "665 mapped to a person, of which 1 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT500_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT500_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:08 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT500_delBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:08 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_lenBT500_delBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_lenBT500_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_lenBT500_delBrainGenes.reg ]\n",
      "665 mapped to a person, of which 1 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT500_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT500_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:08 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT500_dupBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:08 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_lenBT500_dupBurden\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "665 mapped to a person, of which 0 passed filters\n",
      "0 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT500_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT500_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:08 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT500_dupGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:08 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_lenBT500_dupGenes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_lenBT500_dupGenes.reg ]\n",
      "665 mapped to a person, of which 0 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT500_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT500_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:08 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_lenBT500_dupBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:08 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_lenBT500_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo.cnv.map ]\n",
      "1757 (of 1757) markers to be included from [ denovo.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_lenBT500_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_lenBT500_dupBrainGenes.reg ]\n",
      "665 mapped to a person, of which 0 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 665 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_lenBT500_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_lenBT500_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:08 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT0_burden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:08 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_clean_lenBT0_burden\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "No overlapping samples found\n",
      "51 mapped to a person, of which 51 passed filters\n",
      "51 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1       39 / 2\n",
      "     3        8 / 2\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT0_burden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT0_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:08 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT0_genes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:08 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_clean_lenBT0_genes\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_clean_lenBT0_genes.reg ]\n",
      "51 mapped to a person, of which 51 passed filters\n",
      "49 intersected with one or more specified region\n",
      "49 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1       37 / 2\n",
      "     3        8 / 2\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT0_genes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT0_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:09 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT0_brainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:09 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_clean_lenBT0_brainGenes\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_clean_lenBT0_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_clean_lenBT0_brainGenes.reg ]\n",
      "51 mapped to a person, of which 51 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT0_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT0_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:09 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT0_delBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:09 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_clean_lenBT0_delBurden\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "51 mapped to a person, of which 41 passed filters\n",
      "41 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1       39 / 2\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT0_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT0_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:09 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT0_delGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:09 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_clean_lenBT0_delGenes\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_clean_lenBT0_delGenes.reg ]\n",
      "51 mapped to a person, of which 41 passed filters\n",
      "39 intersected with one or more specified region\n",
      "39 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1       37 / 2\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT0_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT0_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:09 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT0_delBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:09 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_clean_lenBT0_delBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_clean_lenBT0_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_clean_lenBT0_delBrainGenes.reg ]\n",
      "51 mapped to a person, of which 41 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT0_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT0_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:09 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT0_dupBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:09 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_clean_lenBT0_dupBurden\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "51 mapped to a person, of which 10 passed filters\n",
      "10 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        8 / 2\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT0_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT0_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:09 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT0_dupGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:09 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_clean_lenBT0_dupGenes\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_clean_lenBT0_dupGenes.reg ]\n",
      "51 mapped to a person, of which 10 passed filters\n",
      "10 intersected with one or more specified region\n",
      "10 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        8 / 2\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT0_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT0_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:09 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT0_dupBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:09 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 0\n",
      "\t--out denovo_clean_lenBT0_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_clean_lenBT0_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_clean_lenBT0_dupBrainGenes.reg ]\n",
      "51 mapped to a person, of which 10 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT0_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT0_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:10 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT500_burden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:10 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_clean_lenBT500_burden\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "No overlapping samples found\n",
      "51 mapped to a person, of which 1 passed filters\n",
      "1 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT500_burden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT500_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:10 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT500_genes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:10 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_clean_lenBT500_genes\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_clean_lenBT500_genes.reg ]\n",
      "51 mapped to a person, of which 1 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT500_genes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT500_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:10 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT500_brainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:10 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_clean_lenBT500_brainGenes\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_clean_lenBT500_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_clean_lenBT500_brainGenes.reg ]\n",
      "51 mapped to a person, of which 1 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT500_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT500_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:10 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT500_delBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:10 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_clean_lenBT500_delBurden\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "51 mapped to a person, of which 1 passed filters\n",
      "1 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT500_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT500_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:10 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT500_delGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:10 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_clean_lenBT500_delGenes\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_clean_lenBT500_delGenes.reg ]\n",
      "51 mapped to a person, of which 1 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT500_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT500_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:10 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT500_delBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:10 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_clean_lenBT500_delBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_clean_lenBT500_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_clean_lenBT500_delBrainGenes.reg ]\n",
      "51 mapped to a person, of which 1 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT500_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT500_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:10 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT500_dupBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:10 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_clean_lenBT500_dupBurden\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "51 mapped to a person, of which 0 passed filters\n",
      "0 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT500_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT500_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:10 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT500_dupGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:10 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_clean_lenBT500_dupGenes\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_clean_lenBT500_dupGenes.reg ]\n",
      "51 mapped to a person, of which 0 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT500_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT500_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:11 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_clean_lenBT500_dupBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:11 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out denovo_clean_lenBT500_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo_clean.cnv.map ]\n",
      "147 (of 147) markers to be included from [ denovo_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_clean_lenBT500_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_clean_lenBT500_dupBrainGenes.reg ]\n",
      "51 mapped to a person, of which 0 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_clean_lenBT500_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_clean_lenBT500_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:11 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT0_burden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:11 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--cnv-kb 0\n",
      "\t--out inherited_lenBT0_burden\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "No overlapping samples found\n",
      "401 mapped to a person, of which 401 passed filters\n",
      "401 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1     95 / 119\n",
      "     3      88 / 99\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT0_burden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT0_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:11 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT0_genes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:11 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-kb 0\n",
      "\t--out inherited_lenBT0_genes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_lenBT0_genes.reg ]\n",
      "401 mapped to a person, of which 401 passed filters\n",
      "322 intersected with one or more specified region\n",
      "322 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      79 / 97\n",
      "     3      69 / 77\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT0_genes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT0_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:11 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT0_brainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:11 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-kb 0\n",
      "\t--out inherited_lenBT0_brainGenes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_lenBT0_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_lenBT0_brainGenes.reg ]\n",
      "401 mapped to a person, of which 401 passed filters\n",
      "2 intersected with one or more specified region\n",
      "2 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 1\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT0_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT0_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:11 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT0_delBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:11 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out inherited_lenBT0_delBurden\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "401 mapped to a person, of which 214 passed filters\n",
      "214 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1     95 / 119\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT0_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT0_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:12 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT0_delGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:12 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out inherited_lenBT0_delGenes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_lenBT0_delGenes.reg ]\n",
      "401 mapped to a person, of which 214 passed filters\n",
      "176 intersected with one or more specified region\n",
      "176 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      79 / 97\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT0_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT0_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:12 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT0_delBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:12 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out inherited_lenBT0_delBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_lenBT0_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_lenBT0_delBrainGenes.reg ]\n",
      "401 mapped to a person, of which 214 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT0_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT0_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:12 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT0_dupBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:12 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 0\n",
      "\t--out inherited_lenBT0_dupBurden\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "401 mapped to a person, of which 187 passed filters\n",
      "187 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      88 / 99\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT0_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT0_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:12 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT0_dupGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:12 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 0\n",
      "\t--out inherited_lenBT0_dupGenes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_lenBT0_dupGenes.reg ]\n",
      "401 mapped to a person, of which 187 passed filters\n",
      "146 intersected with one or more specified region\n",
      "146 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      69 / 77\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT0_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT0_dupGenes.cnv.summary ]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Analysis finished: Thu Dec 21 15:40:13 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT0_dupBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:13 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 0\n",
      "\t--out inherited_lenBT0_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_lenBT0_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_lenBT0_dupBrainGenes.reg ]\n",
      "401 mapped to a person, of which 187 passed filters\n",
      "2 intersected with one or more specified region\n",
      "2 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 1\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT0_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT0_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:13 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT500_burden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:13 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_lenBT500_burden\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "No overlapping samples found\n",
      "401 mapped to a person, of which 3 passed filters\n",
      "3 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 1\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT500_burden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT500_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:13 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT500_genes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:13 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_lenBT500_genes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_lenBT500_genes.reg ]\n",
      "401 mapped to a person, of which 3 passed filters\n",
      "3 intersected with one or more specified region\n",
      "3 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 1\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT500_genes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT500_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:13 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT500_brainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:13 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_lenBT500_brainGenes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_lenBT500_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_lenBT500_brainGenes.reg ]\n",
      "401 mapped to a person, of which 3 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT500_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT500_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:13 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT500_delBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:13 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_lenBT500_delBurden\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "401 mapped to a person, of which 2 passed filters\n",
      "2 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 1\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT500_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT500_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:13 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT500_delGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:13 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_lenBT500_delGenes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_lenBT500_delGenes.reg ]\n",
      "401 mapped to a person, of which 2 passed filters\n",
      "2 intersected with one or more specified region\n",
      "2 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 1\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT500_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT500_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:13 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT500_delBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:14 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_lenBT500_delBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_lenBT500_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_lenBT500_delBrainGenes.reg ]\n",
      "401 mapped to a person, of which 2 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT500_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT500_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:14 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT500_dupBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:14 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_lenBT500_dupBurden\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "401 mapped to a person, of which 1 passed filters\n",
      "1 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT500_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT500_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:14 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT500_dupGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:14 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_lenBT500_dupGenes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_lenBT500_dupGenes.reg ]\n",
      "401 mapped to a person, of which 1 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT500_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT500_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:14 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_lenBT500_dupBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:14 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_lenBT500_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited.cnv.map ]\n",
      "807 (of 807) markers to be included from [ inherited.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_lenBT500_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_lenBT500_dupBrainGenes.reg ]\n",
      "401 mapped to a person, of which 1 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 401 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_lenBT500_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_lenBT500_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:14 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT0_burden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:14 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--cnv-kb 0\n",
      "\t--out inherited_clean_lenBT0_burden\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "No overlapping samples found\n",
      "4 mapped to a person, of which 4 passed filters\n",
      "4 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        2 / 1\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT0_burden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT0_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:14 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT0_genes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:14 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-kb 0\n",
      "\t--out inherited_clean_lenBT0_genes\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_clean_lenBT0_genes.reg ]\n",
      "4 mapped to a person, of which 4 passed filters\n",
      "3 intersected with one or more specified region\n",
      "3 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 1\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT0_genes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT0_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:14 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT0_brainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:14 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-kb 0\n",
      "\t--out inherited_clean_lenBT0_brainGenes\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_clean_lenBT0_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_clean_lenBT0_brainGenes.reg ]\n",
      "4 mapped to a person, of which 4 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT0_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT0_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:14 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT0_delBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:14 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out inherited_clean_lenBT0_delBurden\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "4 mapped to a person, of which 3 passed filters\n",
      "3 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        2 / 1\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT0_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT0_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:14 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT0_delGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:14 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out inherited_clean_lenBT0_delGenes\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_clean_lenBT0_delGenes.reg ]\n",
      "4 mapped to a person, of which 3 passed filters\n",
      "2 intersected with one or more specified region\n",
      "2 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 1\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT0_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT0_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:15 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT0_delBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:15 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 0\n",
      "\t--out inherited_clean_lenBT0_delBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_clean_lenBT0_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_clean_lenBT0_delBrainGenes.reg ]\n",
      "4 mapped to a person, of which 3 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT0_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT0_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:15 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT0_dupBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:15 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\t--cnv-kb 0\n",
      "\t--out inherited_clean_lenBT0_dupBurden\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "4 mapped to a person, of which 1 passed filters\n",
      "1 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT0_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT0_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:15 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT0_dupGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:15 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 0\n",
      "\t--out inherited_clean_lenBT0_dupGenes\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_clean_lenBT0_dupGenes.reg ]\n",
      "4 mapped to a person, of which 1 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT0_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT0_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:15 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT0_dupBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:15 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 0\n",
      "\t--out inherited_clean_lenBT0_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_clean_lenBT0_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_clean_lenBT0_dupBrainGenes.reg ]\n",
      "4 mapped to a person, of which 1 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT0_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT0_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:15 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT500_burden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:15 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_clean_lenBT500_burden\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "No overlapping samples found\n",
      "4 mapped to a person, of which 1 passed filters\n",
      "1 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        0 / 1\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT500_burden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT500_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:15 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT500_genes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:15 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_clean_lenBT500_genes\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_clean_lenBT500_genes.reg ]\n",
      "4 mapped to a person, of which 1 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        0 / 1\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT500_genes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT500_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:15 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT500_brainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:15 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_clean_lenBT500_brainGenes\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_clean_lenBT500_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_clean_lenBT500_brainGenes.reg ]\n",
      "4 mapped to a person, of which 1 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT500_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT500_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:16 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT500_delBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:16 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_clean_lenBT500_delBurden\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "4 mapped to a person, of which 1 passed filters\n",
      "1 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        0 / 1\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT500_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT500_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:16 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT500_delGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:16 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_clean_lenBT500_delGenes\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_clean_lenBT500_delGenes.reg ]\n",
      "4 mapped to a person, of which 1 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        0 / 1\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT500_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT500_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:16 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT500_delBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:16 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_clean_lenBT500_delBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_clean_lenBT500_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_clean_lenBT500_delBrainGenes.reg ]\n",
      "4 mapped to a person, of which 1 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT500_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT500_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:16 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT500_dupBurden.log ]\n",
      "Analysis started: Thu Dec 21 15:40:16 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_clean_lenBT500_dupBurden\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "4 mapped to a person, of which 0 passed filters\n",
      "0 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT500_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT500_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:16 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT500_dupGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:16 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_clean_lenBT500_dupGenes\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_clean_lenBT500_dupGenes.reg ]\n",
      "4 mapped to a person, of which 0 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT500_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT500_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:16 2017\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_clean_lenBT500_dupBrainGenes.log ]\n",
      "Analysis started: Thu Dec 21 15:40:16 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--cnv-kb 500\n",
      "\t--out inherited_clean_lenBT500_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited_clean.cnv.map ]\n",
      "12 (of 12) markers to be included from [ inherited_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_clean_lenBT500_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_clean_lenBT500_dupBrainGenes.reg ]\n",
      "4 mapped to a person, of which 0 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 4 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_clean_lenBT500_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_clean_lenBT500_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Thu Dec 21 15:40:16 2017\n",
      "\n"
     ]
    }
   ],
   "source": [
    "cd ~/data/cnv/xhmm\n",
    "\n",
    "for cnvtype in all denovo inherited; do\n",
    "    for qc in '' '_clean'; do\n",
    "        cnvname=${cnvtype}${qc}.cnv\n",
    "        # apply CNV length threshold in clean and nonclean sets\n",
    "        for kb in 0 500; do\n",
    "            # whole burden\n",
    "            plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "                --noweb --1 --cnv-check-no-overlap --cnv-kb $kb \\\n",
    "                --out ${cnvtype}${qc}_lenBT${kb}_burden;\n",
    "            # gene sets\n",
    "            plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "                --noweb --1 --cnv-intersect glist-hg19 --cnv-verbose-report-regions \\\n",
    "                --cnv-subset genes.txt --cnv-kb $kb \\\n",
    "                --out ${cnvtype}${qc}_lenBT${kb}_genes;\n",
    "            plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "                --noweb --1 --cnv-intersect glist-hg19 --cnv-verbose-report-regions \\\n",
    "                --cnv-subset hg19_allenBrainGene_trimmed.txt --cnv-kb $kb \\\n",
    "                --out ${cnvtype}${qc}_lenBT${kb}_brainGenes;\n",
    "            # subtypes only\n",
    "            for sub in del dup; do\n",
    "                plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "                --noweb --1 --cnv-${sub} --cnv-kb $kb \\\n",
    "                --out ${cnvtype}${qc}_lenBT${kb}_${sub}Burden;\n",
    "                # gene sets\n",
    "                plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "                    --noweb --1 --cnv-intersect glist-hg19 --cnv-verbose-report-regions \\\n",
    "                    --cnv-subset genes.txt --cnv-${sub} --cnv-kb $kb\\\n",
    "                    --out ${cnvtype}${qc}_lenBT${kb}_${sub}Genes;\n",
    "                plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "                    --noweb --1 --cnv-intersect glist-hg19 --cnv-verbose-report-regions \\\n",
    "                    --cnv-subset hg19_allenBrainGene_trimmed.txt --cnv-${sub} \\\n",
    "                    --cnv-kb $kb \\\n",
    "                    --out ${cnvtype}${qc}_lenBT${kb}_${sub}BrainGenes;\n",
    "            done;\n",
    "        done;\n",
    "    done;\n",
    "done"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Let's look at the overall effect of cleaning:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          0,
          4,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          36,
          0,
          1,
          0,
          0,
          4,
          0,
          2
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          1,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_clean_lenBT0_burden",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"cdc54b0c-921e-4310-9d14-f4b97c83fefd\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"cdc54b0c-921e-4310-9d14-f4b97c83fefd\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 36, 0, 1, 0, 0, 4, 0, 2], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_burden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"cdc54b0c-921e-4310-9d14-f4b97c83fefd\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"cdc54b0c-921e-4310-9d14-f4b97c83fefd\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 36, 0, 1, 0, 0, 4, 0, 2], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_burden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 10090, 10164\n",
      "Best protected: 10131, 1895\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          0,
          4,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          35,
          0,
          1,
          0,
          0,
          4,
          0,
          1
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          1,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_clean_lenBT0_genes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"dbb1cf83-de54-47e8-8e60-b29da44cdde0\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"dbb1cf83-de54-47e8-8e60-b29da44cdde0\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 35, 0, 1, 0, 0, 4, 0, 1], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_genes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"dbb1cf83-de54-47e8-8e60-b29da44cdde0\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"dbb1cf83-de54-47e8-8e60-b29da44cdde0\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 35, 0, 1, 0, 0, 4, 0, 1], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_genes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 10090, 10164\n",
      "Best protected: 10131, 1895\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_clean_lenBT0_brainGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"126d2c1d-0bff-4404-b009-c2eaab76bbec\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"126d2c1d-0bff-4404-b009-c2eaab76bbec\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_brainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"126d2c1d-0bff-4404-b009-c2eaab76bbec\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"126d2c1d-0bff-4404-b009-c2eaab76bbec\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_brainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: \n",
      "Best protected: \n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          36,
          0,
          1,
          0,
          0,
          0,
          0,
          2
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_clean_lenBT0_delBurden",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"21d66953-06bc-446d-af93-0007cbe2a719\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"21d66953-06bc-446d-af93-0007cbe2a719\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 36, 0, 1, 0, 0, 0, 0, 2], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_delBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"21d66953-06bc-446d-af93-0007cbe2a719\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"21d66953-06bc-446d-af93-0007cbe2a719\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 36, 0, 1, 0, 0, 0, 0, 2], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_delBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 10448\n",
      "Best protected: 1895\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          35,
          0,
          1,
          0,
          0,
          0,
          0,
          1
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_clean_lenBT0_delGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"e4695e88-c694-41ee-8d5f-e8fdd87e6efc\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"e4695e88-c694-41ee-8d5f-e8fdd87e6efc\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 35, 0, 1, 0, 0, 0, 0, 1], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_delGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"e4695e88-c694-41ee-8d5f-e8fdd87e6efc\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"e4695e88-c694-41ee-8d5f-e8fdd87e6efc\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 35, 0, 1, 0, 0, 0, 0, 1], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_delGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 10448\n",
      "Best protected: 1895\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_clean_lenBT0_delBrainGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"a3d1fbb8-d2d8-48af-a743-5235f9fe2b6f\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"a3d1fbb8-d2d8-48af-a743-5235f9fe2b6f\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_delBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"a3d1fbb8-d2d8-48af-a743-5235f9fe2b6f\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"a3d1fbb8-d2d8-48af-a743-5235f9fe2b6f\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_delBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: \n",
      "Best protected: \n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          0,
          4,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          4,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_clean_lenBT0_dupBurden",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"7f767f86-7c06-44c6-88c1-afb7f6b0c572\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"7f767f86-7c06-44c6-88c1-afb7f6b0c572\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4, 0, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_dupBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"7f767f86-7c06-44c6-88c1-afb7f6b0c572\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"7f767f86-7c06-44c6-88c1-afb7f6b0c572\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4, 0, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_dupBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10090, 10164\n",
      "Best protected: 10448, 10131\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          0,
          4,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          4,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_clean_lenBT0_dupGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"50e13b13-90e8-462a-8f67-289be5140db2\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"50e13b13-90e8-462a-8f67-289be5140db2\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4, 0, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_dupGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"50e13b13-90e8-462a-8f67-289be5140db2\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"50e13b13-90e8-462a-8f67-289be5140db2\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4, 0, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_dupGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10090, 10164\n",
      "Best protected: 10448, 10131\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400126",
          "CLIA_400122",
          "CLIA_400125",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163",
          "CLIA_400216",
          "CLIA_400121",
          "CLIA_400131",
          "CLIA_400128",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400149",
          "CLIA_400205",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400123",
          "CLIA_400130",
          "CLIA_400204"
         ],
         "type": "scatter",
         "x": [
          1,
          2,
          3,
          4,
          5,
          6,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400155",
          "CLIA_400135",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400158",
          "CLIA_400209",
          "CLIA_400175",
          "CLIA_400162",
          "CLIA_400133",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400159",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400148",
          "CLIA_400200",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400195",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          2,
          3,
          3,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          11,
          12,
          13,
          13,
          14,
          14,
          14,
          15,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22,
          22
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_clean_lenBT0_dupBrainGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10041",
          "10042",
          "10164",
          "10215",
          "10128",
          "10369",
          "1895",
          "10459",
          "1892",
          "1893",
          "855",
          "10182",
          "10178",
          "10153",
          "10131",
          "10406",
          "10173",
          "10448",
          "10033",
          "10197",
          "10090",
          "10094",
          "1976"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19,
          20,
          21,
          22
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"8cda9191-24fe-44b4-a04f-7310d6aa7ce9\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"8cda9191-24fe-44b4-a04f-7310d6aa7ce9\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_dupBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"8cda9191-24fe-44b4-a04f-7310d6aa7ce9\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"8cda9191-24fe-44b4-a04f-7310d6aa7ce9\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400126\", \"CLIA_400122\", \"CLIA_400125\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\", \"CLIA_400216\", \"CLIA_400121\", \"CLIA_400131\", \"CLIA_400128\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400149\", \"CLIA_400205\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400123\", \"CLIA_400130\", \"CLIA_400204\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400155\", \"CLIA_400135\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400158\", \"CLIA_400209\", \"CLIA_400175\", \"CLIA_400162\", \"CLIA_400133\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400159\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400148\", \"CLIA_400200\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400195\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 2, 3, 3, 3, 4, 5, 6, 7, 8, 9, 10, 11, 11, 12, 13, 13, 14, 14, 14, 15, 15, 16, 17, 18, 19, 20, 21, 22, 22], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_clean_lenBT0_dupBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10041\", \"10042\", \"10164\", \"10215\", \"10128\", \"10369\", \"1895\", \"10459\", \"1892\", \"1893\", \"855\", \"10182\", \"10178\", \"10153\", \"10131\", \"10406\", \"10173\", \"10448\", \"10033\", \"10197\", \"10090\", \"10094\", \"1976\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: \n",
      "Best protected: \n"
     ]
    }
   ],
   "source": [
    "import glob\n",
    "\n",
    "data_dir = '/home/sudregp/data/cnv/xhmm/'\n",
    "files = glob.glob(data_dir + '/denovo*clean_lenBT0*cnv.indiv')\n",
    "for f in files:\n",
    "    t_str = '.'.join(f.split('/')[-1].split('.')[:-2])\n",
    "    plot_plink_cnvs(f, t_str, verbose=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "As usual, cleaning does help a lot. We still have some weird stuff going on for family 10406, which is catching a lot of deletions. But there might be come cool stuff for 10090 and 10164. Let's take a look at the genes:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CLIA_400122\n",
      "RANGE (+/- 0kb )  [ 2 99235568 99347589 MGAT4A ]\n",
      "RANGE (+/- 0kb )  [ 2 174771186 174830430 SP3 ]\n",
      "RANGE (+/- 0kb )  [ 5 172483354 172566291 CREBRF ]\n",
      "RANGE (+/- 0kb )  [ 13 108870762 108886603 ABHD13 ]\n",
      "CLIA_400123\n",
      "RANGE (+/- 0kb )  [ 4 17812435 17846487 NCAPG ]\n",
      "RANGE (+/- 0kb )  [ 4 17844838 18023483 LCORL ]\n",
      "RANGE (+/- 0kb )  [ 4 83821836 83841284 THAP9 ]\n",
      "RANGE (+/- 0kb )  [ 4 83845756 83934094 LIN54 ]\n",
      "RANGE (+/- 0kb )  [ 12 22778075 22843608 ETNK1 ]\n",
      "CLIA_400178\n",
      "RANGE (+/- 0kb )  [ 1 62901974 62917475 USP1 ]\n",
      "RANGE (+/- 0kb )  [ 1 62920396 63154039 DOCK7 ]\n",
      "RANGE (+/- 0kb )  [ 1 67390577 67454302 MIER1 ]\n",
      "RANGE (+/- 0kb )  [ 1 71512188 71532865 ZRANB2-AS1 ]\n",
      "RANGE (+/- 0kb )  [ 1 71528973 71546972 ZRANB2 ]\n",
      "RANGE (+/- 0kb )  [ 1 71533313 71533399 MIR186 ]\n",
      "RANGE (+/- 0kb )  [ 1 71547006 71703406 ZRANB2-AS2 ]\n",
      "RANGE (+/- 0kb )  [ 1 71868624 72748277 NEGR1 ]\n",
      "RANGE (+/- 0kb )  [ 1 93811477 93828148 DR1 ]\n",
      "RANGE (+/- 0kb )  [ 2 48010220 48034092 MSH6 ]\n",
      "RANGE (+/- 0kb )  [ 2 48034058 48132932 FBXO11 ]\n",
      "RANGE (+/- 0kb )  [ 2 174771186 174830430 SP3 ]\n",
      "RANGE (+/- 0kb )  [ 2 201390864 201448818 SGOL2 ]\n",
      "RANGE (+/- 0kb )  [ 3 42642146 42690233 NKTR ]\n",
      "RANGE (+/- 0kb )  [ 3 180319917 180336135 TTC14 ]\n",
      "RANGE (+/- 0kb )  [ 3 180331795 180397283 CCDC39 ]\n",
      "RANGE (+/- 0kb )  [ 4 17812435 17846487 NCAPG ]\n",
      "RANGE (+/- 0kb )  [ 4 17844838 18023483 LCORL ]\n",
      "RANGE (+/- 0kb )  [ 4 68424445 68472616 STAP1 ]\n",
      "RANGE (+/- 0kb )  [ 4 68481478 68566889 UBA6 ]\n",
      "RANGE (+/- 0kb )  [ 4 100817406 100867883 DNAJB14 ]\n",
      "RANGE (+/- 0kb )  [ 4 113152894 113191211 AP1AR ]\n",
      "RANGE (+/- 0kb )  [ 5 65222381 65376851 ERBB2IP ]\n",
      "RANGE (+/- 0kb )  [ 5 86690078 86708850 CCNH ]\n",
      "RANGE (+/- 0kb )  [ 5 87485449 87564696 TMEM161B ]\n",
      "RANGE (+/- 0kb )  [ 5 89689528 89705603 CETN3 ]\n",
      "RANGE (+/- 0kb )  [ 5 89705816 89706486 LOC731157 ]\n",
      "RANGE (+/- 0kb )  [ 5 89754019 89770585 MBLAC2 ]\n",
      "RANGE (+/- 0kb )  [ 5 89770680 89810369 POLR3G ]\n",
      "RANGE (+/- 0kb )  [ 5 89811444 89825401 LYSMD3 ]\n",
      "RANGE (+/- 0kb )  [ 5 172483354 172566291 CREBRF ]\n",
      "RANGE (+/- 0kb )  [ 6 109687716 109703762 CD164 ]\n",
      "RANGE (+/- 0kb )  [ 6 111620233 111804918 REV3L ]\n",
      "RANGE (+/- 0kb )  [ 6 136552167 136571449 MTFR2 ]\n",
      "RANGE (+/- 0kb )  [ 6 136578000 136610989 BCLAF1 ]\n",
      "RANGE (+/- 0kb )  [ 8 81397853 81438500 ZBTB10 ]\n",
      "RANGE (+/- 0kb )  [ 10 60094738 60130513 UBE2D1 ]\n",
      "RANGE (+/- 0kb )  [ 11 32605312 32624037 EIF3M ]\n",
      "RANGE (+/- 0kb )  [ 11 32623625 32816204 CCDC73 ]\n",
      "RANGE (+/- 0kb )  [ 11 96123157 96126727 JRKL ]\n",
      "RANGE (+/- 0kb )  [ 12 22778075 22843608 ETNK1 ]\n",
      "RANGE (+/- 0kb )  [ 12 44152746 44183346 IRAK4 ]\n",
      "RANGE (+/- 0kb )  [ 12 44187525 44200178 TWF1 ]\n",
      "RANGE (+/- 0kb )  [ 12 50794591 50873788 LARP4 ]\n",
      "RANGE (+/- 0kb )  [ 12 69633316 69668138 CPSF6 ]\n",
      "RANGE (+/- 0kb )  [ 12 69666936 69666998 MIR1279 ]\n",
      "RANGE (+/- 0kb )  [ 12 75874512 75895716 GLIPR1 ]\n",
      "RANGE (+/- 0kb )  [ 12 75891418 75905418 KRR1 ]\n",
      "RANGE (+/- 0kb )  [ 13 103381716 103411422 CCDC168 ]\n",
      "RANGE (+/- 0kb )  [ 16 53088944 53361414 CHD9 ]\n",
      "RANGE (+/- 0kb )  [ 17 60019965 60142643 MED13 ]\n",
      "RANGE (+/- 0kb )  [ 18 19321289 19450918 MIB1 ]\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd /home/sudregp/data/cnv/xhmm/\n",
    "\n",
    "for kid in CLIA_400122 CLIA_400123 CLIA_400178; do\n",
    "    echo $kid\n",
    "    grep -B 3 $kid denovo_clean_lenBT0_genes.reg | grep RANGE;\n",
    "done"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This is a bit more interesting, as CLIA_400178 has CNVs in the same genes as the other two samples, even though the other two don't have intersections. It'd be nice if we could get rid of some of the crap in CLIA_400178, and maybe bring up the numbers for other samples. Let's see how big are these CNVs. The issue here is that CLIA_400178 only has deletions, while the other two samples have duplications. Maybe it's an issue with the genes, but it'd nicer if it was all the same."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Gene-based analysis\n",
    "\n",
    "Let's see if there are any specific genes we should look at, before we do any cleaning:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>NBPF20</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>NBPF9</th>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GOLGA6L10</th>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GOLGA6L9</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GOLGA6L17P</th>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>CROCC</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>PI4KAP1</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSTT1</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>MIR548N</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>PPP1R10</th>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "            count\n",
       "NBPF20          8\n",
       "NBPF9           8\n",
       "GOLGA6L10       7\n",
       "GOLGA6L9        6\n",
       "GOLGA6L17P      6\n",
       "CROCC           5\n",
       "PI4KAP1         5\n",
       "GSTT1           5\n",
       "MIR548N         5\n",
       "PPP1R10         5"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "fid = open('/home/sudregp/data/cnv/xhmm/denovo_lenBT0_genes.reg')\n",
    "genes = {}\n",
    "for line in fid:\n",
    "    if line.find('RANGE') >= 0:\n",
    "        gene = line.split(' ')[-2]\n",
    "        genes[gene] = 0\n",
    "    elif line.find('DUP') > 0 or line.find('DEL') > 0:\n",
    "        genes[gene] += 1\n",
    "fid.close()\n",
    "df = pd.DataFrame.from_dict(genes, orient='index')\n",
    "df.columns = ['count']\n",
    "df.sort_values(by='count', ascending=False).head(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RANGE (+/- 0kb )  [ 1 144146810 146467744 NBPF20 ]\n",
      " FID           IID      PHE  CHR          BP1          BP2   TYPE       KB     OLAP   OLAP_U   OLAP_R\n",
      "   1   CLIA_400170        1    1    144674717    144684957    DEL    10.24        1 0.004412 0.004412\n",
      "   1   CLIA_400123        2    1    144816539    144855868    DEL    39.33        1  0.01695  0.01695\n",
      "   1   CLIA_400181        1    1    144935052    144952344    DEL    17.29        1 0.007451 0.007451\n",
      "   1   CLIA_400179        1    1    144955188    145075113    DUP    119.9        1  0.05167  0.05167\n",
      "   1   CLIA_400170        1    1    144955188    145075903    DEL    120.7        1  0.05201  0.05201\n",
      "   1   CLIA_400132        2    1    145248734    145290569    DUP    41.84        1  0.01803  0.01803\n",
      "   1   CLIA_400166        1    1    145273133    145290569    DUP    17.44        1 0.007513 0.007513\n",
      "   1   CLIA_400142        1    1    145281327    145295626    DEL     14.3        1 0.006161 0.006161\n",
      "\n",
      "\n",
      "RANGE (+/- 0kb )  [ 1 144614958 145370304 NBPF9 ]\n",
      " FID           IID      PHE  CHR          BP1          BP2   TYPE       KB     OLAP   OLAP_U   OLAP_R\n",
      "   1   CLIA_400170        1    1    144674717    144684957    DEL    10.24        1  0.01356  0.01356\n",
      "   1   CLIA_400123        2    1    144816539    144855868    DEL    39.33        1  0.05207  0.05207\n",
      "   1   CLIA_400181        1    1    144935052    144952344    DEL    17.29        1  0.02289  0.02289\n",
      "   1   CLIA_400179        1    1    144955188    145075113    DUP    119.9        1   0.1588   0.1588\n",
      "   1   CLIA_400170        1    1    144955188    145075903    DEL    120.7        1   0.1598   0.1598\n",
      "   1   CLIA_400132        2    1    145248734    145290569    DUP    41.84        1  0.05539  0.05539\n",
      "   1   CLIA_400166        1    1    145273133    145290569    DUP    17.44        1  0.02308  0.02308\n",
      "   1   CLIA_400142        1    1    145281327    145295626    DEL     14.3        1  0.01893  0.01893\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "grep -A 10 NBPF20 ~/data/cnv/xhmm/denovo_lenBT0_genes.reg\n",
    "echo\n",
    "grep -A 10 NBPF9 ~/data/cnv/xhmm/denovo_lenBT0_genes.reg"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# TODO\n",
    "\n",
    "* Apply CNV length cleaning\n",
    "* Play with quality knob.\n",
    "* Do gene-based analysis parsing the .reg file.\n",
    "* Follow QC rules from XHMM papers \n",
    "* Look into sex chromosomes? Something to the idea that adhd is more present in boys...\n",
    "* Play with the HMM parameters\n",
    "* do population frequency filtering first?\n",
    "* Include parent burden in the analysis\n",
    "* Match with file of ranked simplex by Wendy (maybe blindly)?\n",
    "* Try other WES CNV callers?"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Bash",
   "language": "bash",
   "name": "bash"
  },
  "language_info": {
   "codemirror_mode": "shell",
   "file_extension": ".sh",
   "mimetype": "text/x-sh",
   "name": "bash"
  },
  "nav_menu": {},
  "toc": {
   "navigate_menu": true,
   "number_sections": true,
   "sideBar": true,
   "threshold": 6,
   "toc_cell": false,
   "toc_section_display": "block",
   "toc_window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
