{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The idea here is to run only the simplex families with at least 2 kids in them, for the contrast between affected and unaffected. Also have to remember to remove the other 2 samples we always remove."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<script>requirejs.config({paths: { 'plotly': ['https://cdn.plot.ly/plotly-latest.min']},});if(!window.Plotly) {{require(['plotly'],function(plotly) {window.Plotly=plotly;});}}</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<script>requirejs.config({paths: { 'plotly': ['https://cdn.plot.ly/plotly-latest.min']},});if(!window.Plotly) {{require(['plotly'],function(plotly) {window.Plotly=plotly;});}}</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import numpy as np\n",
    "import matplotlib.pyplot as plt\n",
    "import seaborn as sns\n",
    "import pandas as pd\n",
    "from plotly.offline import download_plotlyjs, init_notebook_mode, plot, iplot\n",
    "\n",
    "init_notebook_mode(connected=True)\n",
    "%matplotlib inline\n",
    "\n",
    "import glob\n",
    "ped_file = '/data/sudregp/cnv/simplex.ped'\n",
    "wes_prefix = ['CLIA', 'CCGO', 'WPS']\n",
    "trios = []\n",
    "affected = []\n",
    "controls = []\n",
    "samples = []\n",
    "famids = []\n",
    "fid = open(ped_file, 'r')\n",
    "for line in fid:\n",
    "    famid, sid, fa, mo, sex, aff = line.rstrip().split('\\t')\n",
    "    if fa[:4] in wes_prefix and mo[:4] in wes_prefix and sid[:4] in wes_prefix:\n",
    "        fam = {}\n",
    "        fam['child'] = sid\n",
    "        if aff == '1':\n",
    "            affected.append(sid)\n",
    "        else:\n",
    "            controls.append(sid)\n",
    "        fam['father'] = fa\n",
    "        fam['mother'] = mo\n",
    "        fam['famid'] = famid\n",
    "        trios.append(fam)\n",
    "        samples += [sid, fa, mo]\n",
    "        famids.append(famid)\n",
    "fid.close()\n",
    "samples = set(samples)\n",
    "famids = set(famids)\n",
    "\n",
    "fid = open('/data/sudregp/cnv/kid_samples.txt', 'r')\n",
    "good_kids = [line.rstrip() for line in fid]\n",
    "fid.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10041\n",
      "10042\n",
      "10459\n"
     ]
    }
   ],
   "source": [
    "for fam in famids:\n",
    "    fam_kids = [t['child'] for t in trios if t['famid'] == fam]\n",
    "    all_controls = True\n",
    "    for kid in fam_kids:\n",
    "        all_controls &= kid in affected\n",
    "    if len(fam_kids) < 2 or all_controls:\n",
    "        print fam"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "That means we'll remove 9 samples:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CLIA_400165\n",
      "CLIA_400164\n",
      "CLIA_400155\n",
      "CLIA_400146\n",
      "CLIA_400145\n",
      "CLIA_400126\n",
      "CLIA_400207\n",
      "CLIA_400208\n",
      "CLIA_400209\n"
     ]
    }
   ],
   "source": [
    "for trio in trios:\n",
    "    if trio['famid'] in ['10041', '10042', '10459']:\n",
    "        print '%s\\n%s\\n%s' % (trio['father'], trio['mother'], trio['child'])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "And the other 2 we always remove because of lack of parent:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CLIA_400169\n",
      "CLIA_400168\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cat /home/sudregp/data/cnv/exclude.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# in terminal\n",
    "exome_targets='/data/NCR_SBRB/simplex/SeqCapEZ_Exome_v3.0_Design_Annotation_files/SeqCap_EZ_Exome_v3_hg19_capture_targets.bed'\n",
    "gatk_memory=\"50g\"\n",
    "ref_fa='/fdb/GATK_resource_bundle/hg19-2.8/ucsc.hg19.fasta'\n",
    "out_dir='/data/sudregp/cnv/xhmm_multikids'\n",
    "\n",
    "cd $out_dir\n",
    "module load GATK\n",
    "module load XHMM\n",
    "\n",
    "GATK -m ${gatk_memory} GCContentByInterval -L ${exome_targets} -R ${ref_fa} -o ./DATA.locus_GC.txt\n",
    "cat ./DATA.locus_GC.txt | awk '{if ($2 < 0.1 || $2 > 0.9) print $1}' > ./extreme_gc_targets.txt\n",
    "\n",
    "# merging all subjects in the directory\n",
    "ls -1 *.sample_interval_summary > depth_list.txt;\n",
    "grep -v -f ../exclude.txt depth_list.txt > depth_list2.txt\n",
    "grep -v -f ../exclude_multikids.txt depth_list2.txt > depth_list3.txt\n",
    "\n",
    "xhmm --mergeGATKdepths --GATKdepthsList=depth_list3.txt -o ./DATA.RD.txt;\n",
    "\n",
    "/usr/local/apps/XHMM/2016-01-04/sources/scripts/interval_list_to_pseq_reg ${exome_targets} > ./EXOME.targets.reg\n",
    "\n",
    "xhmm --matrix -r ./DATA.RD.txt --centerData --centerType target \\\n",
    "-o ./DATA.filtered_centered.RD.txt \\\n",
    "--outputExcludedTargets ./DATA.filtered_centered.RD.txt.filtered_targets.txt \\\n",
    "--outputExcludedSamples ./DATA.filtered_centered.RD.txt.filtered_samples.txt \\\n",
    "--minTargetSize 10 --maxTargetSize 10000 \\\n",
    "--minMeanTargetRD 10 --maxMeanTargetRD 500 \\\n",
    "--minMeanSampleRD 25 --maxMeanSampleRD 200 \\\n",
    "--maxSdSampleRD 150\n",
    "\n",
    "xhmm --PCA -r ./DATA.filtered_centered.RD.txt --PCAfiles ./DATA.RD_PCA\n",
    "\n",
    "xhmm --normalize -r ./DATA.filtered_centered.RD.txt --PCAfiles ./DATA.RD_PCA \\\n",
    "--normalizeOutput ./DATA.PCA_normalized.txt \\\n",
    "--PCnormalizeMethod PVE_mean --PVE_mean_factor 0.7\n",
    "\n",
    "xhmm --matrix -r ./DATA.PCA_normalized.txt --centerData --centerType sample --zScoreData \\\n",
    "-o ./DATA.PCA_normalized.filtered.sample_zscores.RD.txt \\\n",
    "--outputExcludedTargets ./DATA.PCA_normalized.filtered.sample_zscores.RD.txt.filtered_targets.txt \\\n",
    "--outputExcludedSamples ./DATA.PCA_normalized.filtered.sample_zscores.RD.txt.filtered_samples.txt \\\n",
    "--maxSdTargetRD 30\n",
    "\n",
    "xhmm --matrix -r ./DATA.RD.txt \\\n",
    "--excludeTargets ./DATA.filtered_centered.RD.txt.filtered_targets.txt \\\n",
    "--excludeTargets ./DATA.PCA_normalized.filtered.sample_zscores.RD.txt.filtered_targets.txt \\\n",
    "--excludeSamples ./DATA.filtered_centered.RD.txt.filtered_samples.txt \\\n",
    "--excludeSamples ./DATA.PCA_normalized.filtered.sample_zscores.RD.txt.filtered_samples.txt \\\n",
    "-o ./DATA.same_filtered.RD.txt\n",
    "\n",
    "xhmm --discover -p params.txt -r ./DATA.PCA_normalized.filtered.sample_zscores.RD.txt \\\n",
    "    -R ./DATA.same_filtered.RD.txt -c ./DATA.xcnv -a ./DATA.aux_xcnv -s ./DATA\n",
    "\n",
    "xhmm --genotype -p params.txt -r ./DATA.PCA_normalized.filtered.sample_zscores.RD.txt \\\n",
    "    -R ./DATA.same_filtered.RD.txt -g ./DATA.xcnv -F $ref_fa -v ./DATA.vcf\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Let's make sure no sample was removed by XHMM this time:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-rw-rw---- 1 sudregp sudregp 0 Jan 19 15:06 DATA.filtered_centered.RD.txt.filtered_samples.txt\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd ~/data/cnv/xhmm_multikids/\n",
    "ls -ltr DATA.filtered_centered.RD.txt.filtered_samples.txtb"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "And now we create PLINK files."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[+] Loading GSL 2.2.1 ...\n",
      "[+] Loading Graphviz v2.38.0 ...\n",
      "[+] Loading gdal 2.0 ...\n",
      "[+] Loading proj 4.9.2 ...\n",
      "[+] Loading gcc 4.9.1 ...\n",
      "[+] Loading openmpi 1.10.0 for GCC 4.9.1\n",
      "[+] Loading tcl_tk 8.6.3\n",
      "[+] Loading Zlib 1.2.8 ...\n",
      "[+] Loading Bzip2 1.0.6 ...\n",
      "[+] Loading pcre 8.38 ...\n",
      "[+] Loading liblzma 5.2.2 ...\n",
      "[-] Unloading Zlib 1.2.8 ...\n",
      "[+] Loading Zlib 1.2.8 ...\n",
      "[-] Unloading liblzma 5.2.2 ...\n",
      "[+] Loading liblzma 5.2.2 ...\n",
      "[+] Loading libjpeg-turbo 1.5.1 ...\n",
      "[+] Loading tiff 4.0.7 ...\n",
      "[+] Loading curl 7.46.0 ...\n",
      "[+] Loading boost libraries v1.65 ...\n",
      "[+] Loading R 3.4.0 on cn3385\n",
      "-------------------------------------------------------------------------------\n",
      "||||||||||||||||||||||||||| PSEQ (v0.10; 14-Jul-14) |||||||||||||||||||||||||||\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Copying this log to file [ DATA_q50.log ]\n",
      "Analysis started Fri Jan 19 16:53:06 2018\n",
      "\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Project : DATA\n",
      "Command : cnv-denovo\n",
      "Options : --noweb\n",
      "          --minSQ 50\n",
      "          --minNQ 50\n",
      "          --out DATA_q50\n",
      "\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Writing to file [ DATA_q50.denovo.cnv ]  : per-site output from cnv-denovo\n",
      "Writing to file [ DATA_q50.denovo.cnv.indiv ]  : per-trio output from cnv-denovo\n",
      "Starting CNV de novo scan...\n",
      "Included 7926 of 7926 variants considered\n",
      "\n",
      "Analysis finished Fri Jan 19 16:53:16 2018\n",
      ":::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::\n",
      "-------------------------------------------------------------------------------\n",
      "||||||||||||||||||||||||||| PSEQ (v0.10; 14-Jul-14) |||||||||||||||||||||||||||\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Copying this log to file [ DATA_q60.log ]\n",
      "Analysis started Fri Jan 19 16:53:36 2018\n",
      "\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Project : DATA\n",
      "Command : cnv-denovo\n",
      "Options : --noweb\n",
      "          --minSQ 60\n",
      "          --minNQ 60\n",
      "          --out DATA_q60\n",
      "\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Writing to file [ DATA_q60.denovo.cnv ]  : per-site output from cnv-denovo\n",
      "Writing to file [ DATA_q60.denovo.cnv.indiv ]  : per-trio output from cnv-denovo\n",
      "Starting CNV de novo scan...\n",
      "Included 7926 of 7926 variants considered\n",
      "\n",
      "Analysis finished Fri Jan 19 16:53:45 2018\n",
      ":::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::\n",
      "-------------------------------------------------------------------------------\n",
      "||||||||||||||||||||||||||| PSEQ (v0.10; 14-Jul-14) |||||||||||||||||||||||||||\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Copying this log to file [ DATA_q70.log ]\n",
      "Analysis started Fri Jan 19 16:54:01 2018\n",
      "\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Project : DATA\n",
      "Command : cnv-denovo\n",
      "Options : --noweb\n",
      "          --minSQ 70\n",
      "          --minNQ 70\n",
      "          --out DATA_q70\n",
      "\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Writing to file [ DATA_q70.denovo.cnv ]  : per-site output from cnv-denovo\n",
      "Writing to file [ DATA_q70.denovo.cnv.indiv ]  : per-trio output from cnv-denovo\n",
      "Starting CNV de novo scan...\n",
      "Included 7926 of 7926 variants considered\n",
      "\n",
      "Analysis finished Fri Jan 19 16:54:11 2018\n",
      ":::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::\n",
      "-------------------------------------------------------------------------------\n",
      "||||||||||||||||||||||||||| PSEQ (v0.10; 14-Jul-14) |||||||||||||||||||||||||||\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Copying this log to file [ DATA_q80.log ]\n",
      "Analysis started Fri Jan 19 16:54:22 2018\n",
      "\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Project : DATA\n",
      "Command : cnv-denovo\n",
      "Options : --noweb\n",
      "          --minSQ 80\n",
      "          --minNQ 80\n",
      "          --out DATA_q80\n",
      "\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Writing to file [ DATA_q80.denovo.cnv ]  : per-site output from cnv-denovo\n",
      "Writing to file [ DATA_q80.denovo.cnv.indiv ]  : per-trio output from cnv-denovo\n",
      "Starting CNV de novo scan...\n",
      "Included 7926 of 7926 variants considered\n",
      "\n",
      "Analysis finished Fri Jan 19 16:54:32 2018\n",
      ":::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::\n",
      "-------------------------------------------------------------------------------\n",
      "||||||||||||||||||||||||||| PSEQ (v0.10; 14-Jul-14) |||||||||||||||||||||||||||\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Copying this log to file [ DATA_q90.log ]\n",
      "Analysis started Fri Jan 19 16:54:41 2018\n",
      "\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Project : DATA\n",
      "Command : cnv-denovo\n",
      "Options : --noweb\n",
      "          --minSQ 90\n",
      "          --minNQ 90\n",
      "          --out DATA_q90\n",
      "\n",
      "-------------------------------------------------------------------------------\n",
      "\n",
      "Writing to file [ DATA_q90.denovo.cnv ]  : per-site output from cnv-denovo\n",
      "Writing to file [ DATA_q90.denovo.cnv.indiv ]  : per-trio output from cnv-denovo\n",
      "Starting CNV de novo scan...\n",
      "Included 7926 of 7926 variants considered\n",
      "\n",
      "Analysis finished Fri Jan 19 16:54:50 2018\n",
      ":::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::\n"
     ]
    }
   ],
   "source": [
    "module load plinkseq\n",
    "module load plink/1.07\n",
    "\n",
    "cd ~/data/cnv/xhmm_multikids\n",
    "\n",
    "for q in 50 60 70 80 90; do\n",
    "    pseq DATA cnv-denovo --noweb --minSQ $q --minNQ $q --out DATA_q${q}\n",
    "    grep DENOVO DATA_q${q}.denovo.cnv > pseq_DENOVO.txt\n",
    "    # borrow the header row\n",
    "    head -1 DATA.xcnv > denovo.xcnv;\n",
    "\n",
    "    # filter out denovo CNVs\n",
    "    while read sample; do\n",
    "        grep $sample DATA.xcnv > sample.xcnv;\n",
    "        for cnv in `grep $sample pseq_DENOVO.txt | cut -f 3 -`; do\n",
    "            # replacing .. by -\n",
    "            cnv=`echo $cnv | sed -e 's/\\.\\./\\-/'`;\n",
    "            grep $cnv sample.xcnv >> denovo.xcnv; \n",
    "        done;\n",
    "    done < ../kid_samples.txt;\n",
    "    /usr/local/apps/XHMM/2016-01-04/sources/scripts/xcnv_to_cnv denovo.xcnv > tmp.cnv\n",
    "    # switch around FAMID and IID columns, and remove header\n",
    "    awk '{OFS=\"\\t\"; if ( $3 != \"CHR\" ) {print $2, $1, $3, $4, $5, $6, $7, $8 }}' tmp.cnv > denovo_q${q}.cnv\n",
    "    rm sample.xcnv pseq_DENOVO.txt tmp.cnv denovo.xcnv\n",
    "    \n",
    "    # filter out inherited cnvs\n",
    "    grep MATERNAL_TRANSMITTED DATA_q${q}.denovo.cnv > pseq_TRANSMITTED.txt\n",
    "    grep PATERNAL_TRANSMITTED DATA_q${q}.denovo.cnv >> pseq_TRANSMITTED.txt\n",
    "    # borrow the header row\n",
    "    head -1 DATA.xcnv > inherited.xcnv;\n",
    "\n",
    "    while read sample; do\n",
    "        grep $sample DATA.xcnv > sample.xcnv;\n",
    "        for cnv in `grep $sample pseq_TRANSMITTED.txt | cut -f 3 -`; do\n",
    "            # replacing .. by -\n",
    "            cnv=`echo $cnv | sed -e 's/\\.\\./\\-/'`;\n",
    "            grep $cnv sample.xcnv >> inherited.xcnv; \n",
    "        done;\n",
    "    done < ../kid_samples.txt;\n",
    "    /usr/local/apps/XHMM/2016-01-04/sources/scripts/xcnv_to_cnv inherited.xcnv > tmp.cnv\n",
    "    # switch around FAMID and IID columns, and remove header\n",
    "    awk '{OFS=\"\\t\"; if ( $3 != \"CHR\" ) {print $2, $1, $3, $4, $5, $6, $7, $8 }}' tmp.cnv > inherited_q${q}.cnv\n",
    "    rm sample.xcnv pseq_TRANSMITTED.txt tmp.cnv inherited.xcnv\n",
    "    \n",
    "    # compile all CNVs for kids\n",
    "    # borrow the header row\n",
    "    head -1 DATA.xcnv > all.xcnv;\n",
    "\n",
    "    # effectively just filtering DATA.xcnv to keep only kids\n",
    "    while read sample; do\n",
    "        grep $sample DATA.xcnv >> all.xcnv;\n",
    "    done < ../kid_samples.txt;\n",
    "    /usr/local/apps/XHMM/2016-01-04/sources/scripts/xcnv_to_cnv all.xcnv > tmp.cnv\n",
    "    # switch around FAMID and IID columns, and remove header\n",
    "    awk '{OFS=\"\\t\"; if ( $3 != \"CHR\" ) {print $2, $1, $3, $4, $5, $6, $7, $8 }}' tmp.cnv > all_q${q}.cnv\n",
    "    rm tmp.cnv all.xcnv\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50.log ]\n",
      "Analysis started: Fri Jan 19 17:00:23 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list all_q50.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--out all_q50\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50.cnv ]\n",
      "Writing new MAP file to [ all_q50.cnv.map ]\n",
      "Wrote 10101 unique positions to file\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:00:23 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_clean.log ]\n",
      "Analysis started: Fri Jan 19 17:00:23 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-exclude bad_regions.list\n",
      "\t--cnv-overlap .5\n",
      "\t--cnv-write\n",
      "\t--out all_q50_clean\n",
      "\n",
      "Reading marker information from [ all_q50.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading CNV intersection list from [ bad_regions.list ]\n",
      "Read 380994 ranges to exclude from CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50.cnv ]\n",
      "6924 mapped to a person, of which 6924 passed filters\n",
      "170 kept after excluding specific regions\n",
      "170 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      78 / 25\n",
      "     3      25 / 42\n",
      "\n",
      "Writing new CNV list to [ all_q50_clean.cnv ]\n",
      "Writing new FAM file to [ all_q50_clean.fam ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:08 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_clean.log ]\n",
      "Analysis started: Fri Jan 19 17:02:08 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list all_q50_clean.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--out all_q50_clean\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50_clean.cnv ]\n",
      "Writing new MAP file to [ all_q50_clean.cnv.map ]\n",
      "Wrote 429 unique positions to file\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:08 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_burden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:08 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--out all_q50_burden\n",
      "\n",
      "Reading marker information from [ all_q50.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50.cnv ]\n",
      "No overlapping samples found\n",
      "6924 mapped to a person, of which 6924 passed filters\n",
      "6924 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1  1698 / 1747\n",
      "     3  1532 / 1947\n",
      "\n",
      "Writing per-individual summary to [ all_q50_burden.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:08 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_genes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:08 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--out all_q50_genes\n",
      "\n",
      "Reading marker information from [ all_q50.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50.cnv ]\n",
      "Writing back list to intersected regions to [ all_q50_genes.reg ]\n",
      "6924 mapped to a person, of which 6924 passed filters\n",
      "5942 intersected with one or more specified region\n",
      "5942 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1  1474 / 1481\n",
      "     3  1297 / 1690\n",
      "\n",
      "Writing per-individual summary to [ all_q50_genes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:14 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_brainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:14 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--out all_q50_brainGenes\n",
      "\n",
      "Reading marker information from [ all_q50.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_q50_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50.cnv ]\n",
      "Writing back list to intersected regions to [ all_q50_brainGenes.reg ]\n",
      "6924 mapped to a person, of which 6924 passed filters\n",
      "54 intersected with one or more specified region\n",
      "54 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      14 / 11\n",
      "     3      13 / 16\n",
      "\n",
      "Writing per-individual summary to [ all_q50_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:15 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_delBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:15 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--out all_q50_delBurden\n",
      "\n",
      "Reading marker information from [ all_q50.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50.cnv ]\n",
      "6924 mapped to a person, of which 3445 passed filters\n",
      "3445 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1  1698 / 1747\n",
      "\n",
      "Writing per-individual summary to [ all_q50_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:15 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_delGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:15 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--out all_q50_delGenes\n",
      "\n",
      "Reading marker information from [ all_q50.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50.cnv ]\n",
      "Writing back list to intersected regions to [ all_q50_delGenes.reg ]\n",
      "6924 mapped to a person, of which 3445 passed filters\n",
      "2955 intersected with one or more specified region\n",
      "2955 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1  1474 / 1481\n",
      "\n",
      "Writing per-individual summary to [ all_q50_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:18 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_delBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:18 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--out all_q50_delBrainGenes\n",
      "\n",
      "Reading marker information from [ all_q50.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_q50_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50.cnv ]\n",
      "Writing back list to intersected regions to [ all_q50_delBrainGenes.reg ]\n",
      "6924 mapped to a person, of which 3445 passed filters\n",
      "25 intersected with one or more specified region\n",
      "25 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      14 / 11\n",
      "\n",
      "Writing per-individual summary to [ all_q50_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:18 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_dupBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:18 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--out all_q50_dupBurden\n",
      "\n",
      "Reading marker information from [ all_q50.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50.cnv ]\n",
      "6924 mapped to a person, of which 3479 passed filters\n",
      "3479 of 6924 mapped as valid segments\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " CopyN Case/Control\n",
      "     3  1532 / 1947\n",
      "\n",
      "Writing per-individual summary to [ all_q50_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:18 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_dupGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:18 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--out all_q50_dupGenes\n",
      "\n",
      "Reading marker information from [ all_q50.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50.cnv ]\n",
      "Writing back list to intersected regions to [ all_q50_dupGenes.reg ]\n",
      "6924 mapped to a person, of which 3479 passed filters\n",
      "2987 intersected with one or more specified region\n",
      "2987 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3  1297 / 1690\n",
      "\n",
      "Writing per-individual summary to [ all_q50_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:22 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_dupBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:22 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--out all_q50_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ all_q50.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_q50_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50.cnv ]\n",
      "Writing back list to intersected regions to [ all_q50_dupBrainGenes.reg ]\n",
      "6924 mapped to a person, of which 3479 passed filters\n",
      "29 intersected with one or more specified region\n",
      "29 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      13 / 16\n",
      "\n",
      "Writing per-individual summary to [ all_q50_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:22 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_clean_burden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:22 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--out all_q50_clean_burden\n",
      "\n",
      "Reading marker information from [ all_q50_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50_clean.cnv ]\n",
      "No overlapping samples found\n",
      "170 mapped to a person, of which 170 passed filters\n",
      "170 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      78 / 25\n",
      "     3      25 / 42\n",
      "\n",
      "Writing per-individual summary to [ all_q50_clean_burden.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_clean_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:22 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_clean_genes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:22 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--out all_q50_clean_genes\n",
      "\n",
      "Reading marker information from [ all_q50_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_q50_clean_genes.reg ]\n",
      "170 mapped to a person, of which 170 passed filters\n",
      "165 intersected with one or more specified region\n",
      "165 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      75 / 25\n",
      "     3      24 / 41\n",
      "\n",
      "Writing per-individual summary to [ all_q50_clean_genes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_clean_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:22 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_clean_brainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:22 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--out all_q50_clean_brainGenes\n",
      "\n",
      "Reading marker information from [ all_q50_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_q50_clean_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_q50_clean_brainGenes.reg ]\n",
      "170 mapped to a person, of which 170 passed filters\n",
      "4 intersected with one or more specified region\n",
      "4 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        2 / 1\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ all_q50_clean_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_clean_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:23 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_clean_delBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:23 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--out all_q50_clean_delBurden\n",
      "\n",
      "Reading marker information from [ all_q50_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50_clean.cnv ]\n",
      "170 mapped to a person, of which 103 passed filters\n",
      "103 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      78 / 25\n",
      "\n",
      "Writing per-individual summary to [ all_q50_clean_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_clean_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:23 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_clean_delGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:23 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--out all_q50_clean_delGenes\n",
      "\n",
      "Reading marker information from [ all_q50_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_q50_clean_delGenes.reg ]\n",
      "170 mapped to a person, of which 103 passed filters\n",
      "100 intersected with one or more specified region\n",
      "100 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      75 / 25\n",
      "\n",
      "Writing per-individual summary to [ all_q50_clean_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_clean_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:23 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_clean_delBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:23 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--out all_q50_clean_delBrainGenes\n",
      "\n",
      "Reading marker information from [ all_q50_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_q50_clean_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_q50_clean_delBrainGenes.reg ]\n",
      "170 mapped to a person, of which 103 passed filters\n",
      "3 intersected with one or more specified region\n",
      "3 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        2 / 1\n",
      "\n",
      "Writing per-individual summary to [ all_q50_clean_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_clean_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:23 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_clean_dupBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:23 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--out all_q50_clean_dupBurden\n",
      "\n",
      "Reading marker information from [ all_q50_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50_clean.cnv ]\n",
      "170 mapped to a person, of which 67 passed filters\n",
      "67 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      25 / 42\n",
      "\n",
      "Writing per-individual summary to [ all_q50_clean_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_clean_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:23 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_clean_dupGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:23 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--out all_q50_clean_dupGenes\n",
      "\n",
      "Reading marker information from [ all_q50_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_q50_clean_dupGenes.reg ]\n",
      "170 mapped to a person, of which 67 passed filters\n",
      "65 intersected with one or more specified region\n",
      "65 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      24 / 41\n",
      "\n",
      "Writing per-individual summary to [ all_q50_clean_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_clean_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:23 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q50_clean_dupBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:23 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--out all_q50_clean_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ all_q50_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_q50_clean_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_q50_clean_dupBrainGenes.reg ]\n",
      "170 mapped to a person, of which 67 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ all_q50_clean_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q50_clean_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:23 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50.log ]\n",
      "Analysis started: Fri Jan 19 17:02:23 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list denovo_q50.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--out denovo_q50\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50.cnv ]\n",
      "Writing new MAP file to [ denovo_q50.cnv.map ]\n",
      "Wrote 2433 unique positions to file\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:23 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_clean.log ]\n",
      "Analysis started: Fri Jan 19 17:02:23 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-exclude bad_regions.list\n",
      "\t--cnv-overlap .5\n",
      "\t--cnv-write\n",
      "\t--out denovo_q50_clean\n",
      "\n",
      "Reading marker information from [ denovo_q50.cnv.map ]\n",
      "2433 (of 2433) markers to be included from [ denovo_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading CNV intersection list from [ bad_regions.list ]\n",
      "Read 380994 ranges to exclude from CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50.cnv ]\n",
      "954 mapped to a person, of which 954 passed filters\n",
      "74 kept after excluding specific regions\n",
      "74 of 954 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1       48 / 4\n",
      "     3      10 / 12\n",
      "\n",
      "Writing new CNV list to [ denovo_q50_clean.cnv ]\n",
      "Writing new FAM file to [ denovo_q50_clean.fam ]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Analysis finished: Fri Jan 19 17:02:39 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_clean.log ]\n",
      "Analysis started: Fri Jan 19 17:02:39 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list denovo_q50_clean.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--out denovo_q50_clean\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50_clean.cnv ]\n",
      "Writing new MAP file to [ denovo_q50_clean.cnv.map ]\n",
      "Wrote 208 unique positions to file\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:39 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_burden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:39 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--out denovo_q50_burden\n",
      "\n",
      "Reading marker information from [ denovo_q50.cnv.map ]\n",
      "2433 (of 2433) markers to be included from [ denovo_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50.cnv ]\n",
      "No overlapping samples found\n",
      "954 mapped to a person, of which 954 passed filters\n",
      "954 of 954 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    320 / 220\n",
      "     3    189 / 225\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_burden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:39 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_genes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:39 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--out denovo_q50_genes\n",
      "\n",
      "Reading marker information from [ denovo_q50.cnv.map ]\n",
      "2433 (of 2433) markers to be included from [ denovo_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q50_genes.reg ]\n",
      "954 mapped to a person, of which 954 passed filters\n",
      "848 intersected with one or more specified region\n",
      "848 of 954 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    292 / 186\n",
      "     3    168 / 202\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_genes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:40 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_brainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:40 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--out denovo_q50_brainGenes\n",
      "\n",
      "Reading marker information from [ denovo_q50.cnv.map ]\n",
      "2433 (of 2433) markers to be included from [ denovo_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_q50_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q50_brainGenes.reg ]\n",
      "954 mapped to a person, of which 954 passed filters\n",
      "10 intersected with one or more specified region\n",
      "10 of 954 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        4 / 3\n",
      "     3        1 / 2\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:40 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_delBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:40 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--out denovo_q50_delBurden\n",
      "\n",
      "Reading marker information from [ denovo_q50.cnv.map ]\n",
      "2433 (of 2433) markers to be included from [ denovo_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50.cnv ]\n",
      "954 mapped to a person, of which 540 passed filters\n",
      "540 of 954 mapped as valid segments\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " CopyN Case/Control\n",
      "     1    320 / 220\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:40 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_delGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:40 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--out denovo_q50_delGenes\n",
      "\n",
      "Reading marker information from [ denovo_q50.cnv.map ]\n",
      "2433 (of 2433) markers to be included from [ denovo_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q50_delGenes.reg ]\n",
      "954 mapped to a person, of which 540 passed filters\n",
      "478 intersected with one or more specified region\n",
      "478 of 954 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    292 / 186\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:41 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_delBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:41 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--out denovo_q50_delBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo_q50.cnv.map ]\n",
      "2433 (of 2433) markers to be included from [ denovo_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_q50_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q50_delBrainGenes.reg ]\n",
      "954 mapped to a person, of which 540 passed filters\n",
      "7 intersected with one or more specified region\n",
      "7 of 954 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        4 / 3\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:41 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_dupBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:41 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--out denovo_q50_dupBurden\n",
      "\n",
      "Reading marker information from [ denovo_q50.cnv.map ]\n",
      "2433 (of 2433) markers to be included from [ denovo_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50.cnv ]\n",
      "954 mapped to a person, of which 414 passed filters\n",
      "414 of 954 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3    189 / 225\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:41 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_dupGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:41 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--out denovo_q50_dupGenes\n",
      "\n",
      "Reading marker information from [ denovo_q50.cnv.map ]\n",
      "2433 (of 2433) markers to be included from [ denovo_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q50_dupGenes.reg ]\n",
      "954 mapped to a person, of which 414 passed filters\n",
      "370 intersected with one or more specified region\n",
      "370 of 954 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3    168 / 202\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:42 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_dupBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:42 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--out denovo_q50_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo_q50.cnv.map ]\n",
      "2433 (of 2433) markers to be included from [ denovo_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_q50_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q50_dupBrainGenes.reg ]\n",
      "954 mapped to a person, of which 414 passed filters\n",
      "3 intersected with one or more specified region\n",
      "3 of 954 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 2\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:42 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_clean_burden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:42 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--out denovo_q50_clean_burden\n",
      "\n",
      "Reading marker information from [ denovo_q50_clean.cnv.map ]\n",
      "208 (of 208) markers to be included from [ denovo_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50_clean.cnv ]\n",
      "No overlapping samples found\n",
      "74 mapped to a person, of which 74 passed filters\n",
      "74 of 74 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1       48 / 4\n",
      "     3      10 / 12\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_clean_burden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_clean_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:42 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_clean_genes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:42 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--out denovo_q50_clean_genes\n",
      "\n",
      "Reading marker information from [ denovo_q50_clean.cnv.map ]\n",
      "208 (of 208) markers to be included from [ denovo_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q50_clean_genes.reg ]\n",
      "74 mapped to a person, of which 74 passed filters\n",
      "71 intersected with one or more specified region\n",
      "71 of 74 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1       46 / 4\n",
      "     3      10 / 11\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_clean_genes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_clean_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:42 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_clean_brainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:42 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--out denovo_q50_clean_brainGenes\n",
      "\n",
      "Reading marker information from [ denovo_q50_clean.cnv.map ]\n",
      "208 (of 208) markers to be included from [ denovo_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_q50_clean_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q50_clean_brainGenes.reg ]\n",
      "74 mapped to a person, of which 74 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 74 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_clean_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_clean_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:42 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_clean_delBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:42 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--out denovo_q50_clean_delBurden\n",
      "\n",
      "Reading marker information from [ denovo_q50_clean.cnv.map ]\n",
      "208 (of 208) markers to be included from [ denovo_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50_clean.cnv ]\n",
      "74 mapped to a person, of which 52 passed filters\n",
      "52 of 74 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1       48 / 4\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_clean_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_clean_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:42 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_clean_delGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:42 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--out denovo_q50_clean_delGenes\n",
      "\n",
      "Reading marker information from [ denovo_q50_clean.cnv.map ]\n",
      "208 (of 208) markers to be included from [ denovo_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q50_clean_delGenes.reg ]\n",
      "74 mapped to a person, of which 52 passed filters\n",
      "50 intersected with one or more specified region\n",
      "50 of 74 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1       46 / 4\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_clean_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_clean_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:42 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_clean_delBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:42 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--out denovo_q50_clean_delBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo_q50_clean.cnv.map ]\n",
      "208 (of 208) markers to be included from [ denovo_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_q50_clean_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q50_clean_delBrainGenes.reg ]\n",
      "74 mapped to a person, of which 52 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 74 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_clean_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_clean_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:43 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_clean_dupBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:43 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--out denovo_q50_clean_dupBurden\n",
      "\n",
      "Reading marker information from [ denovo_q50_clean.cnv.map ]\n",
      "208 (of 208) markers to be included from [ denovo_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50_clean.cnv ]\n",
      "74 mapped to a person, of which 22 passed filters\n",
      "22 of 74 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      10 / 12\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_clean_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_clean_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:43 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_clean_dupGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:43 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--out denovo_q50_clean_dupGenes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Reading marker information from [ denovo_q50_clean.cnv.map ]\n",
      "208 (of 208) markers to be included from [ denovo_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q50_clean_dupGenes.reg ]\n",
      "74 mapped to a person, of which 22 passed filters\n",
      "21 intersected with one or more specified region\n",
      "21 of 74 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      10 / 11\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_clean_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_clean_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:43 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q50_clean_dupBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:43 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--out denovo_q50_clean_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo_q50_clean.cnv.map ]\n",
      "208 (of 208) markers to be included from [ denovo_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_q50_clean_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q50_clean_dupBrainGenes.reg ]\n",
      "74 mapped to a person, of which 22 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 74 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_q50_clean_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q50_clean_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:43 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50.log ]\n",
      "Analysis started: Fri Jan 19 17:02:43 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list inherited_q50.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--out inherited_q50\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50.cnv ]\n",
      "Writing new MAP file to [ inherited_q50.cnv.map ]\n",
      "Wrote 868 unique positions to file\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:43 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_clean.log ]\n",
      "Analysis started: Fri Jan 19 17:02:43 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-exclude bad_regions.list\n",
      "\t--cnv-overlap .5\n",
      "\t--cnv-write\n",
      "\t--out inherited_q50_clean\n",
      "\n",
      "Reading marker information from [ inherited_q50.cnv.map ]\n",
      "868 (of 868) markers to be included from [ inherited_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading CNV intersection list from [ bad_regions.list ]\n",
      "Read 380994 ranges to exclude from CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50.cnv ]\n",
      "434 mapped to a person, of which 434 passed filters\n",
      "5 kept after excluding specific regions\n",
      "5 of 434 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        2 / 2\n",
      "     3        1 / 0\n",
      "\n",
      "Writing new CNV list to [ inherited_q50_clean.cnv ]\n",
      "Writing new FAM file to [ inherited_q50_clean.fam ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:51 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_clean.log ]\n",
      "Analysis started: Fri Jan 19 17:02:51 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list inherited_q50_clean.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--out inherited_q50_clean\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50_clean.cnv ]\n",
      "Writing new MAP file to [ inherited_q50_clean.cnv.map ]\n",
      "Wrote 15 unique positions to file\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:51 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_burden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:51 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--out inherited_q50_burden\n",
      "\n",
      "Reading marker information from [ inherited_q50.cnv.map ]\n",
      "868 (of 868) markers to be included from [ inherited_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50.cnv ]\n",
      "No overlapping samples found\n",
      "434 mapped to a person, of which 434 passed filters\n",
      "434 of 434 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    119 / 124\n",
      "     3      92 / 99\n",
      "\n",
      "Writing per-individual summary to [ inherited_q50_burden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:51 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_genes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:51 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--out inherited_q50_genes\n",
      "\n",
      "Reading marker information from [ inherited_q50.cnv.map ]\n",
      "868 (of 868) markers to be included from [ inherited_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q50_genes.reg ]\n",
      "434 mapped to a person, of which 434 passed filters\n",
      "338 intersected with one or more specified region\n",
      "338 of 434 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      94 / 96\n",
      "     3      71 / 77\n",
      "\n",
      "Writing per-individual summary to [ inherited_q50_genes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:52 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_brainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:52 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--out inherited_q50_brainGenes\n",
      "\n",
      "Reading marker information from [ inherited_q50.cnv.map ]\n",
      "868 (of 868) markers to be included from [ inherited_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_q50_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q50_brainGenes.reg ]\n",
      "434 mapped to a person, of which 434 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 434 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        0 / 1\n",
      "\n",
      "Writing per-individual summary to [ inherited_q50_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:52 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_delBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:52 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--out inherited_q50_delBurden\n",
      "\n",
      "Reading marker information from [ inherited_q50.cnv.map ]\n",
      "868 (of 868) markers to be included from [ inherited_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50.cnv ]\n",
      "434 mapped to a person, of which 243 passed filters\n",
      "243 of 434 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    119 / 124\n",
      "\n",
      "Writing per-individual summary to [ inherited_q50_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:52 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_delGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:52 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--out inherited_q50_delGenes\n",
      "\n",
      "Reading marker information from [ inherited_q50.cnv.map ]\n",
      "868 (of 868) markers to be included from [ inherited_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q50_delGenes.reg ]\n",
      "434 mapped to a person, of which 243 passed filters\n",
      "190 intersected with one or more specified region\n",
      "190 of 434 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      94 / 96\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Writing per-individual summary to [ inherited_q50_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:53 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_delBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:53 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--out inherited_q50_delBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited_q50.cnv.map ]\n",
      "868 (of 868) markers to be included from [ inherited_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_q50_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q50_delBrainGenes.reg ]\n",
      "434 mapped to a person, of which 243 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 434 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_q50_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:53 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_dupBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:53 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--out inherited_q50_dupBurden\n",
      "\n",
      "Reading marker information from [ inherited_q50.cnv.map ]\n",
      "868 (of 868) markers to be included from [ inherited_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50.cnv ]\n",
      "434 mapped to a person, of which 191 passed filters\n",
      "191 of 434 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      92 / 99\n",
      "\n",
      "Writing per-individual summary to [ inherited_q50_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:53 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_dupGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:53 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--out inherited_q50_dupGenes\n",
      "\n",
      "Reading marker information from [ inherited_q50.cnv.map ]\n",
      "868 (of 868) markers to be included from [ inherited_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q50_dupGenes.reg ]\n",
      "434 mapped to a person, of which 191 passed filters\n",
      "148 intersected with one or more specified region\n",
      "148 of 434 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      71 / 77\n",
      "\n",
      "Writing per-individual summary to [ inherited_q50_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:53 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_dupBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:53 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--out inherited_q50_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited_q50.cnv.map ]\n",
      "868 (of 868) markers to be included from [ inherited_q50.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_q50_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q50_dupBrainGenes.reg ]\n",
      "434 mapped to a person, of which 191 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 434 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        0 / 1\n",
      "\n",
      "Writing per-individual summary to [ inherited_q50_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_dupBrainGenes.cnv.summary ]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Analysis finished: Fri Jan 19 17:02:53 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_clean_burden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:53 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--out inherited_q50_clean_burden\n",
      "\n",
      "Reading marker information from [ inherited_q50_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50_clean.cnv ]\n",
      "No overlapping samples found\n",
      "5 mapped to a person, of which 5 passed filters\n",
      "5 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        2 / 2\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ inherited_q50_clean_burden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_clean_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:53 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_clean_genes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:53 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--out inherited_q50_clean_genes\n",
      "\n",
      "Reading marker information from [ inherited_q50_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q50_clean_genes.reg ]\n",
      "5 mapped to a person, of which 5 passed filters\n",
      "4 intersected with one or more specified region\n",
      "4 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 2\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ inherited_q50_clean_genes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_clean_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_clean_brainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--out inherited_q50_clean_brainGenes\n",
      "\n",
      "Reading marker information from [ inherited_q50_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_q50_clean_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q50_clean_brainGenes.reg ]\n",
      "5 mapped to a person, of which 5 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_q50_clean_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_clean_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_clean_delBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--out inherited_q50_clean_delBurden\n",
      "\n",
      "Reading marker information from [ inherited_q50_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50_clean.cnv ]\n",
      "5 mapped to a person, of which 4 passed filters\n",
      "4 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        2 / 2\n",
      "\n",
      "Writing per-individual summary to [ inherited_q50_clean_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_clean_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_clean_delGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--out inherited_q50_clean_delGenes\n",
      "\n",
      "Reading marker information from [ inherited_q50_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q50_clean_delGenes.reg ]\n",
      "5 mapped to a person, of which 4 passed filters\n",
      "3 intersected with one or more specified region\n",
      "3 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 2\n",
      "\n",
      "Writing per-individual summary to [ inherited_q50_clean_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_clean_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_clean_delBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--out inherited_q50_clean_delBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited_q50_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_q50_clean_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q50_clean_delBrainGenes.reg ]\n",
      "5 mapped to a person, of which 4 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_q50_clean_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_clean_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_clean_dupBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--out inherited_q50_clean_dupBurden\n",
      "\n",
      "Reading marker information from [ inherited_q50_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50_clean.cnv ]\n",
      "5 mapped to a person, of which 1 passed filters\n",
      "1 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ inherited_q50_clean_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_clean_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q50_clean_dupGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--out inherited_q50_clean_dupGenes\n",
      "\n",
      "Reading marker information from [ inherited_q50_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q50_clean_dupGenes.reg ]\n",
      "5 mapped to a person, of which 1 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ inherited_q50_clean_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_clean_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Writing this text to log file [ inherited_q50_clean_dupBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q50_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q50_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--out inherited_q50_clean_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited_q50_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q50_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_q50_clean_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q50_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q50_clean_dupBrainGenes.reg ]\n",
      "5 mapped to a person, of which 1 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_q50_clean_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q50_clean_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60.log ]\n",
      "Analysis started: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list all_q60.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--out all_q60\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60.cnv ]\n",
      "Writing new MAP file to [ all_q60.cnv.map ]\n",
      "Wrote 10101 unique positions to file\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_clean.log ]\n",
      "Analysis started: Fri Jan 19 17:02:54 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-exclude bad_regions.list\n",
      "\t--cnv-overlap .5\n",
      "\t--cnv-write\n",
      "\t--out all_q60_clean\n",
      "\n",
      "Reading marker information from [ all_q60.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading CNV intersection list from [ bad_regions.list ]\n",
      "Read 380994 ranges to exclude from CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60.cnv ]\n",
      "6924 mapped to a person, of which 6924 passed filters\n",
      "170 kept after excluding specific regions\n",
      "170 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      78 / 25\n",
      "     3      25 / 42\n",
      "\n",
      "Writing new CNV list to [ all_q60_clean.cnv ]\n",
      "Writing new FAM file to [ all_q60_clean.fam ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:41 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_clean.log ]\n",
      "Analysis started: Fri Jan 19 17:04:41 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list all_q60_clean.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--out all_q60_clean\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60_clean.cnv ]\n",
      "Writing new MAP file to [ all_q60_clean.cnv.map ]\n",
      "Wrote 429 unique positions to file\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:41 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_burden.log ]\n",
      "Analysis started: Fri Jan 19 17:04:41 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--out all_q60_burden\n",
      "\n",
      "Reading marker information from [ all_q60.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60.cnv ]\n",
      "No overlapping samples found\n",
      "6924 mapped to a person, of which 6924 passed filters\n",
      "6924 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1  1698 / 1747\n",
      "     3  1532 / 1947\n",
      "\n",
      "Writing per-individual summary to [ all_q60_burden.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:41 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_genes.log ]\n",
      "Analysis started: Fri Jan 19 17:04:41 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--out all_q60_genes\n",
      "\n",
      "Reading marker information from [ all_q60.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60.cnv ]\n",
      "Writing back list to intersected regions to [ all_q60_genes.reg ]\n",
      "6924 mapped to a person, of which 6924 passed filters\n",
      "5942 intersected with one or more specified region\n",
      "5942 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1  1474 / 1481\n",
      "     3  1297 / 1690\n",
      "\n",
      "Writing per-individual summary to [ all_q60_genes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:47 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_brainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:04:47 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--out all_q60_brainGenes\n",
      "\n",
      "Reading marker information from [ all_q60.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_q60_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60.cnv ]\n",
      "Writing back list to intersected regions to [ all_q60_brainGenes.reg ]\n",
      "6924 mapped to a person, of which 6924 passed filters\n",
      "54 intersected with one or more specified region\n",
      "54 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      14 / 11\n",
      "     3      13 / 16\n",
      "\n",
      "Writing per-individual summary to [ all_q60_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:48 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_delBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:04:48 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--out all_q60_delBurden\n",
      "\n",
      "Reading marker information from [ all_q60.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60.cnv ]\n",
      "6924 mapped to a person, of which 3445 passed filters\n",
      "3445 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1  1698 / 1747\n",
      "\n",
      "Writing per-individual summary to [ all_q60_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:48 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_delGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:04:48 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--out all_q60_delGenes\n",
      "\n",
      "Reading marker information from [ all_q60.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60.cnv ]\n",
      "Writing back list to intersected regions to [ all_q60_delGenes.reg ]\n",
      "6924 mapped to a person, of which 3445 passed filters\n",
      "2955 intersected with one or more specified region\n",
      "2955 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1  1474 / 1481\n",
      "\n",
      "Writing per-individual summary to [ all_q60_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:51 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_delBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:04:51 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--out all_q60_delBrainGenes\n",
      "\n",
      "Reading marker information from [ all_q60.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_q60_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60.cnv ]\n",
      "Writing back list to intersected regions to [ all_q60_delBrainGenes.reg ]\n",
      "6924 mapped to a person, of which 3445 passed filters\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "25 intersected with one or more specified region\n",
      "25 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      14 / 11\n",
      "\n",
      "Writing per-individual summary to [ all_q60_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:52 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_dupBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:04:52 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--out all_q60_dupBurden\n",
      "\n",
      "Reading marker information from [ all_q60.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60.cnv ]\n",
      "6924 mapped to a person, of which 3479 passed filters\n",
      "3479 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3  1532 / 1947\n",
      "\n",
      "Writing per-individual summary to [ all_q60_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:52 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_dupGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:04:52 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--out all_q60_dupGenes\n",
      "\n",
      "Reading marker information from [ all_q60.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60.cnv ]\n",
      "Writing back list to intersected regions to [ all_q60_dupGenes.reg ]\n",
      "6924 mapped to a person, of which 3479 passed filters\n",
      "2987 intersected with one or more specified region\n",
      "2987 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3  1297 / 1690\n",
      "\n",
      "Writing per-individual summary to [ all_q60_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:55 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_dupBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:04:55 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--out all_q60_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ all_q60.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_q60_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60.cnv ]\n",
      "Writing back list to intersected regions to [ all_q60_dupBrainGenes.reg ]\n",
      "6924 mapped to a person, of which 3479 passed filters\n",
      "29 intersected with one or more specified region\n",
      "29 of 6924 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      13 / 16\n",
      "\n",
      "Writing per-individual summary to [ all_q60_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:55 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_clean_burden.log ]\n",
      "Analysis started: Fri Jan 19 17:04:55 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--out all_q60_clean_burden\n",
      "\n",
      "Reading marker information from [ all_q60_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60_clean.cnv ]\n",
      "No overlapping samples found\n",
      "170 mapped to a person, of which 170 passed filters\n",
      "170 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      78 / 25\n",
      "     3      25 / 42\n",
      "\n",
      "Writing per-individual summary to [ all_q60_clean_burden.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_clean_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:55 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_clean_genes.log ]\n",
      "Analysis started: Fri Jan 19 17:04:55 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--out all_q60_clean_genes\n",
      "\n",
      "Reading marker information from [ all_q60_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_q60_clean_genes.reg ]\n",
      "170 mapped to a person, of which 170 passed filters\n",
      "165 intersected with one or more specified region\n",
      "165 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      75 / 25\n",
      "     3      24 / 41\n",
      "\n",
      "Writing per-individual summary to [ all_q60_clean_genes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_clean_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:56 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_clean_brainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:04:56 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--out all_q60_clean_brainGenes\n",
      "\n",
      "Reading marker information from [ all_q60_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_q60_clean_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_q60_clean_brainGenes.reg ]\n",
      "170 mapped to a person, of which 170 passed filters\n",
      "4 intersected with one or more specified region\n",
      "4 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        2 / 1\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ all_q60_clean_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_clean_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:56 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_clean_delBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:04:56 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--out all_q60_clean_delBurden\n",
      "\n",
      "Reading marker information from [ all_q60_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60_clean.cnv ]\n",
      "170 mapped to a person, of which 103 passed filters\n",
      "103 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      78 / 25\n",
      "\n",
      "Writing per-individual summary to [ all_q60_clean_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_clean_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:56 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_clean_delGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:04:56 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--out all_q60_clean_delGenes\n",
      "\n",
      "Reading marker information from [ all_q60_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_q60_clean_delGenes.reg ]\n",
      "170 mapped to a person, of which 103 passed filters\n",
      "100 intersected with one or more specified region\n",
      "100 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      75 / 25\n",
      "\n",
      "Writing per-individual summary to [ all_q60_clean_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_clean_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:56 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_clean_delBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:04:56 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--out all_q60_clean_delBrainGenes\n",
      "\n",
      "Reading marker information from [ all_q60_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_q60_clean_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_q60_clean_delBrainGenes.reg ]\n",
      "170 mapped to a person, of which 103 passed filters\n",
      "3 intersected with one or more specified region\n",
      "3 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        2 / 1\n",
      "\n",
      "Writing per-individual summary to [ all_q60_clean_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_clean_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:56 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_clean_dupBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:04:56 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--out all_q60_clean_dupBurden\n",
      "\n",
      "Reading marker information from [ all_q60_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60_clean.cnv ]\n",
      "170 mapped to a person, of which 67 passed filters\n",
      "67 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      25 / 42\n",
      "\n",
      "Writing per-individual summary to [ all_q60_clean_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_clean_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:56 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_clean_dupGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:04:56 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--out all_q60_clean_dupGenes\n",
      "\n",
      "Reading marker information from [ all_q60_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_q60_clean_dupGenes.reg ]\n",
      "170 mapped to a person, of which 67 passed filters\n",
      "65 intersected with one or more specified region\n",
      "65 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      24 / 41\n",
      "\n",
      "Writing per-individual summary to [ all_q60_clean_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_clean_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:57 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q60_clean_dupBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:04:57 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--out all_q60_clean_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ all_q60_clean.cnv.map ]\n",
      "429 (of 429) markers to be included from [ all_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ all_q60_clean_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ all_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ all_q60_clean_dupBrainGenes.reg ]\n",
      "170 mapped to a person, of which 67 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 170 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ all_q60_clean_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ all_q60_clean_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:57 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60.log ]\n",
      "Analysis started: Fri Jan 19 17:04:57 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list denovo_q60.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--out denovo_q60\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60.cnv ]\n",
      "Writing new MAP file to [ denovo_q60.cnv.map ]\n",
      "Wrote 1683 unique positions to file\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:04:57 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_clean.log ]\n",
      "Analysis started: Fri Jan 19 17:04:57 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-exclude bad_regions.list\n",
      "\t--cnv-overlap .5\n",
      "\t--cnv-write\n",
      "\t--out denovo_q60_clean\n",
      "\n",
      "Reading marker information from [ denovo_q60.cnv.map ]\n",
      "1683 (of 1683) markers to be included from [ denovo_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading CNV intersection list from [ bad_regions.list ]\n",
      "Read 380994 ranges to exclude from CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60.cnv ]\n",
      "644 mapped to a person, of which 644 passed filters\n",
      "51 kept after excluding specific regions\n",
      "51 of 644 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1       38 / 2\n",
      "     3        7 / 4\n",
      "\n",
      "Writing new CNV list to [ denovo_q60_clean.cnv ]\n",
      "Writing new FAM file to [ denovo_q60_clean.fam ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:08 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_clean.log ]\n",
      "Analysis started: Fri Jan 19 17:05:08 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list denovo_q60_clean.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--out denovo_q60_clean\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60_clean.cnv ]\n",
      "Writing new MAP file to [ denovo_q60_clean.cnv.map ]\n",
      "Wrote 143 unique positions to file\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:08 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_burden.log ]\n",
      "Analysis started: Fri Jan 19 17:05:08 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--out denovo_q60_burden\n",
      "\n",
      "Reading marker information from [ denovo_q60.cnv.map ]\n",
      "1683 (of 1683) markers to be included from [ denovo_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60.cnv ]\n",
      "No overlapping samples found\n",
      "644 mapped to a person, of which 644 passed filters\n",
      "644 of 644 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    243 / 139\n",
      "     3    119 / 143\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_burden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:08 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_genes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:08 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--out denovo_q60_genes\n",
      "\n",
      "Reading marker information from [ denovo_q60.cnv.map ]\n",
      "1683 (of 1683) markers to be included from [ denovo_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q60_genes.reg ]\n",
      "644 mapped to a person, of which 644 passed filters\n",
      "566 intersected with one or more specified region\n",
      "566 of 644 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    222 / 117\n",
      "     3    103 / 124\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_genes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:09 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_brainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:09 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--out denovo_q60_brainGenes\n",
      "\n",
      "Reading marker information from [ denovo_q60.cnv.map ]\n",
      "1683 (of 1683) markers to be included from [ denovo_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_q60_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q60_brainGenes.reg ]\n",
      "644 mapped to a person, of which 644 passed filters\n",
      "7 intersected with one or more specified region\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "7 of 644 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        3 / 3\n",
      "     3        0 / 1\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:09 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_delBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:05:09 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--out denovo_q60_delBurden\n",
      "\n",
      "Reading marker information from [ denovo_q60.cnv.map ]\n",
      "1683 (of 1683) markers to be included from [ denovo_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60.cnv ]\n",
      "644 mapped to a person, of which 382 passed filters\n",
      "382 of 644 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    243 / 139\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:09 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_delGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:09 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--out denovo_q60_delGenes\n",
      "\n",
      "Reading marker information from [ denovo_q60.cnv.map ]\n",
      "1683 (of 1683) markers to be included from [ denovo_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q60_delGenes.reg ]\n",
      "644 mapped to a person, of which 382 passed filters\n",
      "339 intersected with one or more specified region\n",
      "339 of 644 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    222 / 117\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:10 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_delBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:10 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--out denovo_q60_delBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo_q60.cnv.map ]\n",
      "1683 (of 1683) markers to be included from [ denovo_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_q60_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q60_delBrainGenes.reg ]\n",
      "644 mapped to a person, of which 382 passed filters\n",
      "6 intersected with one or more specified region\n",
      "6 of 644 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        3 / 3\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:10 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_dupBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:05:10 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--out denovo_q60_dupBurden\n",
      "\n",
      "Reading marker information from [ denovo_q60.cnv.map ]\n",
      "1683 (of 1683) markers to be included from [ denovo_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60.cnv ]\n",
      "644 mapped to a person, of which 262 passed filters\n",
      "262 of 644 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3    119 / 143\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:10 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_dupGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:10 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--out denovo_q60_dupGenes\n",
      "\n",
      "Reading marker information from [ denovo_q60.cnv.map ]\n",
      "1683 (of 1683) markers to be included from [ denovo_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q60_dupGenes.reg ]\n",
      "644 mapped to a person, of which 262 passed filters\n",
      "227 intersected with one or more specified region\n",
      "227 of 644 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3    103 / 124\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:10 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_dupBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:10 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--out denovo_q60_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo_q60.cnv.map ]\n",
      "1683 (of 1683) markers to be included from [ denovo_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_q60_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q60_dupBrainGenes.reg ]\n",
      "644 mapped to a person, of which 262 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 644 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        0 / 1\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:10 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_clean_burden.log ]\n",
      "Analysis started: Fri Jan 19 17:05:10 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--out denovo_q60_clean_burden\n",
      "\n",
      "Reading marker information from [ denovo_q60_clean.cnv.map ]\n",
      "143 (of 143) markers to be included from [ denovo_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60_clean.cnv ]\n",
      "No overlapping samples found\n",
      "51 mapped to a person, of which 51 passed filters\n",
      "51 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1       38 / 2\n",
      "     3        7 / 4\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_clean_burden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_clean_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:10 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_clean_genes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:10 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--out denovo_q60_clean_genes\n",
      "\n",
      "Reading marker information from [ denovo_q60_clean.cnv.map ]\n",
      "143 (of 143) markers to be included from [ denovo_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q60_clean_genes.reg ]\n",
      "51 mapped to a person, of which 51 passed filters\n",
      "48 intersected with one or more specified region\n",
      "48 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1       36 / 2\n",
      "     3        7 / 3\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_clean_genes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_clean_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_clean_brainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60_clean.cnv\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--out denovo_q60_clean_brainGenes\n",
      "\n",
      "Reading marker information from [ denovo_q60_clean.cnv.map ]\n",
      "143 (of 143) markers to be included from [ denovo_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_q60_clean_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q60_clean_brainGenes.reg ]\n",
      "51 mapped to a person, of which 51 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_clean_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_clean_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_clean_delBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--out denovo_q60_clean_delBurden\n",
      "\n",
      "Reading marker information from [ denovo_q60_clean.cnv.map ]\n",
      "143 (of 143) markers to be included from [ denovo_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60_clean.cnv ]\n",
      "51 mapped to a person, of which 40 passed filters\n",
      "40 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1       38 / 2\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_clean_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_clean_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_clean_delGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--out denovo_q60_clean_delGenes\n",
      "\n",
      "Reading marker information from [ denovo_q60_clean.cnv.map ]\n",
      "143 (of 143) markers to be included from [ denovo_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q60_clean_delGenes.reg ]\n",
      "51 mapped to a person, of which 40 passed filters\n",
      "38 intersected with one or more specified region\n",
      "38 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1       36 / 2\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_clean_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_clean_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_clean_delBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--out denovo_q60_clean_delBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo_q60_clean.cnv.map ]\n",
      "143 (of 143) markers to be included from [ denovo_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_q60_clean_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q60_clean_delBrainGenes.reg ]\n",
      "51 mapped to a person, of which 40 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_clean_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_clean_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_clean_dupBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--out denovo_q60_clean_dupBurden\n",
      "\n",
      "Reading marker information from [ denovo_q60_clean.cnv.map ]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "143 (of 143) markers to be included from [ denovo_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60_clean.cnv ]\n",
      "51 mapped to a person, of which 11 passed filters\n",
      "11 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        7 / 4\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_clean_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_clean_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_clean_dupGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--out denovo_q60_clean_dupGenes\n",
      "\n",
      "Reading marker information from [ denovo_q60_clean.cnv.map ]\n",
      "143 (of 143) markers to be included from [ denovo_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q60_clean_dupGenes.reg ]\n",
      "51 mapped to a person, of which 11 passed filters\n",
      "10 intersected with one or more specified region\n",
      "10 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        7 / 3\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_clean_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_clean_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ denovo_q60_clean_dupBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map denovo_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list denovo_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--out denovo_q60_clean_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ denovo_q60_clean.cnv.map ]\n",
      "143 (of 143) markers to be included from [ denovo_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ denovo_q60_clean_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ denovo_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ denovo_q60_clean_dupBrainGenes.reg ]\n",
      "51 mapped to a person, of which 11 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 51 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ denovo_q60_clean_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ denovo_q60_clean_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60.log ]\n",
      "Analysis started: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list inherited_q60.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--out inherited_q60\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60.cnv ]\n",
      "Writing new MAP file to [ inherited_q60.cnv.map ]\n",
      "Wrote 732 unique positions to file\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_clean.log ]\n",
      "Analysis started: Fri Jan 19 17:05:11 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-exclude bad_regions.list\n",
      "\t--cnv-overlap .5\n",
      "\t--cnv-write\n",
      "\t--out inherited_q60_clean\n",
      "\n",
      "Reading marker information from [ inherited_q60.cnv.map ]\n",
      "732 (of 732) markers to be included from [ inherited_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading CNV intersection list from [ bad_regions.list ]\n",
      "Read 380994 ranges to exclude from CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60.cnv ]\n",
      "366 mapped to a person, of which 366 passed filters\n",
      "5 kept after excluding specific regions\n",
      "5 of 366 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        2 / 2\n",
      "     3        1 / 0\n",
      "\n",
      "Writing new CNV list to [ inherited_q60_clean.cnv ]\n",
      "Writing new FAM file to [ inherited_q60_clean.fam ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:19 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_clean.log ]\n",
      "Analysis started: Fri Jan 19 17:05:19 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list inherited_q60_clean.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--out inherited_q60_clean\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60_clean.cnv ]\n",
      "Writing new MAP file to [ inherited_q60_clean.cnv.map ]\n",
      "Wrote 15 unique positions to file\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:19 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_burden.log ]\n",
      "Analysis started: Fri Jan 19 17:05:19 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--out inherited_q60_burden\n",
      "\n",
      "Reading marker information from [ inherited_q60.cnv.map ]\n",
      "732 (of 732) markers to be included from [ inherited_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60.cnv ]\n",
      "No overlapping samples found\n",
      "366 mapped to a person, of which 366 passed filters\n",
      "366 of 366 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    106 / 107\n",
      "     3      76 / 77\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_burden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:19 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_genes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:19 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--out inherited_q60_genes\n",
      "\n",
      "Reading marker information from [ inherited_q60.cnv.map ]\n",
      "732 (of 732) markers to be included from [ inherited_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q60_genes.reg ]\n",
      "366 mapped to a person, of which 366 passed filters\n",
      "290 intersected with one or more specified region\n",
      "290 of 366 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      85 / 87\n",
      "     3      60 / 58\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_genes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:20 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_brainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:20 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--out inherited_q60_brainGenes\n",
      "\n",
      "Reading marker information from [ inherited_q60.cnv.map ]\n",
      "732 (of 732) markers to be included from [ inherited_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_q60_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q60_brainGenes.reg ]\n",
      "366 mapped to a person, of which 366 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 366 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        0 / 1\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:20 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_delBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:05:20 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--out inherited_q60_delBurden\n",
      "\n",
      "Reading marker information from [ inherited_q60.cnv.map ]\n",
      "732 (of 732) markers to be included from [ inherited_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60.cnv ]\n",
      "366 mapped to a person, of which 213 passed filters\n",
      "213 of 366 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1    106 / 107\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:20 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_delGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:20 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--out inherited_q60_delGenes\n",
      "\n",
      "Reading marker information from [ inherited_q60.cnv.map ]\n",
      "732 (of 732) markers to be included from [ inherited_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q60_delGenes.reg ]\n",
      "366 mapped to a person, of which 213 passed filters\n",
      "172 intersected with one or more specified region\n",
      "172 of 366 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1      85 / 87\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_delBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--out inherited_q60_delBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited_q60.cnv.map ]\n",
      "732 (of 732) markers to be included from [ inherited_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_q60_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q60_delBrainGenes.reg ]\n",
      "366 mapped to a person, of which 213 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 366 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_dupBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--out inherited_q60_dupBurden\n",
      "\n",
      "Reading marker information from [ inherited_q60.cnv.map ]\n",
      "732 (of 732) markers to be included from [ inherited_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60.cnv ]\n",
      "366 mapped to a person, of which 153 passed filters\n",
      "153 of 366 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      76 / 77\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_dupGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--out inherited_q60_dupGenes\n",
      "\n",
      "Reading marker information from [ inherited_q60.cnv.map ]\n",
      "732 (of 732) markers to be included from [ inherited_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q60_dupGenes.reg ]\n",
      "366 mapped to a person, of which 153 passed filters\n",
      "118 intersected with one or more specified region\n",
      "118 of 366 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3      60 / 58\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_dupBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--out inherited_q60_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited_q60.cnv.map ]\n",
      "732 (of 732) markers to be included from [ inherited_q60.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_q60_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q60_dupBrainGenes.reg ]\n",
      "366 mapped to a person, of which 153 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 366 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        0 / 1\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_clean_burden.log ]\n",
      "Analysis started: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-check-no-overlap\n",
      "\t--out inherited_q60_clean_burden\n",
      "\n",
      "Reading marker information from [ inherited_q60_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60_clean.cnv ]\n",
      "No overlapping samples found\n",
      "5 mapped to a person, of which 5 passed filters\n",
      "5 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        2 / 2\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_clean_burden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_clean_burden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_clean_genes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--out inherited_q60_clean_genes\n",
      "\n",
      "Reading marker information from [ inherited_q60_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q60_clean_genes.reg ]\n",
      "5 mapped to a person, of which 5 passed filters\n",
      "4 intersected with one or more specified region\n",
      "4 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 2\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_clean_genes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_clean_genes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_clean_brainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--out inherited_q60_clean_brainGenes\n",
      "\n",
      "Reading marker information from [ inherited_q60_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_q60_clean_brainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q60_clean_brainGenes.reg ]\n",
      "5 mapped to a person, of which 5 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_clean_brainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_clean_brainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_clean_delBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-del\n",
      "\t--out inherited_q60_clean_delBurden\n",
      "\n",
      "Reading marker information from [ inherited_q60_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60_clean.cnv ]\n",
      "5 mapped to a person, of which 4 passed filters\n",
      "4 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        2 / 2\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_clean_delBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_clean_delBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_clean_delGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:21 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-del\n",
      "\t--out inherited_q60_clean_delGenes\n",
      "\n",
      "Reading marker information from [ inherited_q60_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q60_clean_delGenes.reg ]\n",
      "5 mapped to a person, of which 4 passed filters\n",
      "3 intersected with one or more specified region\n",
      "3 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     1        1 / 2\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_clean_delGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_clean_delGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:22 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_clean_delBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:22 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-del\n",
      "\t--out inherited_q60_clean_delBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited_q60_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_q60_clean_delBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q60_clean_delBrainGenes.reg ]\n",
      "5 mapped to a person, of which 4 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_clean_delBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_clean_delBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:22 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_clean_dupBurden.log ]\n",
      "Analysis started: Fri Jan 19 17:05:22 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-dup\n",
      "\t--out inherited_q60_clean_dupBurden\n",
      "\n",
      "Reading marker information from [ inherited_q60_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60_clean.cnv ]\n",
      "5 mapped to a person, of which 1 passed filters\n",
      "1 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_clean_dupBurden.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_clean_dupBurden.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:22 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_clean_dupGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:22 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset genes.txt\n",
      "\t--cnv-dup\n",
      "\t--out inherited_q60_clean_dupGenes\n",
      "\n",
      "Reading marker information from [ inherited_q60_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q60_clean.cnv.map ]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ genes.txt ]\n",
      "Looking for subset of 25303 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 25646 ranges to intersect with CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q60_clean_dupGenes.reg ]\n",
      "5 mapped to a person, of which 1 passed filters\n",
      "1 intersected with one or more specified region\n",
      "1 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "     3        1 / 0\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_clean_dupGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_clean_dupGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:22 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ inherited_q60_clean_dupBrainGenes.log ]\n",
      "Analysis started: Fri Jan 19 17:05:22 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map inherited_q60_clean.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list inherited_q60_clean.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-intersect glist-hg19\n",
      "\t--cnv-verbose-report-regions\n",
      "\t--cnv-subset hg19_allenBrainGene_trimmed.txt\n",
      "\t--cnv-dup\n",
      "\t--out inherited_q60_clean_dupBrainGenes\n",
      "\n",
      "Reading marker information from [ inherited_q60_clean.cnv.map ]\n",
      "15 (of 15) markers to be included from [ inherited_q60_clean.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading intersection subset list from [ hg19_allenBrainGene_trimmed.txt ]\n",
      "Looking for subset of 606 ranges\n",
      "Reading CNV intersection list from [ glist-hg19 ]\n",
      "Read 577 ranges to intersect with CNV list\n",
      "Could not find 29 ranges\n",
      "Writing this list to [ inherited_q60_clean_dupBrainGenes.notfound ] \n",
      "\n",
      "Reading segment list (CNVs) from [ inherited_q60_clean.cnv ]\n",
      "Writing back list to intersected regions to [ inherited_q60_clean_dupBrainGenes.reg ]\n",
      "5 mapped to a person, of which 1 passed filters\n",
      "0 intersected with one or more specified region\n",
      "0 of 5 mapped as valid segments\n",
      " CopyN Case/Control\n",
      "\n",
      "Writing per-individual summary to [ inherited_q60_clean_dupBrainGenes.cnv.indiv ]\n",
      "Writing positional summary to [ inherited_q60_clean_dupBrainGenes.cnv.summary ]\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:22 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q70.log ]\n",
      "Analysis started: Fri Jan 19 17:05:22 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--cnv-list all_q70.cnv\n",
      "\t--cnv-make-map\n",
      "\t--noweb\n",
      "\t--out all_q70\n",
      "\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q70.cnv ]\n",
      "Writing new MAP file to [ all_q70.cnv.map ]\n",
      "Wrote 10101 unique positions to file\n",
      "\n",
      "Analysis finished: Fri Jan 19 17:05:22 2018\n",
      "\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ all_q70_clean.log ]\n",
      "Analysis started: Fri Jan 19 17:05:22 2018\n",
      "\n",
      "Options in effect:\n",
      "\t--map all_q70.cnv.map\n",
      "\t--fam ../simplex_nofamid.ped\n",
      "\t--cnv-list all_q70.cnv\n",
      "\t--noweb\n",
      "\t--1\n",
      "\t--cnv-exclude bad_regions.list\n",
      "\t--cnv-overlap .5\n",
      "\t--cnv-write\n",
      "\t--out all_q70_clean\n",
      "\n",
      "Reading marker information from [ all_q70.cnv.map ]\n",
      "10101 (of 10101) markers to be included from [ all_q70.cnv.map ]\n",
      "Reading individual information from [ ../simplex_nofamid.ped ]\n",
      "Reading pedigree information from [ ../simplex_nofamid.ped ] \n",
      "99 individuals read from [ ../simplex_nofamid.ped ] \n",
      "99 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (0=unaff, 1=aff, other=miss)\n",
      "21 cases, 78 controls and 0 missing\n",
      "56 males, 43 females, and 0 of unspecified sex\n",
      "Reading CNV intersection list from [ bad_regions.list ]\n",
      "Read 380994 ranges to exclude from CNV list\n",
      "\n",
      "Reading segment list (CNVs) from [ all_q70.cnv ]\n"
     ]
    }
   ],
   "source": [
    "for q in 50 60 70 80 90; do\n",
    "    for cnvtype in all denovo inherited; do\n",
    "        cnvname=${cnvtype}_q${q}.cnv\n",
    "        plink --cnv-list $cnvname --cnv-make-map --noweb --out ${cnvtype}_q${q};\n",
    "        \n",
    "        # remove bad regions\n",
    "        plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "            --noweb --1 --cnv-exclude bad_regions.list --cnv-overlap .5 \\\n",
    "            --cnv-write --out ${cnvtype}_q${q}_clean\n",
    "        plink --cnv-list ${cnvtype}_q${q}_clean.cnv --cnv-make-map --noweb --1 \\\n",
    "            --out ${cnvtype}_q${q}_clean\n",
    "        \n",
    "        for qc in '' '_clean'; do\n",
    "            cnvname=${cnvtype}_q${q}${qc}.cnv\n",
    "            # whole burden\n",
    "            plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "                --noweb --1 --cnv-check-no-overlap --out ${cnvtype}_q${q}${qc}_burden;\n",
    "            # gene sets\n",
    "            plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "                --noweb --1 --cnv-intersect glist-hg19 --cnv-verbose-report-regions \\\n",
    "                --cnv-subset genes.txt --out ${cnvtype}_q${q}${qc}_genes;\n",
    "            plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "                --noweb --1 --cnv-intersect glist-hg19 --cnv-verbose-report-regions \\\n",
    "                --cnv-subset hg19_allenBrainGene_trimmed.txt \\\n",
    "                --out ${cnvtype}_q${q}${qc}_brainGenes;\n",
    "            # subtypes only\n",
    "            for sub in del dup; do\n",
    "                plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "                --noweb --1 --cnv-${sub} --out ${cnvtype}_q${q}${qc}_${sub}Burden;\n",
    "                # gene sets\n",
    "                plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "                    --noweb --1 --cnv-intersect glist-hg19 --cnv-verbose-report-regions \\\n",
    "                    --cnv-subset genes.txt --cnv-${sub} \\\n",
    "                    --out ${cnvtype}_q${q}${qc}_${sub}Genes;\n",
    "                plink --map ${cnvname}.map --fam ../simplex_nofamid.ped --cnv-list $cnvname \\\n",
    "                    --noweb --1 --cnv-intersect glist-hg19 --cnv-verbose-report-regions \\\n",
    "                    --cnv-subset hg19_allenBrainGene_trimmed.txt --cnv-${sub} \\\n",
    "                    --out ${cnvtype}_q${q}${qc}_${sub}BrainGenes;\n",
    "            done;\n",
    "        done;\n",
    "    done;\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<script>requirejs.config({paths: { 'plotly': ['https://cdn.plot.ly/plotly-latest.min']},});if(!window.Plotly) {{require(['plotly'],function(plotly) {window.Plotly=plotly;});}}</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<script>requirejs.config({paths: { 'plotly': ['https://cdn.plot.ly/plotly-latest.min']},});if(!window.Plotly) {{require(['plotly'],function(plotly) {window.Plotly=plotly;});}}</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import numpy as np\n",
    "import matplotlib.pyplot as plt\n",
    "import seaborn as sns\n",
    "import pandas as pd\n",
    "from plotly.offline import download_plotlyjs, init_notebook_mode, plot, iplot\n",
    "\n",
    "init_notebook_mode(connected=True)\n",
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Working with 88 samples, in 20 families\n"
     ]
    }
   ],
   "source": [
    "import glob\n",
    "ped_file = '/data/sudregp/cnv/simplex.ped'\n",
    "wes_prefix = ['CLIA', 'CCGO', 'WPS']\n",
    "\n",
    "# cleaning up the data for this analysis\n",
    "fid = open('/home/sudregp/data/cnv/exclude.txt', 'r')\n",
    "exclude_list = [line.rstrip() for line in fid]\n",
    "fid.close()\n",
    "fid = open('/home/sudregp/data/cnv/exclude_multikids.txt', 'r')\n",
    "exclude_list += [line.rstrip() for line in fid]\n",
    "fid.close()\n",
    "\n",
    "trios = []\n",
    "affected = []\n",
    "controls = []\n",
    "samples = []\n",
    "famids = []\n",
    "fid = open(ped_file, 'r')\n",
    "for line in fid:\n",
    "    famid, sid, fa, mo, sex, aff = line.rstrip().split('\\t')\n",
    "    if (fa[:4] in wes_prefix and mo[:4] in wes_prefix and\n",
    "        sid[:4] in wes_prefix and sid not in exclude_list):\n",
    "        fam = {}\n",
    "        fam['child'] = sid\n",
    "        if aff == '1':\n",
    "            affected.append(sid)\n",
    "        else:\n",
    "            controls.append(sid)\n",
    "        fam['father'] = fa\n",
    "        fam['mother'] = mo\n",
    "        fam['famid'] = famid\n",
    "        trios.append(fam)\n",
    "        samples += [sid, fa, mo]\n",
    "        famids.append(famid)\n",
    "fid.close()\n",
    "samples = set(samples)\n",
    "famids = set(famids)\n",
    "good_kids = [t['child'] for t in trios]\n",
    "\n",
    "print 'Working with %d samples, in %d families' % (len(samples), len(famids))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "def plot_plink_cnvs(fname, t_str, verbose=False):\n",
    "    \n",
    "    import plotly.graph_objs as go\n",
    "    from plotly import tools\n",
    "\n",
    "    df = pd.read_table(fname, delimiter='\\s+', index_col=1)\n",
    "    df.head()\n",
    "\n",
    "    x_red, x_green, red, green, xticks = [], [], [], [], []\n",
    "    red_text, green_text = [], []\n",
    "\n",
    "    # loop through families\n",
    "    f = 0\n",
    "    for fam in famids:\n",
    "        fam_kids = [t['child'] for t in trios if t['famid'] == fam]\n",
    "        found = False\n",
    "        for kid in fam_kids:\n",
    "            if kid in good_kids:\n",
    "                found = True\n",
    "                idx = [i for i in range(df.shape[0]) if df.index[i].find(kid) >= 0][0]\n",
    "                if kid in affected:\n",
    "                    red.append(int(df.iloc[idx]['NSEG']))\n",
    "                    x_red.append(f)\n",
    "                    red_text.append(kid)\n",
    "                else:\n",
    "                    green.append(int(df.iloc[idx]['NSEG']))\n",
    "                    x_green.append(f)\n",
    "                    green_text.append(kid)\n",
    "        # only increase counter if we added a kid\n",
    "        if found:\n",
    "            xticks.append(fam)\n",
    "            f += 1\n",
    "\n",
    "    fig = go.Figure()\n",
    "                \n",
    "    trace0 = go.Scatter(\n",
    "        x = x_red,\n",
    "        y = red,\n",
    "        mode = 'markers',\n",
    "        name = 'affected',\n",
    "        marker = dict(size = 10, color = 'red'),\n",
    "        text = red_text,\n",
    "        hoverinfo='text+y',\n",
    "        showlegend = True\n",
    "    )\n",
    "    trace1 = go.Scatter(\n",
    "        x = x_green,\n",
    "        y = green,\n",
    "        mode = 'markers',\n",
    "        name = 'unaffected',\n",
    "        marker = dict(size = 10, color = 'green'),\n",
    "        hovertext = green_text,\n",
    "        hoverinfo='text+y',\n",
    "        showlegend = True\n",
    "    )\n",
    "    fig['data'] = [trace0, trace1]\n",
    "    fig['layout'].update(height=400, width=800, title=t_str,\n",
    "                             xaxis1=dict(tickvals=range(len(xticks)),\n",
    "                                        ticktext=xticks,\n",
    "                                        zeroline = False),\n",
    "                             hovermode='closest')\n",
    "    iplot(fig)\n",
    "    \n",
    "    # print family order\n",
    "    if verbose:\n",
    "        score = []\n",
    "        for x in range(max(x_green + x_red)):\n",
    "            idx = [i for i, val in enumerate(x_red) if val == x]\n",
    "            if len(idx) == 0:\n",
    "                aff = 0\n",
    "            else:\n",
    "                aff = red[idx[0]]\n",
    "            idx = [i for i, val in enumerate(x_green) if val == x]\n",
    "            if len(idx) == 0:\n",
    "                unaff = 0\n",
    "            else:\n",
    "                unaff = max([green[i] for i in idx])\n",
    "            score.append(aff - unaff)\n",
    "        order = np.argsort(score)[::-1]\n",
    "        fam_names = fig.layout.xaxis1['ticktext']\n",
    "        disrupted = [fam_names[v] for v in order if score[v] > 0]\n",
    "        print 'Best disrupted:', ', '.join(disrupted)\n",
    "        protected = [fam_names[v] for v in order[::-1] if score[v] < 0]\n",
    "        print 'Best protected:', ', '.join(protected)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "all_q60_clean_brainGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"642f0928-1325-40a7-8c5e-848236679b6d\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"642f0928-1325-40a7-8c5e-848236679b6d\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_brainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"642f0928-1325-40a7-8c5e-848236679b6d\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"642f0928-1325-40a7-8c5e-848236679b6d\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_brainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 10153\n",
      "Best protected: 1976\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          1,
          1,
          8,
          0,
          58,
          0,
          7,
          4,
          6,
          0,
          1,
          0,
          3,
          2,
          3,
          3,
          1,
          1,
          0,
          4
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          1,
          4,
          2,
          2,
          5,
          1,
          1,
          1,
          1,
          15,
          5,
          3,
          1,
          1,
          3,
          0,
          0,
          0,
          1,
          4,
          1,
          3,
          10,
          0,
          0,
          2,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "all_q60_clean_burden",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"8d5f1ac1-13e2-4c9d-a20c-ace789e0bfbb\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"8d5f1ac1-13e2-4c9d-a20c-ace789e0bfbb\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 1, 8, 0, 58, 0, 7, 4, 6, 0, 1, 0, 3, 2, 3, 3, 1, 1, 0, 4], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 4, 2, 2, 5, 1, 1, 1, 1, 15, 5, 3, 1, 1, 3, 0, 0, 0, 1, 4, 1, 3, 10, 0, 0, 2, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_burden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"8d5f1ac1-13e2-4c9d-a20c-ace789e0bfbb\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"8d5f1ac1-13e2-4c9d-a20c-ace789e0bfbb\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 1, 8, 0, 58, 0, 7, 4, 6, 0, 1, 0, 3, 2, 3, 3, 1, 1, 0, 4], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 4, 2, 2, 5, 1, 1, 1, 1, 15, 5, 3, 1, 1, 3, 0, 0, 0, 1, 4, 1, 3, 10, 0, 0, 2, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_burden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 10090, 10164, 10033, 10197, 10448\n",
      "Best protected: 1976, 10369, 10131, 855, 10153, 10173, 10128, 10094, 10182, 10215, 1895\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "all_q60_clean_delBrainGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"654705a5-7c02-4591-bc22-7eace1c0158f\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"654705a5-7c02-4591-bc22-7eace1c0158f\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_delBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"654705a5-7c02-4591-bc22-7eace1c0158f\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"654705a5-7c02-4591-bc22-7eace1c0158f\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_delBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 10153\n",
      "Best protected: 1976\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          1,
          1,
          1,
          0,
          57,
          0,
          6,
          4,
          0,
          0,
          1,
          0,
          0,
          2,
          3,
          0,
          1,
          1,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          3,
          0,
          1,
          0,
          0,
          0,
          1,
          0,
          13,
          0,
          1,
          1,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          1,
          0,
          0,
          2,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "all_q60_clean_delBurden",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"083e21bc-a49a-4029-bdac-95423bd821a5\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"083e21bc-a49a-4029-bdac-95423bd821a5\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 1, 1, 0, 57, 0, 6, 4, 0, 0, 1, 0, 0, 2, 3, 0, 1, 1, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 3, 0, 1, 0, 0, 0, 1, 0, 13, 0, 1, 1, 0, 1, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 2, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_delBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"083e21bc-a49a-4029-bdac-95423bd821a5\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"083e21bc-a49a-4029-bdac-95423bd821a5\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 1, 1, 0, 57, 0, 6, 4, 0, 0, 1, 0, 0, 2, 3, 0, 1, 1, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 3, 0, 1, 0, 0, 0, 1, 0, 13, 0, 1, 1, 0, 1, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 2, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_delBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 10448, 10369, 10178, 10197, 10090, 10128\n",
      "Best protected: 1976, 855, 10131, 10164, 1895\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          1,
          1,
          1,
          0,
          56,
          0,
          4,
          4,
          0,
          0,
          1,
          0,
          0,
          2,
          3,
          0,
          1,
          1,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          3,
          0,
          1,
          0,
          0,
          0,
          1,
          0,
          13,
          0,
          1,
          1,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          1,
          0,
          0,
          2,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "all_q60_clean_delGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"da6215a7-3711-4d72-acba-dfeec93dc948\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"da6215a7-3711-4d72-acba-dfeec93dc948\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 1, 1, 0, 56, 0, 4, 4, 0, 0, 1, 0, 0, 2, 3, 0, 1, 1, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 3, 0, 1, 0, 0, 0, 1, 0, 13, 0, 1, 1, 0, 1, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 2, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_delGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"da6215a7-3711-4d72-acba-dfeec93dc948\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"da6215a7-3711-4d72-acba-dfeec93dc948\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 1, 1, 0, 56, 0, 4, 4, 0, 0, 1, 0, 0, 2, 3, 0, 1, 1, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 3, 0, 1, 0, 0, 0, 1, 0, 13, 0, 1, 1, 0, 1, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 2, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_delGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 10448, 10369, 10178, 10197, 10090, 10128\n",
      "Best protected: 1976, 855, 10131, 10164, 1895\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "all_q60_clean_dupBrainGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"4b7d316d-5894-4563-9d88-f0beb9cb71f6\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"4b7d316d-5894-4563-9d88-f0beb9cb71f6\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_dupBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"4b7d316d-5894-4563-9d88-f0beb9cb71f6\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"4b7d316d-5894-4563-9d88-f0beb9cb71f6\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_dupBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: \n",
      "Best protected: \n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          7,
          0,
          1,
          0,
          1,
          0,
          6,
          0,
          0,
          0,
          3,
          0,
          0,
          3,
          0,
          0,
          0,
          4
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          1,
          1,
          2,
          1,
          5,
          1,
          1,
          0,
          1,
          2,
          5,
          2,
          0,
          1,
          2,
          0,
          0,
          0,
          1,
          4,
          1,
          2,
          9,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "all_q60_clean_dupBurden",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"7b267fb6-73b6-47ab-a739-b3e9ae94b129\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"7b267fb6-73b6-47ab-a739-b3e9ae94b129\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 7, 0, 1, 0, 1, 0, 6, 0, 0, 0, 3, 0, 0, 3, 0, 0, 0, 4], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 1, 2, 1, 5, 1, 1, 0, 1, 2, 5, 2, 0, 1, 2, 0, 0, 0, 1, 4, 1, 2, 9, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_dupBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"7b267fb6-73b6-47ab-a739-b3e9ae94b129\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"7b267fb6-73b6-47ab-a739-b3e9ae94b129\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 7, 0, 1, 0, 1, 0, 6, 0, 0, 0, 3, 0, 0, 3, 0, 0, 0, 4], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 1, 2, 1, 5, 1, 1, 0, 1, 2, 5, 2, 0, 1, 2, 0, 0, 0, 1, 4, 1, 2, 9, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_dupBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10164, 10090, 10033\n",
      "Best protected: 10369, 10131, 1976, 10128, 10448, 10153, 10178, 10182, 10215, 10173, 10094, 855\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          7,
          0,
          0,
          0,
          1,
          0,
          6,
          0,
          0,
          0,
          3,
          0,
          0,
          3,
          0,
          0,
          0,
          4
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          1,
          1,
          2,
          1,
          5,
          1,
          1,
          0,
          1,
          2,
          5,
          2,
          0,
          1,
          2,
          0,
          0,
          0,
          1,
          4,
          1,
          2,
          8,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "all_q60_clean_dupGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"982bf9c2-533f-4237-ac71-bd6687585ba0\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"982bf9c2-533f-4237-ac71-bd6687585ba0\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 7, 0, 0, 0, 1, 0, 6, 0, 0, 0, 3, 0, 0, 3, 0, 0, 0, 4], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 1, 2, 1, 5, 1, 1, 0, 1, 2, 5, 2, 0, 1, 2, 0, 0, 0, 1, 4, 1, 2, 8, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_dupGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"982bf9c2-533f-4237-ac71-bd6687585ba0\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"982bf9c2-533f-4237-ac71-bd6687585ba0\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 7, 0, 0, 0, 1, 0, 6, 0, 0, 0, 3, 0, 0, 3, 0, 0, 0, 4], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 1, 2, 1, 5, 1, 1, 0, 1, 2, 5, 2, 0, 1, 2, 0, 0, 0, 1, 4, 1, 2, 8, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_dupGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10164, 10090, 10033\n",
      "Best protected: 10369, 10131, 1976, 10128, 10448, 10153, 10178, 10182, 10215, 10173, 10094, 10406, 855\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          1,
          1,
          8,
          0,
          56,
          0,
          5,
          4,
          6,
          0,
          1,
          0,
          3,
          2,
          3,
          3,
          1,
          1,
          0,
          4
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          1,
          4,
          2,
          2,
          5,
          1,
          1,
          1,
          1,
          15,
          5,
          3,
          1,
          1,
          3,
          0,
          0,
          0,
          1,
          4,
          1,
          3,
          9,
          0,
          0,
          2,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "all_q60_clean_genes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"46f415b2-e0e3-432d-b0ed-8f1686994869\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"46f415b2-e0e3-432d-b0ed-8f1686994869\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 1, 8, 0, 56, 0, 5, 4, 6, 0, 1, 0, 3, 2, 3, 3, 1, 1, 0, 4], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 4, 2, 2, 5, 1, 1, 1, 1, 15, 5, 3, 1, 1, 3, 0, 0, 0, 1, 4, 1, 3, 9, 0, 0, 2, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_genes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"46f415b2-e0e3-432d-b0ed-8f1686994869\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"46f415b2-e0e3-432d-b0ed-8f1686994869\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 1, 8, 0, 56, 0, 5, 4, 6, 0, 1, 0, 3, 2, 3, 3, 1, 1, 0, 4], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [1, 4, 2, 2, 5, 1, 1, 1, 1, 15, 5, 3, 1, 1, 3, 0, 0, 0, 1, 4, 1, 3, 9, 0, 0, 2, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"all_q60_clean_genes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 10090, 10164, 10033, 10197, 10448\n",
      "Best protected: 1976, 10369, 10131, 855, 10153, 10173, 10128, 10094, 10182, 10215, 1895\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_q60_clean_brainGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"2bdf03c4-8729-4732-a6ae-15f2cd042ad0\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"2bdf03c4-8729-4732-a6ae-15f2cd042ad0\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_brainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"2bdf03c4-8729-4732-a6ae-15f2cd042ad0\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"2bdf03c4-8729-4732-a6ae-15f2cd042ad0\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_brainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: \n",
      "Best protected: \n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          1,
          3,
          0,
          33,
          0,
          2,
          1,
          4,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          1,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          2,
          0,
          0,
          1,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_q60_clean_burden",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"8ff2445e-2bda-4f04-9e74-e009d4dbd6ae\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"8ff2445e-2bda-4f04-9e74-e009d4dbd6ae\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 1, 3, 0, 33, 0, 2, 1, 4, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2, 0, 0, 1, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_burden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"8ff2445e-2bda-4f04-9e74-e009d4dbd6ae\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"8ff2445e-2bda-4f04-9e74-e009d4dbd6ae\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 1, 3, 0, 33, 0, 2, 1, 4, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2, 0, 0, 1, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_burden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 10164, 10090, 855, 1976\n",
      "Best protected: 10131, 10369, 1895\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_q60_clean_delBrainGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"30aec02e-96fb-4b06-bb4d-784c6ccdc443\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"30aec02e-96fb-4b06-bb4d-784c6ccdc443\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_delBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"30aec02e-96fb-4b06-bb4d-784c6ccdc443\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"30aec02e-96fb-4b06-bb4d-784c6ccdc443\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_delBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: \n",
      "Best protected: \n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          1,
          0,
          0,
          33,
          0,
          2,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_q60_clean_delBurden",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"d42848e5-a41a-4e01-b0f2-642bcefe9ec5\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"d42848e5-a41a-4e01-b0f2-642bcefe9ec5\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 1, 0, 0, 33, 0, 2, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_delBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"d42848e5-a41a-4e01-b0f2-642bcefe9ec5\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"d42848e5-a41a-4e01-b0f2-642bcefe9ec5\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 1, 0, 0, 33, 0, 2, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_delBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 855, 10369, 1976, 10448\n",
      "Best protected: 1895\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          1,
          0,
          0,
          32,
          0,
          1,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_q60_clean_delGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"8cf1c829-5e56-4e20-9b83-720b1227b428\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"8cf1c829-5e56-4e20-9b83-720b1227b428\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 1, 0, 0, 32, 0, 1, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_delGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"8cf1c829-5e56-4e20-9b83-720b1227b428\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"8cf1c829-5e56-4e20-9b83-720b1227b428\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 1, 0, 0, 32, 0, 1, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_delGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 855, 10369, 10448\n",
      "Best protected: 1895\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_q60_clean_dupBrainGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"014d1031-691e-434b-ad98-15c73ede461b\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"014d1031-691e-434b-ad98-15c73ede461b\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_dupBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"014d1031-691e-434b-ad98-15c73ede461b\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"014d1031-691e-434b-ad98-15c73ede461b\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_dupBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: \n",
      "Best protected: \n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          3,
          0,
          0,
          0,
          0,
          0,
          4,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          2,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_q60_clean_dupBurden",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"143c10b2-34b2-499a-845f-a3fa8edcf14d\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"143c10b2-34b2-499a-845f-a3fa8edcf14d\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 3, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_dupBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"143c10b2-34b2-499a-845f-a3fa8edcf14d\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"143c10b2-34b2-499a-845f-a3fa8edcf14d\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 3, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_dupBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10164, 10090\n",
      "Best protected: 10369, 10131, 10448\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          3,
          0,
          0,
          0,
          0,
          0,
          4,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_q60_clean_dupGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"ab40a622-e48d-4849-ad50-d4e74df23cc2\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"ab40a622-e48d-4849-ad50-d4e74df23cc2\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 3, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_dupGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"ab40a622-e48d-4849-ad50-d4e74df23cc2\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"ab40a622-e48d-4849-ad50-d4e74df23cc2\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 3, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_dupGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10164, 10090\n",
      "Best protected: 10131, 10369, 10448\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          1,
          3,
          0,
          32,
          0,
          1,
          1,
          4,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          1,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          1,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "denovo_q60_clean_genes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"427bc2e0-8e24-4d1b-a0b7-9c33d7d79dbd\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"427bc2e0-8e24-4d1b-a0b7-9c33d7d79dbd\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 1, 3, 0, 32, 0, 1, 1, 4, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 1, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_genes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"427bc2e0-8e24-4d1b-a0b7-9c33d7d79dbd\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"427bc2e0-8e24-4d1b-a0b7-9c33d7d79dbd\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 1, 3, 0, 32, 0, 1, 1, 4, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 1, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"denovo_q60_clean_genes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10406, 10164, 10090, 855\n",
      "Best protected: 10131, 1895\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "inherited_q60_clean_brainGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"5561d663-b5ae-4295-9e65-572d99fe9d9a\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"5561d663-b5ae-4295-9e65-572d99fe9d9a\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_brainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"5561d663-b5ae-4295-9e65-572d99fe9d9a\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"5561d663-b5ae-4295-9e65-572d99fe9d9a\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_brainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: \n",
      "Best protected: \n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          2,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "inherited_q60_clean_burden",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"2c97ba13-8cd2-4bb4-b083-e6ed088fd3a4\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"2c97ba13-8cd2-4bb4-b083-e6ed088fd3a4\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 2, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_burden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"2c97ba13-8cd2-4bb4-b083-e6ed088fd3a4\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"2c97ba13-8cd2-4bb4-b083-e6ed088fd3a4\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 2, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_burden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 1976, 10448\n",
      "Best protected: 855, 10406\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "inherited_q60_clean_delBrainGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"744f28cf-77e8-401c-8bba-60bc0f7c23b8\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"744f28cf-77e8-401c-8bba-60bc0f7c23b8\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_delBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"744f28cf-77e8-401c-8bba-60bc0f7c23b8\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"744f28cf-77e8-401c-8bba-60bc0f7c23b8\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_delBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: \n",
      "Best protected: \n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "inherited_q60_clean_delBurden",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"0167bf1f-caea-4f05-bc8a-cddee26d0ffb\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"0167bf1f-caea-4f05-bc8a-cddee26d0ffb\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_delBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"0167bf1f-caea-4f05-bc8a-cddee26d0ffb\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"0167bf1f-caea-4f05-bc8a-cddee26d0ffb\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_delBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 1976, 10448\n",
      "Best protected: 855, 10406\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "inherited_q60_clean_delGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"154257e4-0cfe-4bad-8172-cbc250dd327e\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"154257e4-0cfe-4bad-8172-cbc250dd327e\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_delGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"154257e4-0cfe-4bad-8172-cbc250dd327e\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"154257e4-0cfe-4bad-8172-cbc250dd327e\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_delGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 10448\n",
      "Best protected: 855, 10406\n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "inherited_q60_clean_dupBrainGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"92cfeb6d-6f3b-476c-95cf-6b7df687121d\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"92cfeb6d-6f3b-476c-95cf-6b7df687121d\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_dupBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"92cfeb6d-6f3b-476c-95cf-6b7df687121d\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"92cfeb6d-6f3b-476c-95cf-6b7df687121d\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_dupBrainGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: \n",
      "Best protected: \n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "inherited_q60_clean_dupBurden",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"880aca7e-c6db-4a24-ae0d-d2fa2bc7de30\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"880aca7e-c6db-4a24-ae0d-d2fa2bc7de30\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_dupBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"880aca7e-c6db-4a24-ae0d-d2fa2bc7de30\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"880aca7e-c6db-4a24-ae0d-d2fa2bc7de30\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_dupBurden\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 1976\n",
      "Best protected: \n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "inherited_q60_clean_dupGenes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"eae7d8ae-bc62-4a00-aad3-b257cb9d2174\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"eae7d8ae-bc62-4a00-aad3-b257cb9d2174\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_dupGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"eae7d8ae-bc62-4a00-aad3-b257cb9d2174\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"eae7d8ae-bc62-4a00-aad3-b257cb9d2174\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_dupGenes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 1976\n",
      "Best protected: \n"
     ]
    },
    {
     "data": {
      "application/vnd.plotly.v1+json": {
       "data": [
        {
         "hoverinfo": "text+y",
         "marker": {
          "color": "red",
          "size": 10
         },
         "mode": "markers",
         "name": "affected",
         "showlegend": true,
         "text": [
          "CLIA_400131",
          "CLIA_400216",
          "CLIA_400123",
          "CLIA_400140",
          "CLIA_400178",
          "CLIA_400130",
          "CLIA_400204",
          "CLIA_400205",
          "CLIA_400122",
          "CLIA_400149",
          "CLIA_400128",
          "CLIA_400125",
          "CLIA_400121",
          "CLIA_400153",
          "CCGO_800980",
          "CLIA_400127",
          "CLIA_400129",
          "CLIA_400132",
          "CLIA_400134",
          "CLIA_400163"
         ],
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        },
        {
         "hoverinfo": "text+y",
         "hovertext": [
          "CLIA_400159",
          "CLIA_400133",
          "CLIA_400195",
          "CLIA_400138",
          "CLIA_400139",
          "CLIA_400142",
          "CLIA_400179",
          "CLIA_400180",
          "CLIA_400144",
          "CLIA_400203",
          "CLIA_400201",
          "CLIA_400200",
          "CLIA_400135",
          "CLIA_400148",
          "CLIA_400210",
          "CLIA_400212",
          "CLIA_400191",
          "CLIA_400189",
          "CLIA_400193",
          "CLIA_400170",
          "CLIA_400172",
          "CLIA_400152",
          "CCGO_800979",
          "CLIA_400166",
          "CLIA_400181",
          "CLIA_400158",
          "CLIA_400175",
          "CLIA_400162"
         ],
         "marker": {
          "color": "green",
          "size": 10
         },
         "mode": "markers",
         "name": "unaffected",
         "showlegend": true,
         "type": "scatter",
         "x": [
          0,
          1,
          2,
          3,
          3,
          3,
          4,
          4,
          5,
          6,
          6,
          7,
          8,
          9,
          10,
          10,
          11,
          11,
          11,
          12,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "y": [
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0
         ]
        }
       ],
       "layout": {
        "height": 400,
        "hovermode": "closest",
        "title": "inherited_q60_clean_genes",
        "width": 800,
        "xaxis1": {
         "ticktext": [
          "10178",
          "855",
          "10090",
          "10131",
          "10406",
          "10094",
          "1976",
          "10448",
          "10164",
          "10173",
          "10153",
          "10215",
          "10182",
          "10128",
          "10369",
          "10033",
          "10197",
          "1895",
          "1892",
          "1893"
         ],
         "tickvals": [
          0,
          1,
          2,
          3,
          4,
          5,
          6,
          7,
          8,
          9,
          10,
          11,
          12,
          13,
          14,
          15,
          16,
          17,
          18,
          19
         ],
         "zeroline": false
        }
       }
      },
      "text/html": [
       "<div id=\"65f29eca-6a1c-487a-9c3d-0bbc7ee5da0d\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"65f29eca-6a1c-487a-9c3d-0bbc7ee5da0d\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_genes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ],
      "text/vnd.plotly.v1+html": [
       "<div id=\"65f29eca-6a1c-487a-9c3d-0bbc7ee5da0d\" style=\"height: 400px; width: 800px;\" class=\"plotly-graph-div\"></div><script type=\"text/javascript\">require([\"plotly\"], function(Plotly) { window.PLOTLYENV=window.PLOTLYENV || {};window.PLOTLYENV.BASE_URL=\"https://plot.ly\";Plotly.newPlot(\"65f29eca-6a1c-487a-9c3d-0bbc7ee5da0d\", [{\"showlegend\": true, \"name\": \"affected\", \"text\": [\"CLIA_400131\", \"CLIA_400216\", \"CLIA_400123\", \"CLIA_400140\", \"CLIA_400178\", \"CLIA_400130\", \"CLIA_400204\", \"CLIA_400205\", \"CLIA_400122\", \"CLIA_400149\", \"CLIA_400128\", \"CLIA_400125\", \"CLIA_400121\", \"CLIA_400153\", \"CCGO_800980\", \"CLIA_400127\", \"CLIA_400129\", \"CLIA_400132\", \"CLIA_400134\", \"CLIA_400163\"], \"marker\": {\"color\": \"red\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}, {\"showlegend\": true, \"name\": \"unaffected\", \"hovertext\": [\"CLIA_400159\", \"CLIA_400133\", \"CLIA_400195\", \"CLIA_400138\", \"CLIA_400139\", \"CLIA_400142\", \"CLIA_400179\", \"CLIA_400180\", \"CLIA_400144\", \"CLIA_400203\", \"CLIA_400201\", \"CLIA_400200\", \"CLIA_400135\", \"CLIA_400148\", \"CLIA_400210\", \"CLIA_400212\", \"CLIA_400191\", \"CLIA_400189\", \"CLIA_400193\", \"CLIA_400170\", \"CLIA_400172\", \"CLIA_400152\", \"CCGO_800979\", \"CLIA_400166\", \"CLIA_400181\", \"CLIA_400158\", \"CLIA_400175\", \"CLIA_400162\"], \"marker\": {\"color\": \"green\", \"size\": 10}, \"mode\": \"markers\", \"hoverinfo\": \"text+y\", \"y\": [0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], \"x\": [0, 1, 2, 3, 3, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10, 10, 11, 11, 11, 12, 12, 13, 14, 15, 16, 17, 18, 19], \"type\": \"scatter\"}], {\"width\": 800, \"hovermode\": \"closest\", \"title\": \"inherited_q60_clean_genes\", \"xaxis1\": {\"zeroline\": false, \"ticktext\": [\"10178\", \"855\", \"10090\", \"10131\", \"10406\", \"10094\", \"1976\", \"10448\", \"10164\", \"10173\", \"10153\", \"10215\", \"10182\", \"10128\", \"10369\", \"10033\", \"10197\", \"1895\", \"1892\", \"1893\"], \"tickvals\": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]}, \"height\": 400}, {\"linkText\": \"Export to plot.ly\", \"showLink\": true})});</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Best disrupted: 1976, 10448\n",
      "Best protected: 855, 10406\n"
     ]
    }
   ],
   "source": [
    "import glob\n",
    "\n",
    "data_dir = '/home/sudregp/data/cnv/xhmm_multikids/'\n",
    "files = glob.glob(data_dir + '/*q60*clean*cnv.indiv')\n",
    "files.sort()\n",
    "for f in files:\n",
    "    t_str = '.'.join(f.split('/')[-1].split('.')[:-2])\n",
    "    plot_plink_cnvs(f, t_str, verbose=True)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python [default]",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.14"
  },
  "nav_menu": {},
  "toc": {
   "navigate_menu": true,
   "number_sections": true,
   "sideBar": true,
   "threshold": 6,
   "toc_cell": false,
   "toc_section_display": "block",
   "toc_window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
